<!DOCTYPE html>
<!-- saved from url=(0091)https://ec2-107-22-71-56.compute-1.amazonaws.com/a0aae014-bb13-428b-9212-116316e75020/print -->
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<title>IPython Notebook</title>
<link rel="stylesheet" href="./static-ngs-10-blast_files/jquery-ui.min.css" type="text/css">
<link rel="stylesheet" href="./static-ngs-10-blast_files/boilerplate.css" type="text/css">
<link rel="stylesheet" href="./static-ngs-10-blast_files/fbm.css" type="text/css">
<link rel="stylesheet" href="./static-ngs-10-blast_files/page.css" type="text/css">


<script type="text/javascript" src="./static-ngs-10-blast_files/MathJax.js" charset="utf-8"></script>

<script type="text/javascript">
// MathJax disabled, set as null to distingish from *missing* MathJax,
// where it will be undefined, and should prompt a dialog later.
window.mathjax_url = "/static/mathjax/MathJax.js";
</script>
<link rel="stylesheet" href="https://ec2-107-22-71-56.compute-1.amazonaws.com/static/codemirror/lib/codemirror.css?v=63efb">
<link rel="stylesheet" href="https://ec2-107-22-71-56.compute-1.amazonaws.com/static/codemirror/theme/ipython.css?v=fa49d">
<link rel="stylesheet" href="https://ec2-107-22-71-56.compute-1.amazonaws.com/static/prettify/prettify.css?v=83f40">
<link rel="stylesheet" href="./static-ngs-10-blast_files/notebook.css" type="text/css">
<link rel="stylesheet" href="./static-ngs-10-blast_files/printnotebook.css" type="text/css">
<link rel="stylesheet" href="./static-ngs-10-blast_files/renderedhtml.css" type="text/css">



<style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-family: serif; font-size: 85%; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuTitle {background-color: #CCCCCC; margin: -5px 0 0 0; text-align: center; font-style: italic; font-size: 80%; color: #444444; padding: 2px 0; overflow: hidden}
.MathJax_MenuArrow {position: absolute; right: .5em; color: #666666}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
</style><style type="text/css">#MathJax_Message {position: fixed; left: 1em; bottom: 1.5em; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Display {text-align: center; margin: 0; position: relative; display: block; width: 100%}
.MathJax .merror {color: black; border: 1px solid; padding: 1px 3px; font-family: serif; font-size: 80%; vertical-align: ; text-align: left}
.MathJax_Preview {color: #888888}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-family: serif; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; border: 0; padding: 0; margin: 0}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0 !important; padding: 0 !important; margin: 0 !important; vertical-align: 0 !important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax .MathJax_HitBox {cursor: text}
#MathJax_Tooltip * {filter: none; background: transparent}
#MathJax_getScales {font-family: STIXGeneral,STIXNonUnicode,STIXSizeOneSym}
.MathJax .math span {font-family: STIXGeneral,STIXNonUnicode,STIXSizeOneSym}
</style></head>
<body data-project="/usr/local/notebooks" data-base-project-url="/" data-base-kernel-url="/" data-read-only="False" data-notebook-id="a0aae014-bb13-428b-9212-116316e75020"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; "><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none; "></div>
<div id="header" class="border-box-sizing ui-widget ui-widget-content" style="border-top-style: none; border-left-style: none; border-right-style: none; ">
<span id="ipython_notebook"><h1><a href="https://ec2-107-22-71-56.compute-1.amazonaws.com/" alt="dashboard"><img src="./static-ngs-10-blast_files/ipynblogo.png" alt="IPython Notebook"></a></h1></span>

<span id="login_widget">

<button id="logout" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-text-only" role="button" aria-disabled="false"><span class="ui-button-text">Logout</span></button>

</span>



</div>
<div id="site" class="border-box-sizing" style="display: block; ">

<div id="main_app" class="border-box-sizing ui-widget">
<div id="notebook_panel" class="border-box-sizing ui-widget">
<div id="notebook" class="border-box-sizing"><div class="cell border-box-sizing code_cell vbox ui-widget-content ui-corner-all" tabindex="2"><div class="input hbox"><div class="prompt input_prompt">In&nbsp;[16]:</div><div class="input_area box-flex1"><div class="CodeMirror CodeMirror-focused"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 5px; left: 5px; "><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-ipython" tabindex="-1"><div style="position: relative; height: 61px; "><div style="position: relative; top: 0px; "><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"></div><pre class="CodeMirror-cursor" style="top: 0px; left: 0px; ">&nbsp;</pre><div style="position: relative; z-index: -1; display: none; "></div><div style=""><pre><span class="cm-comment"># add the scripts/libraries for the blast parser into the Python library path</span></pre><pre><span class="cm-keyword">import</span> <span class="cm-variable">sys</span></pre><pre><span class="cm-variable">sys.path.append</span>(<span class="cm-string">'/usr/local/notebooks/lib'</span>)</pre></div></div></div></div></div></div></div></div></div><div class="output vbox" style="display: none; "></div></div><div class="cell text_cell border-box-sizing" tabindex="2"><div class="text_cell_input border-box-sizing" style="display: none; "><div class="CodeMirror CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px;"><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-default" tabindex="-1"><div style="position: relative"><div style="position: relative"><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"><pre><span>x</span></pre></div><pre class="CodeMirror-cursor">&nbsp;</pre><div style="position: relative; z-index: -1"></div><div></div></div></div></div></div></div></div></div><div class="text_cell_render border-box-sizing rendered_html" tabindex="-1" style=""><p>This shows the current working directory (think: print working directory)</p></div></div><div class="cell border-box-sizing code_cell vbox" tabindex="2"><div class="input hbox"><div class="prompt input_prompt">In&nbsp;[17]:</div><div class="input_area box-flex1"><div class="CodeMirror"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 5px; left: 5px; "><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-ipython" tabindex="-1"><div style="position: relative; height: 27px; "><div style="position: relative; top: 0px; "><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"></div><pre class="CodeMirror-cursor" style="top: 0px; left: 0px; ">&nbsp;</pre><div style="position: relative; z-index: -1; display: none; "></div><div style=""><pre><span class="cm-operator">!</span><span class="cm-variable">pwd</span></pre></div></div></div></div></div></div></div></div></div><div class="output vbox" style=""><div class="hbox output_area"><div class="prompt"></div><div class="box-flex1 output_subarea output_text output_stream output_stdout"><pre>/mnt
</pre></div></div></div></div><div class="cell text_cell border-box-sizing" tabindex="2"><div class="text_cell_input border-box-sizing" style="display: none; "><div class="CodeMirror CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px;"><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-default" tabindex="-1"><div style="position: relative"><div style="position: relative"><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"><pre><span>x</span></pre></div><pre class="CodeMirror-cursor">&nbsp;</pre><div style="position: relative; z-index: -1"></div><div></div></div></div></div></div></div></div></div><div class="text_cell_render border-box-sizing rendered_html" tabindex="-1" style=""><p><strong>First</strong>, go to the temporary disk space.</p></div></div><div class="cell border-box-sizing code_cell vbox" tabindex="2"><div class="input hbox"><div class="prompt input_prompt">In&nbsp;[18]:</div><div class="input_area box-flex1"><div class="CodeMirror"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 5px; left: 5px; "><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-ipython" tabindex="-1"><div style="position: relative; height: 27px; "><div style="position: relative; top: 0px; "><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"></div><pre class="CodeMirror-cursor" style="top: 0px; left: 0px; ">&nbsp;</pre><div style="position: relative; z-index: -1; display: none; "></div><div style=""><pre><span class="cm-variable">cd</span> <span class="cm-operator">/</span><span class="cm-variable">mnt</span></pre></div></div></div></div></div></div></div></div></div><div class="output vbox" style=""><div class="hbox output_area"><div class="prompt"></div><div class="box-flex1 output_subarea output_text output_stream output_stdout"><pre>/mnt
</pre></div></div></div></div><div class="cell border-box-sizing code_cell vbox" tabindex="2"><div class="input hbox"><div class="prompt input_prompt">In&nbsp;[19]:</div><div class="input_area box-flex1"><div class="CodeMirror"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 5px; left: 5px; "><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-ipython" tabindex="-1"><div style="position: relative; height: 27px; "><div style="position: relative; top: 0px; "><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"></div><pre class="CodeMirror-cursor" style="top: 0px; left: 0px; ">&nbsp;</pre><div style="position: relative; z-index: -1; display: none; "></div><div style=""><pre><span class="cm-variable">ls</span></pre></div></div></div></div></div></div></div></div></div><div class="output vbox" style=""><div class="hbox output_area"><div class="prompt"></div><div class="box-flex1 output_subarea output_text output_stream output_stdout"><pre><span class="ansibold ansiblue">blast-2.2.24</span>/                   mouse.protein.faa.psi
blast-2.2.24-ia32-linux.tar.gz  mouse.protein.faa.psq
formatdb.log                    out.txt
<span class="ansibold ansiblue">lost+found</span>/                     zebrafish.protein.faa
mm-first.fa                     zebrafish.protein.faa.phr
mouse.protein.faa               zebrafish.protein.faa.pin
mouse.protein.faa.phr           zebrafish.protein.faa.pnd
mouse.protein.faa.pin           zebrafish.protein.faa.pni
mouse.protein.faa.pnd           zebrafish.protein.faa.psd
mouse.protein.faa.pni           zebrafish.protein.faa.psi
mouse.protein.faa.psd           zebrafish.protein.faa.psq
</pre></div></div></div></div><div class="cell text_cell border-box-sizing" tabindex="2"><div class="text_cell_input border-box-sizing" style="display: none; "><div class="CodeMirror CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px;"><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-default" tabindex="-1"><div style="position: relative"><div style="position: relative"><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"><pre><span>x</span></pre></div><pre class="CodeMirror-cursor">&nbsp;</pre><div style="position: relative; z-index: -1"></div><div></div></div></div></div></div></div></div></div><div class="text_cell_render border-box-sizing rendered_html" tabindex="-1" style=""><h1>A Random Header</h1>

<p>Now I can type all sorts of things in here and they will be nicely formatted when I hit shift-ENTER.</p>

<p>I think you can also do equations <span class="MathJax_Preview" style="display: none; "></span><span class="MathJax" role="textbox" aria-readonly="true" style=""><nobr><span class="math" id="MathJax-Span-6"><span style="display: inline-block; position: relative; height: 0px; font-size: 117%; width: 2.512em; "><span style="position: absolute; top: -2.555em; left: 0em; clip: rect(1.476em 1000em 3.225em -0.565em); "><span class="mrow" id="MathJax-Span-7"><span class="mi" id="MathJax-Span-8" style="font-family: STIXGeneral; font-style: italic; ">x</span><span class="mo" id="MathJax-Span-9" style="font-family: STIXGeneral; padding-left: 0.313em; ">=</span><span class="mfrac" id="MathJax-Span-10" style="padding-left: 0.432em; padding-right: 0.12em; "><span style="display: inline-block; position: relative; height: 0px; width: 0.479em; "><span style="position: absolute; top: -2.977em; clip: rect(1.898em 1000em 2.735em -0.46em); left: 50%; margin-left: -0.179em; "><span class="mn" id="MathJax-Span-11" style="font-size: 70.7%; font-family: STIXGeneral; ">1</span><span style="display: inline-block; width: 0px; height: 2.555em; "></span></span><span style="position: absolute; top: -2.21em; clip: rect(2.064em 1000em 2.88em -0.555em); left: 50%; margin-left: -0.15em; "><span class="mi" id="MathJax-Span-12" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic; ">y</span><span style="display: inline-block; width: 0px; height: 2.555em; "></span></span><span style="position: absolute; top: -2.775em; left: 0em; clip: rect(2.301em 1000em 2.735em -0.538em); "><span style="border-left-width: 0.479em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 0.075em; vertical-align: 0em; "></span><span style="display: inline-block; width: 0px; height: 2.555em; "></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.555em; "></span></span></span><span style="border-left-width: 0em; border-left-style: solid; display: inline-block; overflow: hidden; width: 0px; height: 1.626em; vertical-align: -0.573em; "></span></span></nobr></span><script type="math/tex" id="MathJax-Element-2">x = \frac{1}{y}</script></p></div></div><div class="cell text_cell border-box-sizing" tabindex="2"><div class="text_cell_input border-box-sizing" style="display: none; "><div class="CodeMirror CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px;"><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-default" tabindex="-1"><div style="position: relative"><div style="position: relative"><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"><pre><span>x</span></pre></div><pre class="CodeMirror-cursor">&nbsp;</pre><div style="position: relative; z-index: -1"></div><div></div></div></div></div></div></div></div></div><div class="text_cell_render border-box-sizing rendered_html" tabindex="-1" style=""><p>Let's check out the disks on this baby.</p></div></div><div class="cell border-box-sizing code_cell vbox" tabindex="2"><div class="input hbox"><div class="prompt input_prompt">In&nbsp;[20]:</div><div class="input_area box-flex1"><div class="CodeMirror"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 5px; left: 5px; "><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-ipython" tabindex="-1"><div style="position: relative; height: 27px; "><div style="position: relative; top: 0px; "><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"></div><pre class="CodeMirror-cursor" style="top: 0px; left: 0px; ">&nbsp;</pre><div style="position: relative; z-index: -1; display: none; "></div><div style=""><pre><span class="cm-operator">!</span><span class="cm-variable">df</span></pre></div></div></div></div></div></div></div></div></div><div class="output vbox" style=""><div class="hbox output_area"><div class="prompt"></div><div class="box-flex1 output_subarea output_text output_stream output_stdout"><pre>Filesystem           1K-blocks      Used Available Use% Mounted on
/dev/xvda1            10321208   3381212   6415720  35% /
udev                   3815928         4   3815924   1% /dev
tmpfs                  1529508       164   1529344   1% /run
none                      5120         0      5120   0% /run/lock
none                   3823768         0   3823768   0% /run/shm
/dev/xvdb1           433455904    387188 411050412   1% /mnt
</pre></div></div></div></div><div class="cell text_cell border-box-sizing" tabindex="2"><div class="text_cell_input border-box-sizing" style="display: none; "><div class="CodeMirror CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px;"><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-default" tabindex="-1"><div style="position: relative"><div style="position: relative"><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"><pre><span>x</span></pre></div><pre class="CodeMirror-cursor">&nbsp;</pre><div style="position: relative; z-index: -1"></div><div></div></div></div></div></div></div></div></div><div class="text_cell_render border-box-sizing rendered_html" tabindex="-1" style=""><p><strong>Second</strong>, install BLAST.</p>

<p>Here, we're using curl to download the BLAST distribution from NCBI; then we're using 'tar' to unpack it into the current directory; and then we're copying the program files into the directory /usr/local/bin, where we can run them from anywhere.</p></div></div><div class="cell border-box-sizing code_cell vbox" tabindex="2"><div class="input hbox"><div class="prompt input_prompt">In&nbsp;[21]:</div><div class="input_area box-flex1"><div class="CodeMirror"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 5px; left: 5px; "><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-ipython" tabindex="-1"><div style="position: relative; height: 78px; "><div style="position: relative; top: 0px; "><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"></div><pre class="CodeMirror-cursor" style="top: 0px; left: 0px; ">&nbsp;</pre><div style="position: relative; z-index: -1; display: none; "></div><div style=""><pre><span class="cm-operator">!</span><span class="cm-variable">curl</span> <span class="cm-operator">-</span><span class="cm-variable">O</span> <span class="cm-variable">ftp</span>:<span class="cm-operator">//</span><span class="cm-variable">ftp.ncbi.nih.gov</span><span class="cm-operator">/</span><span class="cm-variable">blast</span><span class="cm-operator">/</span><span class="cm-variable">executables</span><span class="cm-operator">/</span><span class="cm-variable">release</span><span class="cm-operator">/</span><span class="cm-number">2.2.24</span><span class="cm-operator">/</span><span class="cm-variable">blast</span><span class="cm-operator">-</span><span class="cm-number">2.2.24</span><span class="cm-operator">-</span><span class="cm-variable">ia32</span><span class="cm-operator">-</span><span class="cm-variable">linux.tar.gz</span></pre><pre><span class="cm-operator">!</span><span class="cm-variable">tar</span> <span class="cm-variable">xzf</span> <span class="cm-variable">blast</span><span class="cm-operator">-</span><span class="cm-number">2.2.24</span><span class="cm-operator">-</span><span class="cm-variable">ia32</span><span class="cm-operator">-</span><span class="cm-variable">linux.tar.gz</span></pre><pre><span class="cm-operator">!</span><span class="cm-variable">cp</span> <span class="cm-variable">blast</span><span class="cm-operator">-</span><span class="cm-number">2.2.24</span><span class="cm-operator">/</span><span class="cm-builtin">bin</span><span class="cm-operator">/*</span> <span class="cm-operator">/</span><span class="cm-variable">usr</span><span class="cm-operator">/</span><span class="cm-variable">local</span><span class="cm-operator">/</span><span class="cm-builtin">bin</span></pre><pre><span class="cm-operator">!</span><span class="cm-variable">cp</span> <span class="cm-operator">-</span><span class="cm-variable">r</span> <span class="cm-variable">blast</span><span class="cm-operator">-</span><span class="cm-number">2.2.24</span><span class="cm-operator">/</span><span class="cm-variable">data</span> <span class="cm-operator">/</span><span class="cm-variable">usr</span><span class="cm-operator">/</span><span class="cm-variable">local</span><span class="cm-operator">/</span><span class="cm-variable">blast</span><span class="cm-operator">-</span><span class="cm-variable">data</span></pre></div></div></div></div></div></div></div></div></div><div class="output vbox" style=""><div class="hbox output_area"><div class="prompt"></div><div class="box-flex1 output_subarea output_text output_stream output_stdout"><pre>  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 32 34.1M   32 11.0M    0     0  16.6M      0  0:00:02 --:--:--  0:00:02 17.6M
100 34.1M  100 34.1M    0     0  26.6M      0  0:00:01  0:00:01 --:--:-- 27.3M
</pre></div></div></div></div><div class="cell text_cell border-box-sizing" tabindex="2"><div class="text_cell_input border-box-sizing" style="display: none; "><div class="CodeMirror CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px;"><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-default" tabindex="-1"><div style="position: relative"><div style="position: relative"><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"><pre><span>x</span></pre></div><pre class="CodeMirror-cursor">&nbsp;</pre><div style="position: relative; z-index: -1"></div><div></div></div></div></div></div></div></div></div><div class="text_cell_render border-box-sizing rendered_html" tabindex="-1" style=""><p><strong>Once</strong> we have BLAST installed, we can run it to see what options there are.</p></div></div><div class="cell border-box-sizing code_cell vbox" tabindex="2"><div class="input hbox"><div class="prompt input_prompt">In&nbsp;[22]:</div><div class="input_area box-flex1"><div class="CodeMirror"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 5px; left: 5px; "><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-ipython" tabindex="-1"><div style="position: relative; height: 27px; "><div style="position: relative; top: 0px; "><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"></div><pre class="CodeMirror-cursor" style="top: 0px; left: 0px; ">&nbsp;</pre><div style="position: relative; z-index: -1; display: none; "></div><div style=""><pre><span class="cm-operator">!</span><span class="cm-variable">blastall</span></pre></div></div></div></div></div></div></div></div></div><div class="output vbox" style=""><div class="hbox output_area"><div class="prompt"></div><div class="box-flex1 output_subarea output_text output_stream output_stdout"><pre>
blastall 2.2.24   arguments:

  -p  Program Name [String]
  -d  Database [String]
    default = nr
  -i  Query File [File In]
    default = stdin
  -e  Expectation value (E) [Real]
    default = 10.0
  -m  alignment view options:
0 = pairwise,
1 = query-anchored showing identities,
2 = query-anchored no identities,
3 = flat query-anchored, show identities,
4 = flat query-anchored, no identities,
5 = query-anchored no identities and blunt ends,
6 = flat query-anchored, no identities and blunt ends,
7 = XML Blast output,
8 = tabular, 
9 tabular with comment lines
10 ASN, text
11 ASN, binary [Integer]
    default = 0
    range from 0 to 11
  -o  BLAST report Output File [File Out]  Optional
    default = stdout
  -F  Filter query sequence (DUST with blastn, SEG with others) [String]
    default = T
  -G  Cost to open a gap (-1 invokes default behavior) [Integer]
    default = -1
  -E  Cost to extend a gap (-1 invokes default behavior) [Integer]
    default = -1
  -X  X dropoff value for gapped alignment (in bits) (zero invokes default behavior)
      blastn 30, megablast 20, tblastx 0, all others 15 [Integer]
    default = 0
  -I  Show GI's in deflines [T/F]
    default = F
  -q  Penalty for a nucleotide mismatch (blastn only) [Integer]
    default = -3
  -r  Reward for a nucleotide match (blastn only) [Integer]
    default = 1
  -v  Number of database sequences to show one-line descriptions for (V) [Integer]
    default = 500
  -b  Number of database sequence to show alignments for (B) [Integer]
    default = 250
  -f  Threshold for extending hits, default if zero
      blastp 11, blastn 0, blastx 12, tblastn 13
      tblastx 13, megablast 0 [Real]
    default = 0
  -g  Perform gapped alignment (not available with tblastx) [T/F]
    default = T
  -Q  Query Genetic code to use [Integer]
    default = 1
  -D  DB Genetic code (for tblast[nx] only) [Integer]
    default = 1
  -a  Number of processors to use [Integer]
    default = 1
  -O  SeqAlign file [File Out]  Optional
  -J  Believe the query defline [T/F]
    default = F
  -M  Matrix [String]
    default = BLOSUM62
  -W  Word size, default if zero (blastn 11, megablast 28, all others 3) [Integer]
    default = 0
  -z  Effective length of the database (use zero for the real size) [Real]
    default = 0
  -K  Number of best hits from a region to keep. Off by default.
If used a value of 100 is recommended.  Very high values of -v or -b is also suggested [Integer]
    default = 0
  -P  0 for multiple hit, 1 for single hit (does not apply to blastn) [Integer]
    default = 0
  -Y  Effective length of the search space (use zero for the real size) [Real]
    default = 0
  -S  Query strands to search against database (for blast[nx], and tblastx)
       3 is both, 1 is top, 2 is bottom [Integer]
    default = 3
  -T  Produce HTML output [T/F]
    default = F
  -l  Restrict search of database to list of GI's [String]  Optional
  -U  Use lower case filtering of FASTA sequence [T/F]  Optional
  -y  X dropoff value for ungapped extensions in bits (0.0 invokes default behavior)
      blastn 20, megablast 10, all others 7 [Real]
    default = 0.0
  -Z  X dropoff value for final gapped alignment in bits (0.0 invokes default behavior)
      blastn/megablast 100, tblastx 0, all others 25 [Integer]
    default = 0
  -R  PSI-TBLASTN checkpoint file [File In]  Optional
  -n  MegaBlast search [T/F]
    default = F
  -L  Location on query sequence [String]  Optional
  -A  Multiple Hits window size, default if zero (blastn/megablast 0, all others 40 [Integer]
    default = 0
  -w  Frame shift penalty (OOF algorithm for blastx) [Integer]
    default = 0
  -t  Length of the largest intron allowed in a translated nucleotide sequence when linking multiple distinct alignments. (0 invokes default behavior; a negative value disables linking.) [Integer]
    default = 0
  -B  Number of concatenated queries, for blastn and tblastn [Integer]  Optional
    default = 0
  -V  Force use of the legacy BLAST engine [T/F]  Optional
    default = F
  -C  Use composition-based score adjustments for blastp or tblastn:
      As first character:
      D or d: default (equivalent to T)
      0 or F or f: no composition-based statistics
      2 or T or t: Composition-based score adjustments as in Bioinformatics 21:902-911,
      1: Composition-based statistics as in NAR 29:2994-3005, 2001
          2005, conditioned on sequence properties
      3: Composition-based score adjustment as in Bioinformatics 21:902-911,
          2005, unconditionally
      For programs other than tblastn, must either be absent or be D, F or 0.
           As second character, if first character is equivalent to 1, 2, or 3:
      U or u: unified p-value combining alignment p-value and compositional p-value in round 1 only
 [String]
    default = D
  -s  Compute locally optimal Smith-Waterman alignments (This option is only
      available for gapped tblastn.) [T/F]
    default = F

</pre></div></div></div></div><div class="cell text_cell border-box-sizing" tabindex="2"><div class="text_cell_input border-box-sizing" style="display: none; "><div class="CodeMirror CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px;"><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-default" tabindex="-1"><div style="position: relative"><div style="position: relative"><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"><pre><span>x</span></pre></div><pre class="CodeMirror-cursor">&nbsp;</pre><div style="position: relative; z-index: -1"></div><div></div></div></div></div></div></div></div></div><div class="text_cell_render border-box-sizing rendered_html" tabindex="-1" style=""><p>So, let's actually do something! Let's try BLASTing the first few sequences of the mouse RefSeq proteome against the zebrafish protein.</p>

<p>Before we run BLAST, we need to get the FASTA files, then uncompress them.</p></div></div><div class="cell border-box-sizing code_cell vbox" tabindex="2"><div class="input hbox"><div class="prompt input_prompt">In&nbsp;[23]:</div><div class="input_area box-flex1"><div class="CodeMirror"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 5px; left: 5px; "><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-ipython" tabindex="-1"><div style="position: relative; height: 44px; "><div style="position: relative; top: 0px; "><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"></div><pre class="CodeMirror-cursor" style="top: 0px; left: 0px; ">&nbsp;</pre><div style="position: relative; z-index: -1; display: none; "></div><div style=""><pre><span class="cm-operator">!</span><span class="cm-variable">curl</span> <span class="cm-operator">-</span><span class="cm-variable">O</span> <span class="cm-variable">ftp</span>:<span class="cm-operator">//</span><span class="cm-variable">ftp.ncbi.nih.gov</span><span class="cm-operator">/</span><span class="cm-variable">refseq</span><span class="cm-operator">/</span><span class="cm-variable">M_musculus</span><span class="cm-operator">/</span><span class="cm-variable">mRNA_Prot</span><span class="cm-operator">/</span><span class="cm-variable">mouse.protein.faa.gz</span></pre><pre><span class="cm-operator">!</span><span class="cm-variable">curl</span> <span class="cm-operator">-</span><span class="cm-variable">O</span> <span class="cm-variable">ftp</span>:<span class="cm-operator">//</span><span class="cm-variable">ftp.ncbi.nih.gov</span><span class="cm-operator">/</span><span class="cm-variable">refseq</span><span class="cm-operator">/</span><span class="cm-variable">D_rerio</span><span class="cm-operator">/</span><span class="cm-variable">mRNA_Prot</span><span class="cm-operator">/</span><span class="cm-variable">zebrafish.protein.faa.gz</span></pre></div></div></div></div></div></div></div></div></div><div class="output vbox" style=""><div class="hbox output_area"><div class="prompt"></div><div class="box-flex1 output_subarea output_text output_stream output_stdout"><pre>  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 8923k  100 8923k    0     0  15.6M      0 --:--:-- --:--:-- --:--:-- 15.8M
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 8730k  100 8730k    0     0  16.6M      0 --:--:-- --:--:-- --:--:-- 16.8M
</pre></div></div></div></div><div class="cell text_cell border-box-sizing" tabindex="2"><div class="text_cell_input border-box-sizing" style="display: none; "><div class="CodeMirror CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px;"><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-default" tabindex="-1"><div style="position: relative"><div style="position: relative"><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"><pre><span>x</span></pre></div><pre class="CodeMirror-cursor">&nbsp;</pre><div style="position: relative; z-index: -1"></div><div></div></div></div></div></div></div></div></div><div class="text_cell_render border-box-sizing rendered_html" tabindex="-1" style=""><p>Here, you can see all of the files we've downloaded:</p></div></div><div class="cell border-box-sizing code_cell vbox" tabindex="2"><div class="input hbox"><div class="prompt input_prompt">In&nbsp;[24]:</div><div class="input_area box-flex1"><div class="CodeMirror"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 5px; left: 5px; "><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-ipython" tabindex="-1"><div style="position: relative; height: 27px; "><div style="position: relative; top: 0px; "><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"></div><pre class="CodeMirror-cursor" style="top: 0px; left: 0px; ">&nbsp;</pre><div style="position: relative; z-index: -1; display: none; "></div><div style=""><pre><span class="cm-operator">!</span><span class="cm-variable">ls</span> <span class="cm-operator">-</span><span class="cm-variable">l</span></pre></div></div></div></div></div></div></div></div></div><div class="output vbox" style=""><div class="hbox output_area"><div class="prompt"></div><div class="box-flex1 output_subarea output_text output_stream output_stdout"><pre>total 126352
drwxr-xr-x 5 3755 5333     4096 2010-08-08 14:25 blast-2.2.24
-rw-r--r-- 1 root root 35780218 2012-06-05 17:30 blast-2.2.24-ia32-linux.tar.gz
-rw-r--r-- 1 root root      466 2012-06-05 15:38 formatdb.log
drwx------ 2 root root    16384 2011-07-13 09:38 lost+found
-rw-r--r-- 1 root root      857 2012-06-05 15:37 mm-first.fa
-rw-r--r-- 1 root root 18256603 2012-06-05 17:27 mouse.protein.faa
-rw-r--r-- 1 root root  9137384 2012-06-05 17:30 mouse.protein.faa.gz
-rw-r--r-- 1 root root  3566300 2012-06-05 15:38 mouse.protein.faa.phr
-rw-r--r-- 1 root root   231400 2012-06-05 15:38 mouse.protein.faa.pin
-rw-r--r-- 1 root root   231320 2012-06-05 15:38 mouse.protein.faa.pnd
-rw-r--r-- 1 root root      948 2012-06-05 15:38 mouse.protein.faa.pni
-rw-r--r-- 1 root root  1024064 2012-06-05 15:38 mouse.protein.faa.psd
-rw-r--r-- 1 root root    22374 2012-06-05 15:38 mouse.protein.faa.psi
-rw-r--r-- 1 root root 15594018 2012-06-05 15:38 mouse.protein.faa.psq
-rw-r--r-- 1 root root     6347 2012-06-05 15:38 out.txt
-rw-r--r-- 1 root root 16993873 2012-06-05 17:28 zebrafish.protein.faa
-rw-r--r-- 1 root root  8940147 2012-06-05 17:30 zebrafish.protein.faa.gz
-rw-r--r-- 1 root root  3436310 2012-06-05 15:37 zebrafish.protein.faa.phr
-rw-r--r-- 1 root root   218416 2012-06-05 15:38 zebrafish.protein.faa.pin
-rw-r--r-- 1 root root   218328 2012-06-05 15:37 zebrafish.protein.faa.pnd
-rw-r--r-- 1 root root      900 2012-06-05 15:37 zebrafish.protein.faa.pni
-rw-r--r-- 1 root root  1025629 2012-06-05 15:37 zebrafish.protein.faa.psd
-rw-r--r-- 1 root root    23771 2012-06-05 15:38 zebrafish.protein.faa.psi
-rw-r--r-- 1 root root 14409654 2012-06-05 15:37 zebrafish.protein.faa.psq
</pre></div></div></div></div><div class="cell border-box-sizing code_cell vbox" tabindex="2"><div class="input hbox"><div class="prompt input_prompt">In&nbsp;[25]:</div><div class="input_area box-flex1"><div class="CodeMirror"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 5px; left: 5px; "><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-ipython" tabindex="-1"><div style="position: relative; height: 27px; "><div style="position: relative; top: 0px; "><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"></div><pre class="CodeMirror-cursor" style="top: 0px; left: 0px; ">&nbsp;</pre><div style="position: relative; z-index: -1; display: none; "></div><div style=""><pre><span class="cm-operator">!</span><span class="cm-variable">ls</span> <span class="cm-operator">*</span><span class="cm-variable">.faa.gz</span></pre></div></div></div></div></div></div></div></div></div><div class="output vbox" style=""><div class="hbox output_area"><div class="prompt"></div><div class="box-flex1 output_subarea output_text output_stream output_stdout"><pre>mouse.protein.faa.gz  zebrafish.protein.faa.gz
</pre></div></div></div></div><div class="cell text_cell border-box-sizing" tabindex="2"><div class="text_cell_input border-box-sizing" style="display: none; "><div class="CodeMirror CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px;"><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-default" tabindex="-1"><div style="position: relative"><div style="position: relative"><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"><pre><span>x</span></pre></div><pre class="CodeMirror-cursor">&nbsp;</pre><div style="position: relative; z-index: -1"></div><div></div></div></div></div></div></div></div></div><div class="text_cell_render border-box-sizing rendered_html" tabindex="-1" style=""><p>Uncompress 'em.</p></div></div><div class="cell border-box-sizing code_cell vbox" tabindex="2"><div class="input hbox"><div class="prompt input_prompt">In&nbsp;[26]:</div><div class="input_area box-flex1"><div class="CodeMirror"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 5px; left: 5px; "><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-ipython" tabindex="-1"><div style="position: relative; height: 44px; "><div style="position: relative; top: 0px; "><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"></div><pre class="CodeMirror-cursor" style="top: 0px; left: 0px; ">&nbsp;</pre><div style="position: relative; z-index: -1; display: none; "></div><div style=""><pre><span class="cm-operator">!</span><span class="cm-variable">gunzip</span> <span class="cm-operator">-</span><span class="cm-variable">f</span> <span class="cm-operator">*</span><span class="cm-variable">.faa.gz</span></pre><pre><span class="cm-operator">!</span><span class="cm-variable">ls</span></pre></div></div></div></div></div></div></div></div></div><div class="output vbox" style=""><div class="hbox output_area"><div class="prompt"></div><div class="box-flex1 output_subarea output_text output_stream output_stdout"><pre>blast-2.2.24			mouse.protein.faa.psi
blast-2.2.24-ia32-linux.tar.gz	mouse.protein.faa.psq
formatdb.log			out.txt
lost+found			zebrafish.protein.faa
mm-first.fa			zebrafish.protein.faa.phr
mouse.protein.faa		zebrafish.protein.faa.pin
mouse.protein.faa.phr		zebrafish.protein.faa.pnd
mouse.protein.faa.pin		zebrafish.protein.faa.pni
mouse.protein.faa.pnd		zebrafish.protein.faa.psd
mouse.protein.faa.pni		zebrafish.protein.faa.psi
mouse.protein.faa.psd		zebrafish.protein.faa.psq
</pre></div></div></div></div><div class="cell text_cell border-box-sizing" tabindex="2"><div class="text_cell_input border-box-sizing" style="display: none; "><div class="CodeMirror CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px;"><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-default" tabindex="-1"><div style="position: relative"><div style="position: relative"><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"><pre><span>x</span></pre></div><pre class="CodeMirror-cursor">&nbsp;</pre><div style="position: relative; z-index: -1"></div><div></div></div></div></div></div></div></div></div><div class="text_cell_render border-box-sizing rendered_html" tabindex="-1" style=""><p>Let's use the 'head' command to look at the first few sequences.  This takes the first 10 lines of a text file and prints them out.</p>

<p>Note that you can also specify a specific number of lines using '-' -- e.g. 'head -15 filename.txt' will print out the first 15 lines.</p></div></div><div class="cell border-box-sizing code_cell vbox" tabindex="2"><div class="input hbox"><div class="prompt input_prompt">In&nbsp;[27]:</div><div class="input_area box-flex1"><div class="CodeMirror"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 5px; left: 5px; "><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-ipython" tabindex="-1"><div style="position: relative; height: 27px; "><div style="position: relative; top: 0px; "><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"></div><pre class="CodeMirror-cursor" style="top: 0px; left: 0px; ">&nbsp;</pre><div style="position: relative; z-index: -1; display: none; "></div><div style=""><pre><span class="cm-operator">!</span><span class="cm-variable">head</span> <span class="cm-operator">-</span><span class="cm-number">11</span> <span class="cm-variable">mouse.protein.faa</span></pre></div></div></div></div></div></div></div></div></div><div class="output vbox" style=""><div class="hbox output_area"><div class="prompt"></div><div class="box-flex1 output_subarea output_text output_stream output_stdout"><pre>&gt;gi|83029092|ref|XP_357633.3| PREDICTED: similar to Ig kappa chain V-V region L7 precursor [Mus musculus]
MRSLPASELSLPYHQDFRPDSSCPGLTDNHLLCEPTQKPRYPDLGRKRSGENRHRYRDVCISFIRRSFSALCHNPQKLRE
NLKMVSTSQLLGLLLFWTSASRCDIVMTQSPATLSVTPGDRVSLSCRASQSISDYLHWYQQKSHESPRLLIKYASQSISG
IPSRFSGSGSGSDFTLSINSVEPEDVGVYYCQNGHSFPPTMMQTIAKTSRKAEVRG
&gt;gi|70778726|ref|NP_067474.2| WD repeat domain 8 [Mus musculus]
MNFSESFKLSGLLCRFSPDGKYLASCVQYRLVIRDVTTLQILQLYTCLDQIQHIEWSADSLFILCAMYRRGLVQVWSLEQ
PEWHCKIDEGSAGLVASCWSPDGRHILNTTEFHLRITVWSLCTKSVSYIKYPKACQQGLTFTRDGRYLALAERRDCRDYV
SIFVCSDWQLLRHFDTDTQDLTGIEWAPNGCVLAAWDTCLEYKVLLYSLDGRLLSAYCAYEWSLGIKSVAWSPSSQFLAI
GSYDGKVRLLNHVTWKMITEFGHPATINNPKTVVYKEAEKSPLLGLGHLSFPPPRAMAGALSTSESKYEIASGPVSLQTL
KPVADRANPRMGVGMLAFSSDSYFLASRNDNVPNAVWIWDIQKLKLFVVLEHMSPVRSFQWDPQQPRLAICTGGSKVYLW
SPAGCVSVQVPGEGDFPVLGLCWHLSGDSLALLSKDHFCLCFLETKERVGTAYEQRDGMPRT
</pre></div></div></div></div><div class="cell text_cell border-box-sizing" tabindex="2"><div class="text_cell_input border-box-sizing" style="display: none; "><div class="CodeMirror CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px;"><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-default" tabindex="-1"><div style="position: relative"><div style="position: relative"><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"><pre><span>x</span></pre></div><pre class="CodeMirror-cursor">&nbsp;</pre><div style="position: relative; z-index: -1"></div><div></div></div></div></div></div></div></div></div><div class="text_cell_render border-box-sizing rendered_html" tabindex="-1" style=""><p>Let's take the first 11 lines (through to the end of the second sequence -- I know because I cheated) and put them in a new file.  This way we can play with a subset of the data, which is always a good idea when you have lots of data!</p></div></div><div class="cell border-box-sizing code_cell vbox" tabindex="2"><div class="input hbox"><div class="prompt input_prompt">In&nbsp;[28]:</div><div class="input_area box-flex1"><div class="CodeMirror"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 5px; left: 5px; "><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-ipython" tabindex="-1"><div style="position: relative; height: 27px; "><div style="position: relative; top: 0px; "><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"></div><pre class="CodeMirror-cursor" style="top: 0px; left: 0px; ">&nbsp;</pre><div style="position: relative; z-index: -1; display: none; "></div><div style=""><pre><span class="cm-operator">!</span><span class="cm-variable">head</span> <span class="cm-operator">-</span><span class="cm-number">11</span> <span class="cm-variable">mouse.protein.faa</span> <span class="cm-operator">&gt;</span> <span class="cm-variable">mm</span><span class="cm-operator">-</span><span class="cm-variable">first.fa</span></pre></div></div></div></div></div></div></div></div></div><div class="output vbox" style="display: none; "></div></div><div class="cell text_cell border-box-sizing" tabindex="2"><div class="text_cell_input border-box-sizing" style="display: none; "><div class="CodeMirror CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px;"><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-default" tabindex="-1"><div style="position: relative"><div style="position: relative"><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"><pre><span>x</span></pre></div><pre class="CodeMirror-cursor">&nbsp;</pre><div style="position: relative; z-index: -1"></div><div></div></div></div></div></div></div></div></div><div class="text_cell_render border-box-sizing rendered_html" tabindex="-1" style=""><p>Now, format the FASTA files for BLAST.  This makes it easy for BLAST to search them quickly.</p></div></div><div class="cell text_cell border-box-sizing" tabindex="2"><div class="text_cell_input border-box-sizing" style="display: none; "><div class="CodeMirror CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px;"><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-default" tabindex="-1"><div style="position: relative"><div style="position: relative"><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"><pre><span>x</span></pre></div><pre class="CodeMirror-cursor">&nbsp;</pre><div style="position: relative; z-index: -1"></div><div></div></div></div></div></div></div></div></div><div class="text_cell_render border-box-sizing rendered_html" tabindex="-1" style=""><p>Note, you can use 'formatdb -i filename -o T -p F' to format nucleotide databases.</p></div></div><div class="cell border-box-sizing code_cell vbox" tabindex="2"><div class="input hbox"><div class="prompt input_prompt">In&nbsp;[29]:</div><div class="input_area box-flex1"><div class="CodeMirror"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 5px; left: 5px; "><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-ipython" tabindex="-1"><div style="position: relative; height: 44px; "><div style="position: relative; top: 0px; "><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"></div><pre class="CodeMirror-cursor" style="top: 0px; left: 0px; ">&nbsp;</pre><div style="position: relative; z-index: -1; display: none; "></div><div style=""><pre><span class="cm-operator">!</span><span class="cm-variable">formatdb</span> <span class="cm-operator">-</span><span class="cm-variable">i</span> <span class="cm-variable">zebrafish.protein.faa</span> <span class="cm-operator">-</span><span class="cm-variable">o</span> <span class="cm-variable">T</span> <span class="cm-operator">-</span><span class="cm-variable">p</span> <span class="cm-variable">T</span></pre><pre><span class="cm-operator">!</span><span class="cm-variable">formatdb</span> <span class="cm-operator">-</span><span class="cm-variable">i</span> <span class="cm-variable">mouse.protein.faa</span> <span class="cm-operator">-</span><span class="cm-variable">o</span> <span class="cm-variable">T</span> <span class="cm-operator">-</span><span class="cm-variable">p</span> <span class="cm-variable">T</span></pre></div></div></div></div></div></div></div></div></div><div class="output vbox" style="display: none; "></div></div><div class="cell text_cell border-box-sizing" tabindex="2"><div class="text_cell_input border-box-sizing" style="display: none; "><div class="CodeMirror CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px;"><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-default" tabindex="-1"><div style="position: relative"><div style="position: relative"><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"><pre><span>x</span></pre></div><pre class="CodeMirror-cursor">&nbsp;</pre><div style="position: relative; z-index: -1"></div><div></div></div></div></div></div></div></div></div><div class="text_cell_render border-box-sizing rendered_html" tabindex="-1" style=""><p>And now... run a BLAST!</p></div></div><div class="cell border-box-sizing code_cell vbox" tabindex="2"><div class="input hbox"><div class="prompt input_prompt">In&nbsp;[30]:</div><div class="input_area box-flex1"><div class="CodeMirror"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 5px; left: 5px; "><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-ipython" tabindex="-1"><div style="position: relative; height: 27px; "><div style="position: relative; top: 0px; "><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"></div><pre class="CodeMirror-cursor" style="top: 0px; left: 0px; ">&nbsp;</pre><div style="position: relative; z-index: -1; display: none; "></div><div style=""><pre><span class="cm-operator">!</span><span class="cm-variable">blastall</span> <span class="cm-operator">-</span><span class="cm-variable">i</span> <span class="cm-variable">mm</span><span class="cm-operator">-</span><span class="cm-variable">first.fa</span> <span class="cm-operator">-</span><span class="cm-variable">d</span> <span class="cm-variable">zebrafish.protein.faa</span> <span class="cm-operator">-</span><span class="cm-variable">p</span> <span class="cm-variable">blastp</span></pre></div></div></div></div></div></div></div></div></div><div class="output vbox" style=""><div class="hbox output_area"><div class="prompt"></div><div class="box-flex1 output_subarea output_text output_stream output_stdout"><pre>BLASTP 2.2.24 [Aug-08-2010]


Reference: Altschul, Stephen F., Thomas L. Madden, Alejandro A. Schaffer, 
Jinghui Zhang, Zheng Zhang, Webb Miller, and David J. Lipman (1997), 
"Gapped BLAST and PSI-BLAST: a new generation of protein database search
programs",  Nucleic Acids Res. 25:3389-3402.

Reference for compositional score matrix adjustment: Altschul, Stephen F., 
John C. Wootton, E. Michael Gertz, Richa Agarwala, Aleksandr Morgulis,
Alejandro A. Schaffer, and Yi-Kuo Yu (2005) "Protein database searches
using compositionally adjusted substitution matrices", FEBS J. 272:5101-5109.

Query= gi|83029092|ref|XP_357633.3| PREDICTED: similar to Ig kappa
chain V-V region L7 precursor [Mus musculus]
         (216 letters)

Database: zebrafish.protein.faa 
           27,291 sequences; 14,382,362 total letters

Searching..................................................done



                                                                 Score    E
Sequences producing significant alignments:                      (bits) Value

ref|NP_001038873.1| uncharacterized protein LOC751695 precursor ...    74   6e-14
ref|XP_002667009.2| PREDICTED: hypothetical protein LOC793066 [D...    74   1e-13
ref|XP_003201396.1| PREDICTED: ig kappa chain V-I region Scw-lik...    69   3e-12
ref|XP_002665586.1| PREDICTED: ig kappa chain V-III region B6 [D...    67   7e-12
ref|XP_003201398.1| PREDICTED: ig kappa chain V-III region B6-li...    67   1e-11
ref|XP_001920187.2| PREDICTED: hypothetical protein LOC100150116...    65   4e-11
ref|XP_003201399.1| PREDICTED: ig kappa chain V19-17-like [Danio...    62   2e-10
ref|XP_003200636.1| PREDICTED: ig kappa chain V-I region WEA-lik...    54   8e-08
ref|NP_001020667.1| novel immune-type receptor 12 precursor [Dan...    47   1e-05
ref|XP_001923479.1| PREDICTED: hypothetical protein LOC100151290...    46   2e-05
ref|XP_003200433.1| PREDICTED: t-cell receptor beta chain T17T-2...    38   0.005
ref|XP_003198184.1| PREDICTED: ig heavy chain V-I region HG3-lik...    37   0.011
ref|XP_003197959.1| PREDICTED: hypothetical protein LOC100535098...    37   0.015
ref|XP_003197951.1| PREDICTED: ig kappa chain V-III region VH-li...    36   0.020
ref|XP_003197960.1| PREDICTED: hypothetical protein LOC100535742...    36   0.020
ref|XP_003197945.1| PREDICTED: ig lambda chain V-II region BO-li...    36   0.020
ref|NP_001005768.1| novel immune-type receptor 5 precursor [Dani...    35   0.028
ref|XP_003197957.1| PREDICTED: immunoglobulin omega chain-like [...    35   0.028
ref|XP_002666277.1| PREDICTED: hypothetical protein LOC100331885...    35   0.029
ref|NP_001018511.1| ganglioside-induced differentiation-associat...    35   0.053
ref|NP_954581.3| novel immune-type receptor 6a precursor [Danio ...    35   0.057
ref|XP_003198968.1| PREDICTED: hypothetical protein LOC100538107...    34   0.082
ref|XP_003197944.1| PREDICTED: ig kappa chain V-VI region NQ2-6....    34   0.086
ref|NP_001121856.1| novel immune-type receptor 13 isoform 1 prec...    33   0.11 
ref|NP_001232002.1| similar to novel immune-type receptor 5, all...    33   0.12 
ref|XP_003201256.1| PREDICTED: basement membrane-specific hepara...    33   0.13 
ref|NP_001020668.1| novel immune-type receptor 11b precursor [Da...    33   0.14 
ref|XP_001331043.4| PREDICTED: hypothetical protein LOC791658 [D...    33   0.16 
ref|XP_003198238.1| PREDICTED: ig lambda chain V-III region LOI-...    33   0.16 
ref|XP_003198962.1| PREDICTED: hypothetical protein LOC100537152...    33   0.19 
ref|XP_002666721.1| PREDICTED: hypothetical protein LOC100332522...    33   0.20 
ref|XP_003198969.1| PREDICTED: hypothetical protein LOC100538145...    33   0.20 
ref|NP_001020672.1| novel immune-type receptor 11a precursor [Da...    32   0.26 
ref|NP_938160.1| immune-type receptor 1o [Danio rerio]                 32   0.26 
ref|XP_003198963.1| PREDICTED: hypothetical protein LOC100537250...    32   0.27 
ref|XP_003197958.1| PREDICTED: ig lambda chain V-VI region AR-li...    32   0.31 
ref|NP_001018173.1| novel immune-type receptor 1b precursor [Dan...    32   0.36 
ref|NP_001135740.1| immune-type receptor 5-like precursor [Danio...    32   0.40 
ref|XP_003198961.1| PREDICTED: hypothetical protein LOC100537054...    32   0.40 
ref|XP_001333297.2| PREDICTED: hypothetical protein LOC794617 is...    32   0.41 
ref|XP_003197933.1| PREDICTED: hypothetical protein LOC100534730...    32   0.42 
ref|NP_001032660.1| novel immune-type receptor 1n precursor [Dan...    32   0.48 
ref|XP_001921797.2| PREDICTED: LOW QUALITY PROTEIN: hypothetical...    32   0.48 
ref|XP_001333206.2| PREDICTED: hypothetical protein LOC794617 is...    31   0.50 
ref|XP_003197949.1| PREDICTED: immunoglobulin iota chain-like [D...    31   0.53 
ref|NP_938161.1| novel immune-type receptor 1l precursor [Danio ...    31   0.62 
ref|NP_001186301.1| uncharacterized protein LOC100331689 precurs...    31   0.62 
ref|NP_938155.1| novel immune-type receptor 1m precursor [Danio ...    31   0.63 
ref|NP_938162.1| novel immune-type receptor 1k precursor [Danio ...    31   0.64 
ref|XP_003197946.1| PREDICTED: immunoglobulin omega chain-like [...    31   0.64 
ref|XP_002662802.1| PREDICTED: hypothetical protein LOC100330879...    31   0.66 
ref|XP_002661242.2| PREDICTED: ig heavy chain V-III region HIL [...    31   0.75 
ref|XP_002667279.1| PREDICTED: hypothetical protein LOC100330441...    31   0.77 
ref|XP_003197952.1| PREDICTED: immunoglobulin omega chain-like [...    31   0.80 
ref|NP_571547.1| vascular endothelial growth factor receptor kdr...    31   0.82 
ref|XP_001920377.3| PREDICTED: b-cell receptor CD22-like [Danio ...    31   0.85 
ref|NP_001011881.1| novel immune-type receptor 1j precursor [Dan...    30   0.88 
ref|NP_571721.1| novel immune-type receptor 1i [Danio rerio]           30   0.90 
ref|XP_003197934.1| PREDICTED: hypothetical protein LOC100534813...    30   0.92 
ref|XP_003197935.1| PREDICTED: ig heavy chain V-III region CAM-l...    30   0.94 
ref|XP_003197820.1| PREDICTED: hypothetical protein LOC100537163...    30   0.98 
ref|XP_003201143.1| PREDICTED: hypothetical protein LOC100537953...    30   1.00 
ref|NP_938163.1| novel immune-type receptor 1d precursor [Danio ...    30   1.2  
ref|XP_003197947.1| PREDICTED: immunoglobulin omega chain-like [...    30   1.2  
ref|NP_938164.1| novel immune-type receptor 1c precursor [Danio ...    30   1.3  
ref|NP_938169.1| novel immune-type receptor 3d isoform 1 precurs...    30   1.3  
ref|NP_001120804.1| secreted immunoglobulin domain 1 precursor [...    30   1.4  
ref|NP_001245248.1| diverse immunoglobulin domain-containing pro...    30   1.5  
ref|XP_001923759.3| PREDICTED: hypothetical protein LOC100004978...    30   1.6  
ref|NP_938168.1| novel immune-type receptor 3d isoform 2 precurs...    30   1.6  
ref|NP_001007212.1| novel immune-type receptor 3b [Danio rerio]        30   1.9  
ref|NP_991212.1| uncharacterized protein LOC402947 [Danio rerio]       29   2.0  
ref|NP_571727.1| novel immune-type receptor 3a precursor [Danio ...    29   2.2  
ref|NP_001007203.1| novel immune-type receptor precursor [Danio ...    29   2.4  
ref|XP_001921605.2| PREDICTED: hypothetical protein LOC559375 [D...    29   2.7  
ref|XP_002663948.1| PREDICTED: hypothetical protein LOC100329481...    29   2.8  
ref|NP_001153138.1| immunoglobulin superfamily member 3 precurso...    29   2.9  
ref|XP_003197821.1| PREDICTED: polymeric immunoglobulin receptor...    29   2.9  
ref|NP_694508.1| mucosa associated lymphoid tissue lymphoma tran...    29   3.0  
ref|NP_001245161.1| diverse immunoglobulin domain-containing pro...    29   3.2  
ref|NP_001018172.1| novel immune-type receptor 2a precursor [Dan...    29   3.2  
ref|NP_001005578.1| novel immune-type receptor 7b precursor [Dan...    28   3.3  
ref|XP_001342685.4| PREDICTED: hypothetical protein LOC100003040...    28   3.5  
ref|XP_003201336.1| PREDICTED: hypothetical protein LOC100537913...    28   3.7  
ref|NP_001007220.1| novel immune-type receptor 1a precursor [Dan...    28   4.0  
ref|XP_003200226.1| PREDICTED: hypothetical protein LOC100006926...    28   4.0  
ref|NP_001005576.1| novel immune-type receptor 9 precursor [Dani...    28   4.3  
ref|XP_003197826.1| PREDICTED: CMRF35-like molecule 9-like [Dani...    28   4.4  
ref|XP_001923800.1| PREDICTED: titin [Danio rerio]                     28   4.4  
ref|NP_001073429.2| matrix-remodelling associated 8b precursor [...    28   5.2  
ref|XP_003200434.1| PREDICTED: hypothetical protein LOC100534684...    28   5.4  
ref|NP_956958.1| cell adhesion molecule 2a precursor [Danio rerio]     28   5.5  
ref|XP_003197943.1| PREDICTED: immunoglobulin omega chain-like [...    28   6.3  
ref|XP_002662355.1| PREDICTED: myosin light chain kinase, smooth...    28   6.5  
ref|NP_001018542.1| uncharacterized protein LOC553735 [Danio rerio]    28   6.9  
ref|NP_571730.1| novel immune-type receptor 4a isoform 2 precurs...    28   7.0  
ref|NP_001007293.1| novel immune-type receptor 1f precursor [Dan...    28   7.1  
ref|XP_684938.3| PREDICTED: butyrophilin subfamily 2 member A2-l...    28   7.2  
ref|NP_001120726.1| novel immune-type receptor 13 isoform 2 prec...    27   7.3  
ref|XP_001341831.3| PREDICTED: hypothetical protein LOC100001930...    27   7.5  
ref|XP_003199104.1| PREDICTED: leucine-rich repeat neuronal prot...    27   7.7  
ref|XP_001922166.2| PREDICTED: e3 ubiquitin-protein ligase TRIM2...    27   7.8  
ref|XP_003197961.1| PREDICTED: CMRF35-like molecule 5-like [Dani...    27   7.8  
ref|XP_002667765.2| PREDICTED: butyrophilin-like protein 2-like ...    27   8.0  
ref|XP_003199219.1| PREDICTED: myosin light chain kinase, smooth...    27   8.6  
ref|NP_001018583.1| leucine rich repeat containing 4C precursor ...    27   8.7  
ref|XP_003197822.1| PREDICTED: polymeric immunoglobulin receptor...    27   9.0  
ref|XP_003201334.1| PREDICTED: butyrophilin subfamily 2 member A...    27   9.1  
ref|XP_003198941.1| PREDICTED: b-cell receptor CD22-like [Danio ...    27   9.4  
ref|XP_684768.2| PREDICTED: butyrophilin-like protein 8-like [Da...    27   9.5  
ref|XP_001340824.2| PREDICTED: butyrophilin-like protein 8-like ...    27   9.7  

&gt;ref|NP_001038873.1| uncharacterized protein LOC751695 precursor [Danio rerio]
          Length = 238

 Score = 74.3 bits (181), Expect = 6e-14,   Method: Compositional matrix adjust.
 Identities = 41/110 (37%), Positives = 58/110 (52%), Gaps = 7/110 (6%)

Query: 94  LLFWTSASRCD-----IVMTQSPATLSVTPGDRVSLSCRASQS--ISDYLHWYQQKSHES 146
           +L WT A+ C      + +TQ+P+ L   PG  +++ CR S+      Y+ WY QK  E+
Sbjct: 6   ILIWTLAALCGECIGQVTVTQTPSVLLSEPGQSLTVHCRLSRDPVCCYYVSWYLQKPGEA 65

Query: 147 PRLLIKYASQXXXXXXXXXXXXXXXXDFTLSINSVEPEDVGVYYCQNGHS 196
           P+LLI Y +                 DFTLSI+ V+ ED G YYCQ+ HS
Sbjct: 66  PKLLIYYTTVLQSGTASRFSGSGSNSDFTLSISGVQTEDAGHYYCQSYHS 115


&gt;ref|XP_002667009.2| PREDICTED: hypothetical protein LOC793066 [Danio rerio]
          Length = 270

 Score = 73.6 bits (179), Expect = 1e-13,   Method: Compositional matrix adjust.
 Identities = 41/96 (42%), Positives = 55/96 (57%), Gaps = 5/96 (5%)

Query: 105 IVMTQSPATLSVTPGDRVSLSCRASQSIS---DYLHWYQQKSHESPRLLIKYASQ--XXX 159
           + +TQSPA  S+T G+ +++SC+ SQ IS   D + WYQQK  E+P+LLI Y S      
Sbjct: 55  VTVTQSPAVRSITQGETLTISCKTSQPISDCNDCVSWYQQKPGETPKLLIYYISSPYSNT 114

Query: 160 XXXXXXXXXXXXXDFTLSINSVEPEDVGVYYCQNGH 195
                        DFTLSI+ V+ ED G YYC + H
Sbjct: 115 PSRFSGGGSDYGSDFTLSISGVQTEDAGDYYCLSYH 150


&gt;ref|XP_003201396.1| PREDICTED: ig kappa chain V-I region Scw-like [Danio rerio]
          Length = 132

 Score = 68.6 bits (166), Expect = 3e-12,   Method: Compositional matrix adjust.
 Identities = 40/109 (36%), Positives = 55/109 (50%), Gaps = 7/109 (6%)

Query: 94  LLFWTSASRCD-----IVMTQSPATLSVTPGDRVSLSCRASQSISDYLH--WYQQKSHES 146
           +L WT A+ C      + +TQ+P+ L   PG  +++ CR S +     H  WY QK  E+
Sbjct: 14  ILIWTLAALCGECIGQVTVTQTPSVLLSEPGRSLTVHCRFSSNPDCCYHVSWYLQKPGEA 73

Query: 147 PRLLIKYASQXXXXXXXXXXXXXXXXDFTLSINSVEPEDVGVYYCQNGH 195
           P+LLI Y                   DFTLSI+ V+ ED G YYCQ+ H
Sbjct: 74  PKLLIYYTDVLQPGTASRFSGSGSNSDFTLSISGVQTEDAGHYYCQSYH 122


&gt;ref|XP_002665586.1| PREDICTED: ig kappa chain V-III region B6 [Danio rerio]
          Length = 183

 Score = 67.4 bits (163), Expect = 7e-12,   Method: Compositional matrix adjust.
 Identities = 32/86 (37%), Positives = 45/86 (52%)

Query: 108 TQSPATLSVTPGDRVSLSCRASQSISDYLHWYQQKSHESPRLLIKYASQXXXXXXXXXXX 167
            QSP  LS+  G+  S+SC  +  + + + WY QK  ++P+LLI   S            
Sbjct: 89  VQSPECLSIQAGESASISCSGTSGVGNDMSWYLQKPGKAPKLLIYGISHLEPGVSDRFIG 148

Query: 168 XXXXXDFTLSINSVEPEDVGVYYCQN 193
                DFTL+I +V+PED GVYYC  
Sbjct: 149 SGSEPDFTLNIRAVQPEDAGVYYCMG 174


&gt;ref|XP_003201398.1| PREDICTED: ig kappa chain V-III region B6-like [Danio rerio]
          Length = 125

 Score = 66.6 bits (161), Expect = 1e-11,   Method: Compositional matrix adjust.
 Identities = 41/112 (36%), Positives = 55/112 (49%), Gaps = 10/112 (8%)

Query: 93  LLLFWT-----SASRCDIVMTQSPATLSVTPGDRVSLSCRASQSISDY----LHWYQQKS 143
           ++  WT     S     I +TQSP   +V PG+ V +SC+ S S +      + WY QK 
Sbjct: 5   IIFIWTLSFCFSGCAAQITVTQSPVE-TVKPGENVQMSCKLSSSETGCYPPCVSWYLQKP 63

Query: 144 HESPRLLIKYASQXXXXXXXXXXXXXXXXDFTLSINSVEPEDVGVYYCQNGH 195
            E+P+LLI   S                 DFTLSI+ V+ ED G YYCQ+ H
Sbjct: 64  GEAPKLLIYRTSSRQSGTASRFSGSGSNVDFTLSISGVQTEDAGHYYCQSRH 115


&gt;ref|XP_001920187.2| PREDICTED: hypothetical protein LOC100150116 [Danio rerio]
          Length = 246

 Score = 64.7 bits (156), Expect = 4e-11,   Method: Compositional matrix adjust.
 Identities = 38/95 (40%), Positives = 52/95 (54%), Gaps = 5/95 (5%)

Query: 105 IVMTQSPATLSVTPGDRVSLSCRASQ--SISDY--LHWYQQKSHESPRLLIKYASQXXXX 160
           I +TQSP    V PG+ V + C+ +Q  + S Y  L+WY QK  E+P+LLI Y ++    
Sbjct: 143 ITVTQSPVE-PVKPGENVEMRCKLNQYSNCSPYPCLYWYLQKPGEAPKLLIYYVNRLQSG 201

Query: 161 XXXXXXXXXXXXDFTLSINSVEPEDVGVYYCQNGH 195
                       DFTLSI+ V+ ED G YYC + H
Sbjct: 202 TASRFSGSGSHVDFTLSISGVQTEDAGHYYCMSLH 236



 Score = 30.4 bits (67), Expect = 1.1,   Method: Compositional matrix adjust.
 Identities = 13/20 (65%), Positives = 15/20 (75%)

Query: 173 DFTLSINSVEPEDVGVYYCQ 192
           DFTLSI+ V+ ED G YY Q
Sbjct: 123 DFTLSISGVQTEDAGHYYSQ 142


&gt;ref|XP_003201399.1| PREDICTED: ig kappa chain V19-17-like [Danio rerio]
          Length = 124

 Score = 62.4 bits (150), Expect = 2e-10,   Method: Compositional matrix adjust.
 Identities = 39/96 (40%), Positives = 48/96 (50%), Gaps = 5/96 (5%)

Query: 105 IVMTQSPATLSVTPGDRVSLSCRASQ----SISDYLHWYQQKSHESPRLLIKYASQXXXX 160
           I +TQ P    V PG+ V +SC+ SQ    S    + WY QK  E P+LLI   S     
Sbjct: 24  ITVTQIPVE-PVKPGENVQMSCKLSQPAPCSPKPCVSWYLQKPGEDPKLLIYVTSYRYSG 82

Query: 161 XXXXXXXXXXXXDFTLSINSVEPEDVGVYYCQNGHS 196
                       DFTLSI+ V+ ED G YYCQ+ HS
Sbjct: 83  TASRFSGSGSNVDFTLSISGVQTEDAGHYYCQSYHS 118


&gt;ref|XP_003200636.1| PREDICTED: ig kappa chain V-I region WEA-like [Danio rerio]
          Length = 116

 Score = 53.9 bits (128), Expect = 8e-08,   Method: Compositional matrix adjust.
 Identities = 28/79 (35%), Positives = 38/79 (48%)

Query: 108 TQSPATLSVTPGDRVSLSCRASQSISDYLHWYQQKSHESPRLLIKYASQXXXXXXXXXXX 167
            QSP +LSV  GD  S+SC  +  +   + WY QK  ++P+LLI   S            
Sbjct: 22  VQSPESLSVKAGDSASISCTGTSGVGYDMSWYLQKPGKAPKLLIYGVSYLESGVSDRFSG 81

Query: 168 XXXXXDFTLSINSVEPEDV 186
                 FTL+I  V+PED 
Sbjct: 82  NGSEPAFTLNIRGVQPEDA 100


&gt;ref|NP_001020667.1| novel immune-type receptor 12 precursor [Danio rerio]
          Length = 318

 Score = 47.0 bits (110), Expect = 1e-05,   Method: Compositional matrix adjust.
 Identities = 28/109 (25%), Positives = 44/109 (40%), Gaps = 10/109 (9%)

Query: 90  LLGLLLFWTSASRCDIVMTQSPATLSVTPGDRVSLSCRASQSISDYLHWYQQKSHESPRL 149
           L+  LLF T  S     + Q     S T G  V+L C  S ++ +Y  W++Q   E+P  
Sbjct: 6   LVYFLLFRTGFSG---AVVQKKVLESFTAGQTVTLECLISVNLENYFSWFKQTLGEAPTC 62

Query: 150 LIKYASQXXX-------XXXXXXXXXXXXXDFTLSINSVEPEDVGVYYC 191
           ++   ++                        F L+I   +P D G+YYC
Sbjct: 63  IVSLYAESSTPVFYGEFKNNHRMSVQKEKNTFVLNIKEAKPSDAGIYYC 111


&gt;ref|XP_001923479.1| PREDICTED: hypothetical protein LOC100151290 [Danio rerio]
          Length = 318

 Score = 45.8 bits (107), Expect = 2e-05,   Method: Compositional matrix adjust.
 Identities = 29/109 (26%), Positives = 43/109 (39%), Gaps = 10/109 (9%)

Query: 90  LLGLLLFWTSASRCDIVMTQSPATLSVTPGDRVSLSCRASQSISDYLHWYQQKSHESPRL 149
           L+  LLF T  S   I   Q     S T G  V+L C  S ++ +Y  W++Q   E+P  
Sbjct: 6   LVYFLLFRTGFSGAVI---QKKVLESFTAGQTVTLECLISLNLENYFSWFKQTLGEAPTC 62

Query: 150 LIKYASQXXX-------XXXXXXXXXXXXXDFTLSINSVEPEDVGVYYC 191
           ++   ++                        F L I   +P D G+YYC
Sbjct: 63  IVSLYAESSTPVFYGEFKNNHRMSVQKGKNTFVLIIKEAKPSDAGIYYC 111


&gt;ref|XP_003200433.1| PREDICTED: t-cell receptor beta chain T17T-22-like [Danio rerio]
          Length = 275

 Score = 38.1 bits (87), Expect = 0.005,   Method: Compositional matrix adjust.
 Identities = 16/46 (34%), Positives = 26/46 (56%)

Query: 105 IVMTQSPATLSVTPGDRVSLSCRASQSISDYLHWYQQKSHESPRLL 150
           +++TQ P  +S  PG  V + C  + +  DY +WY+Q   E P L+
Sbjct: 9   VLITQWPKYISSLPGFSVDMHCYQNDTDYDYKYWYRQIKGEGPVLI 54


&gt;ref|XP_003198184.1| PREDICTED: ig heavy chain V-I region HG3-like [Danio rerio]
          Length = 131

 Score = 37.0 bits (84), Expect = 0.011,   Method: Compositional matrix adjust.
 Identities = 32/123 (26%), Positives = 52/123 (42%), Gaps = 16/123 (13%)

Query: 89  QLLGLLLFWTSASRCDIVMTQSPATLSVTPGDRVSLSCRAS--QSISDYLHWYQQKSHES 146
             L +++ +   S  +I + QSP +++  PG+ V +SC+ S     S Y+HW +QK  ++
Sbjct: 7   HFLCVVIVFQGISGDEIRLDQSP-SVTKRPGETVKISCKISGFTMTSSYMHWIRQKPGKA 65

Query: 147 PRLLIKYASQXXXXXXXXXXXXXXXXDFTLS-----------INSVEPEDVGVYYCQN-- 193
              + +  S                  FT+S             S+  ED  VYYC    
Sbjct: 66  LEWIGRVDSGSSSSSDYLIYADSVKNHFTMSEDVSQSTQFLEAKSLREEDTAVYYCARAP 125

Query: 194 GHS 196
           GHS
Sbjct: 126 GHS 128


&gt;ref|XP_003197959.1| PREDICTED: hypothetical protein LOC100535098 [Danio rerio]
          Length = 199

 Score = 36.6 bits (83), Expect = 0.015,   Method: Compositional matrix adjust.
 Identities = 36/121 (29%), Positives = 52/121 (42%), Gaps = 14/121 (11%)

Query: 101 SRCDI--VMTQSPATLSVTPGDRVSLSCR-ASQSISDYLHWYQQKSHESPR-LLIKYASQ 156
           SRCD    + Q   T S   GD V+  C   + S    L WYQQK    P+ +L K+++ 
Sbjct: 83  SRCDSQDKVDQHTKTQSAFEGDTVTTDCTYQTSSTGPTLFWYQQKVSGVPKHMLNKFSTT 142

Query: 157 XXXXXXXXXXXXXXXXDFT---LSINSVEPEDVGVYYCQNGHSFPPTMMQ---TIAKTSR 210
                             T   L+I ++  +D  VYYC    +  PT+ Q   T+ KT R
Sbjct: 143 GNSEEELKKKRFHATLSKTSVPLTIQNLRVQDSAVYYC----ALKPTVTQTHSTLTKTPR 198

Query: 211 K 211
           +
Sbjct: 199 Q 199


&gt;ref|XP_003197951.1| PREDICTED: ig kappa chain V-III region VH-like [Danio rerio]
          Length = 124

 Score = 36.2 bits (82), Expect = 0.020,   Method: Compositional matrix adjust.
 Identities = 35/109 (32%), Positives = 44/109 (40%), Gaps = 13/109 (11%)

Query: 93  LLLFWTSASRCDIVMTQSPATLSVTPGDR--VSLSCRASQSI-SDYLHWYQQKSHESPRL 149
           L+LF TSA+     + + P    V   +R  V+LSC  + S  SDYLHWY+Q     P  
Sbjct: 6   LILFCTSAAAVFGNVIK-PINTDVFTEERSNVTLSCSFTASGGSDYLHWYRQYGGSKPEF 64

Query: 150 LI-------KYASQXXXXXXXXXXXXXXXXDFTLSINSVEPEDVGVYYC 191
           L+       K A                  D  LS  S    D  VYYC
Sbjct: 65  LVLIYGSATKAAQSDIDTRFTANITKKEHVDLLLS--SAAVSDSAVYYC 111


&gt;ref|XP_003197960.1| PREDICTED: hypothetical protein LOC100535742 [Danio rerio]
          Length = 113

 Score = 36.2 bits (82), Expect = 0.020,   Method: Compositional matrix adjust.
 Identities = 23/65 (35%), Positives = 33/65 (50%), Gaps = 2/65 (3%)

Query: 89  QLLGLLLFWTSASRC-DIVMTQSPATLSVTPGDRVSLSCRASQSI-SDYLHWYQQKSHES 146
           +L  L+LF TSA+     V+      +    G  V+LSC  + S  +DYLHWY+Q     
Sbjct: 2   KLHSLILFCTSAAAVFGNVIRPINTDVFAEEGSNVTLSCSFTASGGTDYLHWYRQYGGSK 61

Query: 147 PRLLI 151
           P  L+
Sbjct: 62  PEFLV 66


&gt;ref|XP_003197945.1| PREDICTED: ig lambda chain V-II region BO-like [Danio rerio]
          Length = 111

 Score = 36.2 bits (82), Expect = 0.020,   Method: Compositional matrix adjust.
 Identities = 22/76 (28%), Positives = 37/76 (48%), Gaps = 3/76 (3%)

Query: 119 GDRVSLSCRASQSISD-YLHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXXXXXDFT-- 175
           G+ V+LSC    + +D +L+WY+Q S+  P  ++   ++                D T  
Sbjct: 30  GESVTLSCTYETNSNDIWLYWYRQFSNREPEYILFKGARSRSEEDLPDGFQSRTSDSTTE 89

Query: 176 LSINSVEPEDVGVYYC 191
           ++INSV   D  +YYC
Sbjct: 90  ITINSVRLSDSALYYC 105


&gt;ref|NP_001005768.1| novel immune-type receptor 5 precursor [Danio rerio]
          Length = 338

 Score = 35.4 bits (80), Expect = 0.028,   Method: Compositional matrix adjust.
 Identities = 27/96 (28%), Positives = 37/96 (38%), Gaps = 6/96 (6%)

Query: 107 MTQSPATLSVTPGDRVSLSCRASQSISDYLHWYQQKSHESPRLL---IKYASQXXXXXXX 163
           +TQ    +S   G+ V L C  + S      WY+Q + E PRL+   I  +S+       
Sbjct: 23  ITQPDTVVSFQEGETVVLRCFITNSQMSMTLWYKQVTGEEPRLIASSILRSSEIQFHNEF 82

Query: 164 XXXXXXXXXD---FTLSINSVEPEDVGVYYCQNGHS 196
                    D   F L I +    D G YYC    S
Sbjct: 83  DSSHFEVLRDTGGFNLKIVNAVQSDSGTYYCATSFS 118


&gt;ref|XP_003197957.1| PREDICTED: immunoglobulin omega chain-like [Danio rerio]
          Length = 157

 Score = 35.4 bits (80), Expect = 0.028,   Method: Compositional matrix adjust.
 Identities = 26/107 (24%), Positives = 46/107 (42%), Gaps = 12/107 (11%)

Query: 97  WTSASRCDIV----------MTQSPATLSVTPGDRVSLSCRASQSISDY-LHWYQQKSHE 145
           W   S C IV          + Q    ++ + GD+V +SC  S + +   L WY++  + 
Sbjct: 4   WAFISLCFIVHYDVCRGQDRVEQPIREMTESIGDKVVISCIYSTTYTSLTLFWYKKLPNR 63

Query: 146 SPRLLI-KYASQXXXXXXXXXXXXXXXXDFTLSINSVEPEDVGVYYC 191
           SP  ++ +Y+++                 F L+I ++   D  VYYC
Sbjct: 64  SPTYILNQYSTEADFKERFSATLDSTEKSFPLTIKNLHVSDSAVYYC 110


&gt;ref|XP_002666277.1| PREDICTED: hypothetical protein LOC100331885 [Danio rerio]
          Length = 213

 Score = 35.4 bits (80), Expect = 0.029,   Method: Compositional matrix adjust.
 Identities = 29/107 (27%), Positives = 47/107 (43%), Gaps = 14/107 (13%)

Query: 89  QLLGLLLFWTSASRCDIVMTQSPATLSVTPGDRVSLSCRASQSISDYLHWYQQKSHESPR 148
           QL   LL WT          +S     V  G+ V+L C+        ++W  QK  +SP 
Sbjct: 5   QLSLYLLLWTQT-------VESLIDKRVKLGENVTLDCQIG---VKEMYWVFQKLIDSPV 54

Query: 149 LLIK-YASQXXXXXXXXXXXXXXXXDFTLS---INSVEPEDVGVYYC 191
           L+++ Y+S+                  TLS   I+S+  +++G+YYC
Sbjct: 55  LILRTYSSESTASFIPEDRLRDKYSSLTLSRLFISSITIDELGIYYC 101


&gt;ref|NP_001018511.1| ganglioside-induced differentiation-associated protein 1 [Danio
           rerio]
          Length = 362

 Score = 34.7 bits (78), Expect = 0.053,   Method: Compositional matrix adjust.
 Identities = 17/40 (42%), Positives = 22/40 (55%), Gaps = 1/40 (2%)

Query: 3   SLPASELSLPYHQDFRPDSSCPGLT-DNHLLCEPTQKPRY 41
           SLP SE + P+     P    P L  DNH++C+PTQ   Y
Sbjct: 71  SLPLSEHNEPWFMRLNPTGEVPVLVHDNHVICDPTQIMDY 110


&gt;ref|NP_954581.3| novel immune-type receptor 6a precursor [Danio rerio]
          Length = 249

 Score = 34.7 bits (78), Expect = 0.057,   Method: Compositional matrix adjust.
 Identities = 22/80 (27%), Positives = 36/80 (45%), Gaps = 7/80 (8%)

Query: 119 GDRVSLSCRASQ-SISDYLHWYQQKSHESPRLL-IKYASQXXXXXXXXXXXXXXXX---- 172
           GD V + C +++ S+ + L WY+QK+ + PR + I Y                       
Sbjct: 31  GDDVIIKCFSTKISLGNTLVWYKQKTGQIPRAITISYIQLNKVIFEDEFKDGRFSILPSE 90

Query: 173 -DFTLSINSVEPEDVGVYYC 191
             F L+I +   +D G+YYC
Sbjct: 91  DSFHLNITAATKQDTGIYYC 110


&gt;ref|XP_003198968.1| PREDICTED: hypothetical protein LOC100538107 [Danio rerio]
          Length = 239

 Score = 33.9 bits (76), Expect = 0.082,   Method: Compositional matrix adjust.
 Identities = 21/80 (26%), Positives = 36/80 (45%), Gaps = 7/80 (8%)

Query: 119 GDRVSLSCRAS-QSISDYLHWYQQKSHESPRLL-IKYASQXXXXXXXXX-----XXXXXX 171
           GD V + C ++ +S+ + L WY+Q + + PR + I Y  +                    
Sbjct: 31  GDDVIIKCFSTRKSVGNTLVWYKQSTGQIPRAITISYNHKGDVIFEDEFKDGRFSILSSE 90

Query: 172 XDFTLSINSVEPEDVGVYYC 191
             F L+I +   +D G+YYC
Sbjct: 91  DSFHLNITAATKQDTGIYYC 110


&gt;ref|XP_003197944.1| PREDICTED: ig kappa chain V-VI region NQ2-6.1-like [Danio rerio]
          Length = 140

 Score = 33.9 bits (76), Expect = 0.086,   Method: Compositional matrix adjust.
 Identities = 19/59 (32%), Positives = 28/59 (47%), Gaps = 1/59 (1%)

Query: 93  LLLFWTSASRCDIVMTQSPATLSVTPGDRVSLSCRASQSISDYLHWYQQKSHESPRLLI 151
           +LL     +R    +T     +  +  +RV LSC  + S+ D LHWY+Q     P  LI
Sbjct: 7   ILLILIEGNRAKDAITPYRDAVFASESERVKLSCNYTGSV-DSLHWYRQYPGSRPHFLI 64


&gt;ref|NP_001121856.1| novel immune-type receptor 13 isoform 1 precursor [Danio rerio]
          Length = 350

 Score = 33.5 bits (75), Expect = 0.11,   Method: Compositional matrix adjust.
 Identities = 25/95 (26%), Positives = 41/95 (43%), Gaps = 10/95 (10%)

Query: 102 RCDIVMTQSPATLSVTPGDRVSLSC-----RASQSISDYLHWYQQKSHESPRLLIKYASQ 156
           +  + + Q P++ SV  G  ++L C     R+++ I   + W++  S + P   I Y + 
Sbjct: 138 KSKVSVVQHPSSHSVRSGVALTLICSVLSERSTEDIK--MFWFRSDSGDKPVPEILYTNN 195

Query: 157 XXXXXXXXXXXXXXXXDFTLSINSVEPEDVGVYYC 191
                            F LSIN+V   D G YYC
Sbjct: 196 QSLQCESDFTHSCT---FNLSINTVSQMDTGTYYC 227



 Score = 28.1 bits (61), Expect = 5.5,   Method: Compositional matrix adjust.
 Identities = 16/55 (29%), Positives = 27/55 (49%), Gaps = 1/55 (1%)

Query: 93  LLLFWTSASRCDIVMTQSPATLSVTPGDRVSLSCRASQSISDYLHWYQQKSHESP 147
           +LL     +  +IV    P  ++ + G+ V+L C   Q  SD + WY+Q +   P
Sbjct: 9   MLLCRIKRTESEIVQQPDPVMMA-SVGENVTLRCIILQEHSDPITWYKQTAGHQP 62


&gt;ref|NP_001232002.1| similar to novel immune-type receptor 5, allele 3 precursor [Danio
           rerio]
          Length = 338

 Score = 33.5 bits (75), Expect = 0.12,   Method: Compositional matrix adjust.
 Identities = 26/96 (27%), Positives = 36/96 (37%), Gaps = 6/96 (6%)

Query: 107 MTQSPATLSVTPGDRVSLSCRASQSISDYLHWYQQKSHESPRLL---IKYASQXXXXXXX 163
           + Q    +S   G+ V L C  + S      WY+Q + E PRL+   I  +S+       
Sbjct: 23  IIQPDTVVSFQEGETVVLRCFITNSQMSMTLWYKQVTGEEPRLIASSILRSSEIQFHNEF 82

Query: 164 XXXXXXXXXD---FTLSINSVEPEDVGVYYCQNGHS 196
                    D   F L I +    D G YYC    S
Sbjct: 83  DSSHFEVLRDTGGFNLKIVNAVQSDSGTYYCTTSFS 118


&gt;ref|XP_003201256.1| PREDICTED: basement membrane-specific heparan sulfate proteoglycan
            core protein [Danio rerio]
          Length = 3711

 Score = 33.1 bits (74), Expect = 0.13,   Method: Compositional matrix adjust.
 Identities = 22/87 (25%), Positives = 35/87 (40%), Gaps = 20/87 (22%)

Query: 106  VMTQSPATLSVTPGDRVSLSCRASQSISDY-LHWYQQKSHESPRLLIKYASQXXXXXXXX 164
            V  + P   SV  G  VS  C A   +  Y L W ++ + + P   + +           
Sbjct: 1957 VFIEEPKAQSVRVGSTVSFICTAKSKLPAYTLVWTRKGNGKLPNRAMDF----------- 2005

Query: 165  XXXXXXXXDFTLSINSVEPEDVGVYYC 191
                    +  L+I +V+PED G+Y C
Sbjct: 2006 --------NGILTIQNVQPEDAGIYVC 2024


&gt;ref|NP_001020668.1| novel immune-type receptor 11b precursor [Danio rerio]
          Length = 224

 Score = 33.1 bits (74), Expect = 0.14,   Method: Compositional matrix adjust.
 Identities = 24/118 (20%), Positives = 48/118 (40%), Gaps = 19/118 (16%)

Query: 84  MVSTSQLLGLLLFWTSASRCDIVMTQSPATLSVTPGDRVSLSCRASQSISDYLHWYQQKS 143
           ++ ++ L+GLL          +V +     +S  PG+ V++ C+ + +    +HW++Q  
Sbjct: 4   LLISALLMGLL---------SLVKSSQIVNISAQPGENVTIWCKHTSNTGKNIHWFKQTH 54

Query: 144 HESP-RLLIKYASQXXXXXXXXXXXXXXXXDFTLSINS---------VEPEDVGVYYC 191
              P  ++ K  S                  F +S+N+         VE  D  +Y+C
Sbjct: 55  GAVPLNIVYKMISYKFEELHLKYLNDFQPDRFVMSLNTENTSLSILNVEISDSALYFC 112


&gt;ref|XP_001331043.4| PREDICTED: hypothetical protein LOC791658 [Danio rerio]
          Length = 323

 Score = 33.1 bits (74), Expect = 0.16,   Method: Compositional matrix adjust.
 Identities = 32/120 (26%), Positives = 51/120 (42%), Gaps = 18/120 (15%)

Query: 84  MVSTSQLLGL-----LLFWTSASRCDIVMTQSPATLSVTPGDRVSL-------SCRASQS 131
           +VS+ Q  G+     LL   +A+  D  + QS    +V PGD V+L       SC    S
Sbjct: 113 VVSSYQATGMGSGTRLLVRDTAADRDTTLHQSLID-TVDPGDSVNLQCSIFTESCEGDHS 171

Query: 132 ISDYLHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXXXXXDFTLSINSVEPEDVGVYYC 191
           I    +W++Q S +S  +L     +                 ++L  N++   D G+YYC
Sbjct: 172 I----YWFKQSSGDSEGVLYTKGERNGRCKNSTESQTQSCV-YSLHKNNISRSDSGIYYC 226


&gt;ref|XP_003198238.1| PREDICTED: ig lambda chain V-III region LOI-like [Danio rerio]
          Length = 124

 Score = 33.1 bits (74), Expect = 0.16,   Method: Compositional matrix adjust.
 Identities = 25/98 (25%), Positives = 41/98 (41%), Gaps = 8/98 (8%)

Query: 106 VMTQSPATLSVTPGDRVSLSCRASQSISDYLHWYQQKSHESPR-LLIKYASQXX------ 158
           V+TQSP  ++V+ G    L C         + WY+Q     P+ +L  Y ++        
Sbjct: 20  VVTQSPV-ITVSKGQTAQLDCNLGTVTDSGVRWYKQTPAGVPQYVLYDYHTRSSPIYGSG 78

Query: 159 XXXXXXXXXXXXXXDFTLSINSVEPEDVGVYYCQNGHS 196
                         D++L I++V+  D  VYYC+   S
Sbjct: 79  FAAPKFTSTHSSTSDYSLIISNVDVSDSAVYYCKTWDS 116


&gt;ref|XP_003198962.1| PREDICTED: hypothetical protein LOC100537152 [Danio rerio]
          Length = 326

 Score = 32.7 bits (73), Expect = 0.19,   Method: Compositional matrix adjust.
 Identities = 28/117 (23%), Positives = 50/117 (42%), Gaps = 14/117 (11%)

Query: 84  MVSTSQLLGLLLFWTSASRCDIVMTQSPATL------SVTPGDRVSLSCR--ASQSISDY 135
           ++S  Q +G+     S +R  +  T   +TL      +V PGD V+L C         D+
Sbjct: 112 VISAYQTIGM----GSGTRLLVGATDRNSTLHQSLIDAVDPGDAVNLQCSIFTESCAGDH 167

Query: 136 -LHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXXXXXDFTLSINSVEPEDVGVYYC 191
            ++W++Q S +S  +L     +                 ++L  N++   D G+YYC
Sbjct: 168 SIYWFKQSSGDSEGVLYTKGERNGRCKNSTESQTQSCV-YSLHKNNISRSDSGIYYC 223


&gt;ref|XP_002666721.1| PREDICTED: hypothetical protein LOC100332522 [Danio rerio]
          Length = 196

 Score = 32.7 bits (73), Expect = 0.20,   Method: Compositional matrix adjust.
 Identities = 23/78 (29%), Positives = 31/78 (39%), Gaps = 3/78 (3%)

Query: 119 GDRVSLSCRASQSISDYLHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXXXXXDFTLSI 178
           GDR+ LSC  S S       +++ S E  R++  Y                   DF+LSI
Sbjct: 25  GDRIELSCLQSSSKHKGNALWKKSSGE--RVIWNYYGDSNKGGSFTNRTVKLNDDFSLSI 82

Query: 179 NSVEPEDVGVYY-CQNGH 195
                 D G+Y  C NG 
Sbjct: 83  EGCAQSDQGIYILCINGE 100


&gt;ref|XP_003198969.1| PREDICTED: hypothetical protein LOC100538145 [Danio rerio]
          Length = 240

 Score = 32.7 bits (73), Expect = 0.20,   Method: Compositional matrix adjust.
 Identities = 20/80 (25%), Positives = 34/80 (42%), Gaps = 7/80 (8%)

Query: 119 GDRVSLSCRASQ-SISDYLHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXX------XX 171
           GD V + C  ++ S+ + L WY+Q + + PR++    +Q                     
Sbjct: 31  GDDVIIKCFLTKMSLGNTLVWYKQSTGQIPRVITISYNQLNKVRFVDEFKDGRFSVLPSE 90

Query: 172 XDFTLSINSVEPEDVGVYYC 191
             F L I +   +D G+YYC
Sbjct: 91  DSFHLKITAATKQDTGIYYC 110


&gt;ref|NP_001020672.1| novel immune-type receptor 11a precursor [Danio rerio]
          Length = 195

 Score = 32.3 bits (72), Expect = 0.26,   Method: Compositional matrix adjust.
 Identities = 14/66 (21%), Positives = 32/66 (48%), Gaps = 9/66 (13%)

Query: 82  LKMVSTSQLLGLLLFWTSASRCDIVMTQSPATLSVTPGDRVSLSCRASQSISDYLHWYQQ 141
           +++V ++ L+GLL           V +     ++  P + V++ C+ +  +  Y+HW++Q
Sbjct: 2   VRVVISALLMGLL---------SSVKSSQTGNITAQPVENVTIWCKHNAGVGQYIHWFKQ 52

Query: 142 KSHESP 147
                P
Sbjct: 53  THGAVP 58


&gt;ref|NP_938160.1| immune-type receptor 1o [Danio rerio]
          Length = 358

 Score = 32.3 bits (72), Expect = 0.26,   Method: Compositional matrix adjust.
 Identities = 28/116 (24%), Positives = 52/116 (44%), Gaps = 10/116 (8%)

Query: 84  MVSTSQLLGL-----LLFWTSASRCDIVMTQSPATLSVTPGDRVSLSCR--ASQSISDY- 135
           +VS+ Q +G+     L+   +A+  +  + QS    +V PGD V+L C         D+ 
Sbjct: 146 IVSSYQAIGMGSGTRLIVRDAATDRNTTLHQSLID-TVDPGDSVNLQCSIFTESCAGDHN 204

Query: 136 LHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXXXXXDFTLSINSVEPEDVGVYYC 191
           ++W++Q S +S  +L     +                 ++L  N++   D G+YYC
Sbjct: 205 IYWFKQSSGDSEGVLYTKGERNGRCKNSAESETQSCV-YSLHKNNISRSDTGIYYC 259


&gt;ref|XP_003198963.1| PREDICTED: hypothetical protein LOC100537250 [Danio rerio]
          Length = 239

 Score = 32.3 bits (72), Expect = 0.27,   Method: Compositional matrix adjust.
 Identities = 20/80 (25%), Positives = 35/80 (43%), Gaps = 7/80 (8%)

Query: 119 GDRVSLSCRAS-QSISDYLHWYQQKSHESPRLL-IKYASQXXXXXXXXXXXXXXXX---- 172
           GD V + C ++ + + + L WY+Q + + PR + I Y  +                    
Sbjct: 31  GDDVIIKCFSTRKGVGNTLVWYKQSTGQIPRAITISYNHKSEVIFDDEFKDGRFSILPSE 90

Query: 173 -DFTLSINSVEPEDVGVYYC 191
             F L+I +   +D G+YYC
Sbjct: 91  DSFHLNITAATKQDTGIYYC 110


&gt;ref|XP_003197958.1| PREDICTED: ig lambda chain V-VI region AR-like [Danio rerio]
          Length = 159

 Score = 32.0 bits (71), Expect = 0.31,   Method: Compositional matrix adjust.
 Identities = 25/88 (28%), Positives = 36/88 (40%), Gaps = 11/88 (12%)

Query: 119 GDRVSLSCRASQSISDYLHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXXXXXD----- 173
           G+  SLSC+   S+++ LHWY+Q     P  L  Y  +                D     
Sbjct: 34  GETASLSCKYEGSVTN-LHWYRQYPGSRPEFL-AYIYEHGATSKPLPPRLLPKVDKNSKR 91

Query: 174 FTLSINSVEPEDVGVYYCQNGHSFPPTM 201
            +L I+  E  D  +YYC    +  PTM
Sbjct: 92  VSLEISEAEVTDSALYYC----ALTPTM 115


&gt;ref|NP_001018173.1| novel immune-type receptor 1b precursor [Danio rerio]
          Length = 324

 Score = 32.0 bits (71), Expect = 0.36,   Method: Compositional matrix adjust.
 Identities = 29/116 (25%), Positives = 50/116 (43%), Gaps = 10/116 (8%)

Query: 84  MVSTSQLLGL-----LLFWTSASRCDIVMTQSPATLSVTPGDRVSLSCR--ASQSISDY- 135
           ++S  Q +G+     LL   SA+  +  + QS    +V PGD V+L C         D+ 
Sbjct: 112 VISAYQTIGMGSGTRLLVGDSATDRNSTLHQSLID-TVDPGDSVNLQCSIFTESCAGDHS 170

Query: 136 LHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXXXXXDFTLSINSVEPEDVGVYYC 191
           ++W++Q S   P  +I    +                 ++L  N++   D G+YYC
Sbjct: 171 VYWFKQSSGH-PEGVIYTKGERNGRCKNSTESQTQSCVYSLHKNNISHSDTGIYYC 225


&gt;ref|NP_001135740.1| immune-type receptor 5-like precursor [Danio rerio]
          Length = 337

 Score = 31.6 bits (70), Expect = 0.40,   Method: Compositional matrix adjust.
 Identities = 25/96 (26%), Positives = 35/96 (36%), Gaps = 6/96 (6%)

Query: 107 MTQSPATLSVTPGDRVSLSCRASQSISDYLHWYQQKSHESPRLL---IKYASQXXXXXXX 163
           + Q    +S   G+   L C  + S      WY+Q + E PRL+   I  +S+       
Sbjct: 23  IIQPDTVVSFQEGETAVLRCFITNSQMSMTLWYKQVTGEEPRLIASSILRSSEIQFHNEF 82

Query: 164 XXXXXXXXXD---FTLSINSVEPEDVGVYYCQNGHS 196
                    D   F L I +    D G YYC    S
Sbjct: 83  DSSHFEVLRDTGGFNLKIVNAVQSDSGTYYCATSFS 118


&gt;ref|XP_003198961.1| PREDICTED: hypothetical protein LOC100537054 [Danio rerio]
          Length = 240

 Score = 31.6 bits (70), Expect = 0.40,   Method: Compositional matrix adjust.
 Identities = 20/80 (25%), Positives = 33/80 (41%), Gaps = 7/80 (8%)

Query: 119 GDRVSLSCRASQ-SISDYLHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXX------XX 171
           GD V + C  ++ S+ + L WY+Q + + PR +    +Q                     
Sbjct: 31  GDDVIIKCFLTKMSLGNTLVWYKQSTGQIPRAITISYNQLNKVRFVDEFKDGRFSVLPSE 90

Query: 172 XDFTLSINSVEPEDVGVYYC 191
             F L I +   +D G+YYC
Sbjct: 91  DSFHLKITAATKQDTGIYYC 110


&gt;ref|XP_001333297.2| PREDICTED: hypothetical protein LOC794617 isoform 2 [Danio rerio]
          Length = 199

 Score = 31.6 bits (70), Expect = 0.41,   Method: Compositional matrix adjust.
 Identities = 23/121 (19%), Positives = 47/121 (38%), Gaps = 19/121 (15%)

Query: 84  MVSTSQLLGLLLFWTSASRCDIVMTQSPATLSVTPGDRVSLSCRASQSISDYLHWYQQKS 143
           ++ ++ L+GLL          +V +     +S  PG   ++ C+ + +    +HW++Q  
Sbjct: 4   LLISAMLMGLL---------SLVKSSQIVNISAQPGKNFTIWCKHTSNTGKTIHWFKQTH 54

Query: 144 HESP-RLLIKYASQXXXXXXXXXXXXXXXXDFTLSIN---------SVEPEDVGVYYCQN 193
              P  ++ K  S                  F +S+N         +V+  D  +Y+C  
Sbjct: 55  GAVPVDIVYKMISYQLKEIYVNYLNGFQQDHFVMSLNKENTSLSILNVDISDSALYFCGW 114

Query: 194 G 194
           G
Sbjct: 115 G 115


&gt;ref|XP_003197933.1| PREDICTED: hypothetical protein LOC100534730 [Danio rerio]
          Length = 123

 Score = 31.6 bits (70), Expect = 0.42,   Method: Compositional matrix adjust.
 Identities = 27/105 (25%), Positives = 45/105 (42%), Gaps = 6/105 (5%)

Query: 90  LLGLLLFWTSASRCDIVMTQSPATLSVTPGDRVSLSCRASQSISDYLHWYQQKSHESPRL 149
           LL   LF  +A + +++ + S   + V  G   +LSC+ + +I + L WY+Q     P  
Sbjct: 6   LLLFTLFKGNAGK-EVISSLSTEQI-VLAGKNATLSCKYNSNIRN-LQWYRQYPGSKPEN 62

Query: 150 LIKYA---SQXXXXXXXXXXXXXXXXDFTLSINSVEPEDVGVYYC 191
           +I +    +Q                   LSI   E +D  +YYC
Sbjct: 63  IIFHTESNNQSYSRLRLLAVAEKEIKRMNLSIFHTEMQDSALYYC 107


&gt;ref|NP_001032660.1| novel immune-type receptor 1n precursor [Danio rerio]
          Length = 325

 Score = 31.6 bits (70), Expect = 0.48,   Method: Compositional matrix adjust.
 Identities = 28/116 (24%), Positives = 51/116 (43%), Gaps = 10/116 (8%)

Query: 84  MVSTSQLLGL-----LLFWTSASRCDIVMTQSPATLSVTPGDRVSLSCR--ASQSISDY- 135
           +VS+ Q +G+     L+   +A+  +  + QS    SV PGD V+L C         D+ 
Sbjct: 113 VVSSYQAIGMGSGTRLIVRDAATDRNTTLHQSSID-SVDPGDSVNLQCSIFTESCAGDHS 171

Query: 136 LHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXXXXXDFTLSINSVEPEDVGVYYC 191
           ++W++Q S +S  +L     +                 ++   N++   D G+YYC
Sbjct: 172 VYWFKQSSGDSEGVLYTKGERNGRCKNSTESQTQSCV-YSFHKNNISRSDSGIYYC 226


&gt;ref|XP_001921797.2| PREDICTED: LOW QUALITY PROTEIN: hypothetical protein LOC100150420
           [Danio rerio]
          Length = 366

 Score = 31.6 bits (70), Expect = 0.48,   Method: Compositional matrix adjust.
 Identities = 20/80 (25%), Positives = 35/80 (43%), Gaps = 4/80 (5%)

Query: 115 SVTPGDRVSLSCR--ASQSISDY-LHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXXXX 171
           +V PGD V+L C         D+ ++W++Q S +S  +L     +               
Sbjct: 188 TVDPGDSVNLQCSIFTESCAGDHSIYWFKQSSGDSEGVLYTKGERNGRCKNSAESETQSC 247

Query: 172 XDFTLSINSVEPEDVGVYYC 191
             ++L  N++   D G+YYC
Sbjct: 248 V-YSLHKNNISRSDTGIYYC 266


&gt;ref|XP_001333206.2| PREDICTED: hypothetical protein LOC794617 isoform 1 [Danio rerio]
          Length = 224

 Score = 31.2 bits (69), Expect = 0.50,   Method: Compositional matrix adjust.
 Identities = 23/121 (19%), Positives = 47/121 (38%), Gaps = 19/121 (15%)

Query: 84  MVSTSQLLGLLLFWTSASRCDIVMTQSPATLSVTPGDRVSLSCRASQSISDYLHWYQQKS 143
           ++ ++ L+GLL          +V +     +S  PG   ++ C+ + +    +HW++Q  
Sbjct: 4   LLISAMLMGLL---------SLVKSSQIVNISAQPGKNFTIWCKHTSNTGKTIHWFKQTH 54

Query: 144 HESP-RLLIKYASQXXXXXXXXXXXXXXXXDFTLSIN---------SVEPEDVGVYYCQN 193
              P  ++ K  S                  F +S+N         +V+  D  +Y+C  
Sbjct: 55  GAVPVDIVYKMISYQLKEIYVNYLNGFQQDHFVMSLNKENTSLSILNVDISDSALYFCGW 114

Query: 194 G 194
           G
Sbjct: 115 G 115


&gt;ref|XP_003197949.1| PREDICTED: immunoglobulin iota chain-like [Danio rerio]
          Length = 131

 Score = 31.2 bits (69), Expect = 0.53,   Method: Compositional matrix adjust.
 Identities = 23/87 (26%), Positives = 36/87 (41%), Gaps = 9/87 (10%)

Query: 119 GDRVSLSCRASQSISDYLHWYQQKSHESPRLLIKY----ASQXXXXXXXXXXXXXXXXDF 174
           G+ V+LSC+  + + + LHWY+Q     P+ L       A+                   
Sbjct: 34  GETVTLSCKY-EGVVNNLHWYRQYPGFRPKFLAYIYEHGATSKPLPPRLLPKVDKNSKRV 92

Query: 175 TLSINSVEPEDVGVYYCQNGHSFPPTM 201
           +L I+  E  D  +YYC    +  PTM
Sbjct: 93  SLEISEAEVTDSALYYC----ALTPTM 115


&gt;ref|NP_938161.1| novel immune-type receptor 1l precursor [Danio rerio]
          Length = 316

 Score = 31.2 bits (69), Expect = 0.62,   Method: Compositional matrix adjust.
 Identities = 20/80 (25%), Positives = 35/80 (43%), Gaps = 4/80 (5%)

Query: 115 SVTPGDRVSLSCR--ASQSISDY-LHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXXXX 171
           +V PGD V+L C         D+ ++W++Q S +S  +L     +               
Sbjct: 143 TVDPGDSVNLQCSIFTESCAGDHSIYWFKQSSGDSEGVLYTKGERNGRCKNSTESQTQSC 202

Query: 172 XDFTLSINSVEPEDVGVYYC 191
             ++L  N++   D G+YYC
Sbjct: 203 V-YSLHKNNISRSDSGIYYC 221


&gt;ref|NP_001186301.1| uncharacterized protein LOC100331689 precursor [Danio rerio]
          Length = 203

 Score = 31.2 bits (69), Expect = 0.62,   Method: Compositional matrix adjust.
 Identities = 20/73 (27%), Positives = 29/73 (39%), Gaps = 1/73 (1%)

Query: 119 GDRVSLSCRASQSISDYLHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXXXXXDFTLSI 178
           G  V + C  + S +  L WY+Q  + SP L++   S                  F L I
Sbjct: 36  GAHVFVQCNYTSS-NPTLFWYKQLPNRSPTLILSQYSAEADKKRFSATLNSKEKSFPLLI 94

Query: 179 NSVEPEDVGVYYC 191
             ++  D  VYYC
Sbjct: 95  QDLQVSDSAVYYC 107


&gt;ref|NP_938155.1| novel immune-type receptor 1m precursor [Danio rerio]
          Length = 356

 Score = 31.2 bits (69), Expect = 0.63,   Method: Compositional matrix adjust.
 Identities = 19/80 (23%), Positives = 36/80 (45%), Gaps = 4/80 (5%)

Query: 115 SVTPGDRVSLSCR--ASQSISDY-LHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXXXX 171
           ++ PGD V+L C         D+ ++W++Q+S +S  +L     +               
Sbjct: 159 TIDPGDSVNLQCSIFTESCAGDHSVYWFKQRSGDSEGVLYTKGERNGRCKNSTESQTQSC 218

Query: 172 XDFTLSINSVEPEDVGVYYC 191
             ++L  N++   D G+YYC
Sbjct: 219 V-YSLHKNNISRSDSGIYYC 237


&gt;ref|NP_938162.1| novel immune-type receptor 1k precursor [Danio rerio]
          Length = 328

 Score = 31.2 bits (69), Expect = 0.64,   Method: Compositional matrix adjust.
 Identities = 28/116 (24%), Positives = 50/116 (43%), Gaps = 10/116 (8%)

Query: 84  MVSTSQLLGL-----LLFWTSASRCDIVMTQSPATLSVTPGDRVSLSCR--ASQSISDY- 135
           ++S  Q +G+     LL   SA+  +  + QS    +V PGD V+L C         D+ 
Sbjct: 112 VISAYQTIGMGSGTRLLVGDSAADRNTTLHQSLID-TVDPGDSVNLQCSIFTESCAGDHS 170

Query: 136 LHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXXXXXDFTLSINSVEPEDVGVYYC 191
           ++W++Q S   P  ++    +                 ++L  N++   D G+YYC
Sbjct: 171 VYWFKQSSGH-PEGVLYTKGERNGRCKNSTESQTQSCVYSLHKNNISRSDSGIYYC 225


&gt;ref|XP_003197946.1| PREDICTED: immunoglobulin omega chain-like [Danio rerio]
          Length = 136

 Score = 31.2 bits (69), Expect = 0.64,   Method: Compositional matrix adjust.
 Identities = 25/99 (25%), Positives = 39/99 (39%), Gaps = 12/99 (12%)

Query: 114 LSVTPGDRVSLSC--RASQSISDYLHWYQQKSHESPRLLI------KYASQXXXXXXXXX 165
           L+   G+ V L C  +   + + YL+WY+Q    SP  ++      K  ++         
Sbjct: 31  LTSAEGNEVILLCNYKLMSAANAYLYWYKQLPDTSPAFILSEFSIGKGKAEPEFKDRFSA 90

Query: 166 XXXXXXXDFTLSINSVEPEDVGVYYCQNGHSFPPTMMQT 204
                     L+I  V   D  VYYC    +  PT+ QT
Sbjct: 91  KLDSKLETVPLTIKDVRVSDSAVYYC----ALQPTVTQT 125


&gt;ref|XP_002662802.1| PREDICTED: hypothetical protein LOC100330879 isoform 2 [Danio
           rerio]
          Length = 316

 Score = 30.8 bits (68), Expect = 0.66,   Method: Compositional matrix adjust.
 Identities = 22/84 (26%), Positives = 35/84 (41%), Gaps = 12/84 (14%)

Query: 115 SVTPGDRVSL-------SCRASQSISDYLHWYQQKSHESPRLLIKYASQXXXXXXXXXXX 167
           +V PGD V+L       SC    SI    +W++Q S +S  +L     +           
Sbjct: 143 TVDPGDSVNLQCSIFTESCEGDHSI----YWFKQSSGDSEGVLYTKGERNGRCKDSTESQ 198

Query: 168 XXXXXDFTLSINSVEPEDVGVYYC 191
                 ++L  N++   D G+YYC
Sbjct: 199 TQSCV-YSLHKNNISRSDSGIYYC 221


&gt;ref|XP_002661242.2| PREDICTED: ig heavy chain V-III region HIL [Danio rerio]
          Length = 136

 Score = 30.8 bits (68), Expect = 0.75,   Method: Compositional matrix adjust.
 Identities = 22/85 (25%), Positives = 38/85 (44%), Gaps = 9/85 (10%)

Query: 116 VTPGDRVSLSCRASQSI--SDYLHWYQQKSHESPRLL--IKYASQXXXXXXXXXXXXXXX 171
           + PG+ +SLSCR S     S+ +HW +Q+  ++   +  + Y +Q               
Sbjct: 40  LKPGETLSLSCRGSGFTFGSNNMHWIRQQPGKALMWMGRVWYNAQGHDYANSFKGRIEIT 99

Query: 172 XD-----FTLSINSVEPEDVGVYYC 191
            D       L ++++  ED  VYYC
Sbjct: 100 RDNSKSMVYLKLSALTEEDSAVYYC 124


&gt;ref|XP_002667279.1| PREDICTED: hypothetical protein LOC100330441 [Danio rerio]
          Length = 242

 Score = 30.8 bits (68), Expect = 0.77,   Method: Compositional matrix adjust.
 Identities = 21/84 (25%), Positives = 34/84 (40%), Gaps = 2/84 (2%)

Query: 110 SPATLSVTPGDRVSLSCRASQSISDYLHWYQQKSHES-PRLLIKYASQXXXXXXXXXXXX 168
           +   + V PG+ V+L C  +     ++ W +  SHE+ P L I                 
Sbjct: 23  TEVNMRVKPGESVTLYCDCALPFGSHIVWMRNCSHENQPSLFIDPPKLFLETFPRFIFLL 82

Query: 169 XXXXD-FTLSINSVEPEDVGVYYC 191
               + + L I +V   D G+YYC
Sbjct: 83  NSSSNSYDLHIENVSVSDEGLYYC 106


&gt;ref|XP_003197952.1| PREDICTED: immunoglobulin omega chain-like [Danio rerio]
          Length = 122

 Score = 30.8 bits (68), Expect = 0.80,   Method: Compositional matrix adjust.
 Identities = 24/76 (31%), Positives = 33/76 (43%), Gaps = 7/76 (9%)

Query: 122 VSLSCRASQSISDY--LHWYQQKSHESPRLLI-KYASQXXXXXXXXXXXXXXXXDFT--- 175
           V LSC  S SI +Y  L+WY+Q  H  P+ L+ +YA                   +T   
Sbjct: 37  VKLSCSYSASI-EYVVLYWYRQYKHGEPQYLLRRYARSYNDYEATSDDRFQSDSSYTSTE 95

Query: 176 LSINSVEPEDVGVYYC 191
           L+I      D  +YYC
Sbjct: 96  LTIKDARLSDSALYYC 111


&gt;ref|NP_571547.1| vascular endothelial growth factor receptor kdr-like precursor
           [Danio rerio]
          Length = 1301

 Score = 30.8 bits (68), Expect = 0.82,   Method: Compositional matrix adjust.
 Identities = 10/34 (29%), Positives = 20/34 (58%)

Query: 115 SVTPGDRVSLSCRASQSISDYLHWYQQKSHESPR 148
           +   GD ++L+CR ++ + D L WY    ++ P+
Sbjct: 553 TAVAGDDITLTCRGTRYLYDRLTWYDPLGYKVPK 586



 Score = 28.5 bits (62), Expect = 3.7,   Method: Compositional matrix adjust.
 Identities = 23/110 (20%), Positives = 46/110 (41%), Gaps = 21/110 (19%)

Query: 85  VSTSQLLGLLLFWTSASRCDIVMTQSPATLSVTPGDRVSLSCRASQSISDYLHWYQQKS- 143
           ++T++++ +    T   R    + Q+     V     ++L+C A    + ++ WY+ K+ 
Sbjct: 622 INTNKVVNVTSALTIDERQGPWLMQNLTNQDVNSSSTLTLACLAYGVPAPFITWYKDKTP 681

Query: 144 -HESPRLLIKYASQXXXXXXXXXXXXXXXXDFTLSINSVEPEDVGVYYCQ 192
             E P + +K                    D TL I  V+ +D G+Y C+
Sbjct: 682 VTEGPGITLK-------------------DDGTLIIERVKKDDEGIYECR 712


&gt;ref|XP_001920377.3| PREDICTED: b-cell receptor CD22-like [Danio rerio]
          Length = 480

 Score = 30.8 bits (68), Expect = 0.85,   Method: Compositional matrix adjust.
 Identities = 38/166 (22%), Positives = 58/166 (34%), Gaps = 29/166 (17%)

Query: 35  PTQKPRYPDLGRKRSGENRHRYRDVCISFIRRSFSALCHNPQKLRENLKMVSTSQLLGLL 94
           P Q   YPDL        R +Y        +     L H  +K         T+ + G +
Sbjct: 64  PKQGQEYPDLSEDPEYSQRLQYLG---DKQQNCHLRLSHVTKKDEHQYYFRFTTNVTGKM 120

Query: 95  LFWTSASRCDIVMTQSPATLSVTPGDRVSLSCRASQSISD--YLHWYQQKSHESPRLLIK 152
              T   R  +   Q  +   VT GD V L+C++S  ++D     WY+     + ++  K
Sbjct: 121 WLGTPGVRLSVTDLQLESPERVTEGDSVRLTCKSSCKLTDTPTFIWYRNSVTLTGKIGNK 180

Query: 153 YASQXXXXXXXXXXXXXXXXDFTLSINSVEPEDVGVYYCQ-NGHSF 197
                                  L +N V  ED G Y C  +GH+ 
Sbjct: 181 -----------------------LILNPVRREDAGRYSCGVDGHTL 203


&gt;ref|NP_001011881.1| novel immune-type receptor 1j precursor [Danio rerio]
          Length = 323

 Score = 30.4 bits (67), Expect = 0.88,   Method: Compositional matrix adjust.
 Identities = 28/116 (24%), Positives = 51/116 (43%), Gaps = 10/116 (8%)

Query: 84  MVSTSQLLGL-----LLFWTSASRCDIVMTQSPATLSVTPGDRVSLSCR--ASQSISDY- 135
           ++STS  +G+     LL   +A+  +  + QS    +V PGD V+L C         D+ 
Sbjct: 110 VISTSYNIGMGSGTRLLVRDAAADKNTTLHQSLID-TVDPGDSVNLQCSIFTESCAGDHS 168

Query: 136 LHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXXXXXDFTLSINSVEPEDVGVYYC 191
           ++W++Q S +S  +L     +                 ++   N++   D G+YYC
Sbjct: 169 VYWFKQSSGDSEGVLYT-KGETNGHCNKSTKSQTQSCVYSFHKNNISRSDSGIYYC 223


&gt;ref|NP_571721.1| novel immune-type receptor 1i [Danio rerio]
          Length = 335

 Score = 30.4 bits (67), Expect = 0.90,   Method: Compositional matrix adjust.
 Identities = 20/80 (25%), Positives = 35/80 (43%), Gaps = 4/80 (5%)

Query: 115 SVTPGDRVSLSCR--ASQSISDY-LHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXXXX 171
           +V PGD V+L C         D+ ++W++Q S +S  L+     +               
Sbjct: 162 AVDPGDSVNLQCSIFTESCAGDHSIYWFKQISGDSVGLIYTKGERNGRCKNSTESQTQSC 221

Query: 172 XDFTLSINSVEPEDVGVYYC 191
             ++L  N++   D G+YYC
Sbjct: 222 V-YSLHKNNISRSDSGIYYC 240


&gt;ref|XP_003197934.1| PREDICTED: hypothetical protein LOC100534813 [Danio rerio]
          Length = 146

 Score = 30.4 bits (67), Expect = 0.92,   Method: Compositional matrix adjust.
 Identities = 21/79 (26%), Positives = 30/79 (37%), Gaps = 4/79 (5%)

Query: 116 VTPGDRVSLSCRASQSISDYLHWYQQKSHESPRLLIKYAS---QXXXXXXXXXXXXXXXX 172
           V  G   +LSC+ + ++   L WY+Q     P  +I +     Q                
Sbjct: 53  VLAGKNATLSCKYNGNVYS-LQWYRQYPGSKPENIIYHTESNKQSHSRLRLLAVTEKGMK 111

Query: 173 DFTLSINSVEPEDVGVYYC 191
              LSI   E ED  +YYC
Sbjct: 112 RMNLSIFHTEMEDSALYYC 130


&gt;ref|XP_003197935.1| PREDICTED: ig heavy chain V-III region CAM-like [Danio rerio]
          Length = 176

 Score = 30.4 bits (67), Expect = 0.94,   Method: Compositional matrix adjust.
 Identities = 25/93 (26%), Positives = 36/93 (38%), Gaps = 12/93 (12%)

Query: 116 VTPGDRVSLSCRASQSISDY-LHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXXXXXDF 174
           +T G  ++LSC  + +I+ Y LHWY+Q     P  L+                     + 
Sbjct: 16  LTEGSNITLSC--TFNITPYSLHWYRQMPGSRPEFLLLIVRSTKTKVAASPPQPHMSINL 73

Query: 175 T-----LSINSVEPEDVGVYYCQNG----HSFP 198
           +     L I+S    D  VYYC       H FP
Sbjct: 74  SEKRVDLEISSAAVSDSAVYYCALEPTLYHQFP 106


&gt;ref|XP_003197820.1| PREDICTED: hypothetical protein LOC100537163 [Danio rerio]
          Length = 393

 Score = 30.4 bits (67), Expect = 0.98,   Method: Compositional matrix adjust.
 Identities = 17/78 (21%), Positives = 36/78 (46%)

Query: 114 LSVTPGDRVSLSCRASQSISDYLHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXXXXXD 173
           +S  PG  +S++C  ++ + + + +  + S+ S    I   S                 +
Sbjct: 137 VSGEPGKHLSITCSYTRDLKELIRFLCKGSNPSHCKNIIKVSSETKTNGRFSLTDDSERN 196

Query: 174 FTLSINSVEPEDVGVYYC 191
           FT++I+ +  ED G+Y+C
Sbjct: 197 FTVNISDLTEEDSGIYWC 214


&gt;ref|XP_003201143.1| PREDICTED: hypothetical protein LOC100537953 [Danio rerio]
          Length = 192

 Score = 30.4 bits (67), Expect = 1.00,   Method: Compositional matrix adjust.
 Identities = 19/83 (22%), Positives = 38/83 (45%), Gaps = 6/83 (7%)

Query: 119 GDRVSLSCRASQSISDYLHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXXXXXDF--TL 176
           G  VS++C   +   + ++W  QK+ E P ++++  S                  F   L
Sbjct: 26  GSNVSINCNLDE---NEVYWIIQKTPEPPTVILRSFSTPAASYFNKSFRSKYSVQFKHNL 82

Query: 177 SINSVEPEDVGVYYCQNGHSFPP 199
            I+++  +++G+YYC    + PP
Sbjct: 83  VIHNITTDELGIYYCMKTGT-PP 104


&gt;ref|NP_938163.1| novel immune-type receptor 1d precursor [Danio rerio]
          Length = 324

 Score = 30.0 bits (66), Expect = 1.2,   Method: Compositional matrix adjust.
 Identities = 20/80 (25%), Positives = 34/80 (42%), Gaps = 4/80 (5%)

Query: 115 SVTPGDRVSLSCR--ASQSISDY-LHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXXXX 171
           +V PGD V L C         D+ ++W++Q S +S  +L     +               
Sbjct: 147 AVDPGDSVHLQCSIFTESCAGDHRVYWFKQSSGDSEGVLYTKGERNGRCKNSTESQTQSC 206

Query: 172 XDFTLSINSVEPEDVGVYYC 191
             ++L  N++   D G+YYC
Sbjct: 207 V-YSLHKNNISRSDSGIYYC 225


&gt;ref|XP_003197947.1| PREDICTED: immunoglobulin omega chain-like [Danio rerio]
          Length = 126

 Score = 30.0 bits (66), Expect = 1.2,   Method: Compositional matrix adjust.
 Identities = 24/95 (25%), Positives = 39/95 (41%), Gaps = 13/95 (13%)

Query: 119 GDRVSLSCRASQSISD---YLHWYQQKSHESPRLLI-KYASQXXXXXXXXXXXXXXXXDF 174
           GD+++L C  + + ++   YL WY+Q    SP  ++ K+                   + 
Sbjct: 36  GDQITLMCNYTTTSTNADVYLFWYKQLPDRSPTFILNKFPFSEGTIEPEFNKRFSAKLNS 95

Query: 175 TLS-----INSVEPEDVGVYYCQNGHSFPPTMMQT 204
           TL      I  +   D  VYYC    +  PT+ QT
Sbjct: 96  TLRTVPLLIQDLRVSDSAVYYC----ALRPTLTQT 126


&gt;ref|NP_938164.1| novel immune-type receptor 1c precursor [Danio rerio]
          Length = 328

 Score = 30.0 bits (66), Expect = 1.3,   Method: Compositional matrix adjust.
 Identities = 20/80 (25%), Positives = 35/80 (43%), Gaps = 4/80 (5%)

Query: 115 SVTPGDRVSLSCRA---SQSISDYLHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXXXX 171
           +V PGD V+L C     S + +  ++W++Q S   P  +I    +               
Sbjct: 147 TVDPGDSVNLQCSIFTESCAKNHSVYWFKQSSGH-PEGVIYTKGERNDRCKNSTESQTQS 205

Query: 172 XDFTLSINSVEPEDVGVYYC 191
             ++L  N++   D G+YYC
Sbjct: 206 CVYSLHKNNISHSDSGIYYC 225


&gt;ref|NP_938169.1| novel immune-type receptor 3d isoform 1 precursor [Danio rerio]
          Length = 223

 Score = 30.0 bits (66), Expect = 1.3,   Method: Compositional matrix adjust.
 Identities = 21/81 (25%), Positives = 31/81 (38%), Gaps = 9/81 (11%)

Query: 119 GDRVSLSCRASQSISDYLHWYQQKSHESPRLLIKYAS--------QXXXXXXXXXXXXXX 170
           G  V+L C  S      + WY+  + + P LLI +++        Q              
Sbjct: 35  GSSVTLPCFHSDDFITTISWYKHSAGKKP-LLIAHSAPNSGRVTYQNAFNNTNRFFITIA 93

Query: 171 XXDFTLSINSVEPEDVGVYYC 191
              + LSI  +E ED   YYC
Sbjct: 94  SGSYNLSIIHLEKEDFATYYC 114


&gt;ref|NP_001120804.1| secreted immunoglobulin domain 1 precursor [Danio rerio]
          Length = 136

 Score = 30.0 bits (66), Expect = 1.4,   Method: Compositional matrix adjust.
 Identities = 15/45 (33%), Positives = 26/45 (57%), Gaps = 7/45 (15%)

Query: 113 TLSVTPGDRVSLSC----RASQSISDYLHWYQQKSHESPRLLIKY 153
           T+SV  G  V+LSC     +S+ +   L WY+Q + +  R+++ Y
Sbjct: 29  TISVQQGQNVTLSCPLMPNSSEGV---LSWYKQTAGQGLRIILSY 70


&gt;ref|NP_001245248.1| diverse immunoglobulin domain-containing protein 1.1 precursor
           [Danio rerio]
          Length = 318

 Score = 29.6 bits (65), Expect = 1.5,   Method: Compositional matrix adjust.
 Identities = 32/123 (26%), Positives = 43/123 (34%), Gaps = 14/123 (11%)

Query: 84  MVSTSQLLGLLLFWTSASRCDIVMTQSPATLSVTPGDRVSLSCRASQSI--SDYLHWYQQ 141
           M   S L  LL+FWT ++    V       +    G+ V L CR +     +    W   
Sbjct: 1   MADRSPLFLLLIFWTLSTG---VFGAGETRVFSRSGETVHLPCRNTDQHCRTSGTTWI-- 55

Query: 142 KSHESPRLLIKYASQXXXXXXXXXXXXXXXXDFTLSINSVEPEDVGVYYCQ-----NGHS 196
             + SPRL                       D +L I+ V  ED G Y CQ      GH 
Sbjct: 56  --YSSPRLTAAVELIELGVKKTNSERLNLGSDCSLIISRVTTEDAGSYICQQWSGKGGHH 113

Query: 197 FPP 199
           + P
Sbjct: 114 YGP 116


&gt;ref|XP_001923759.3| PREDICTED: hypothetical protein LOC100004978 [Danio rerio]
          Length = 243

 Score = 29.6 bits (65), Expect = 1.6,   Method: Compositional matrix adjust.
 Identities = 23/82 (28%), Positives = 34/82 (41%), Gaps = 6/82 (7%)

Query: 114 LSVTPGDRVSLSCRASQSISDYLHWYQQKSHES-PRLLI---KYASQXXXXXXXXXXXXX 169
           + V PG+ V+L C  +     ++ W +  SHE+ P L I   K   +             
Sbjct: 27  MRVKPGESVTLYCDCALPFGSHIVWMRNCSHENQPSLFIDPPKLFLETFPRFSFLLNSSS 86

Query: 170 XXXDFTLSINSVEPEDVGVYYC 191
              D  L I +V   D G+YYC
Sbjct: 87  NSYD--LHIENVSVSDEGLYYC 106


&gt;ref|NP_938168.1| novel immune-type receptor 3d isoform 2 precursor [Danio rerio]
          Length = 223

 Score = 29.6 bits (65), Expect = 1.6,   Method: Compositional matrix adjust.
 Identities = 21/81 (25%), Positives = 31/81 (38%), Gaps = 9/81 (11%)

Query: 119 GDRVSLSCRASQSISDYLHWYQQKSHESPRLLIKYAS--------QXXXXXXXXXXXXXX 170
           G  V+L C  S      + WY+  + + P LLI +++        Q              
Sbjct: 35  GSSVTLPCFHSDDFITTISWYKHSAGKKP-LLIAHSAPNSGRVTYQNTFNNTNRFFITIA 93

Query: 171 XXDFTLSINSVEPEDVGVYYC 191
              + LSI  +E ED   YYC
Sbjct: 94  SGSYNLSILHLEKEDFATYYC 114


&gt;ref|NP_001007212.1| novel immune-type receptor 3b [Danio rerio]
          Length = 230

 Score = 29.6 bits (65), Expect = 1.9,   Method: Compositional matrix adjust.
 Identities = 21/81 (25%), Positives = 31/81 (38%), Gaps = 9/81 (11%)

Query: 119 GDRVSLSCRASQSISDYLHWYQQKSHESPRLLIKYAS--------QXXXXXXXXXXXXXX 170
           G  V+L C  S      + WY+  + + P LLI +++        Q              
Sbjct: 35  GSSVTLPCFHSDDFITTISWYKHSAGKKP-LLIAHSAPNSGSVTYQYAFNNTNRFFITIA 93

Query: 171 XXDFTLSINSVEPEDVGVYYC 191
              + LSI  +E ED   YYC
Sbjct: 94  SGSYNLSILHLEKEDFANYYC 114


&gt;ref|NP_991212.1| uncharacterized protein LOC402947 [Danio rerio]
          Length = 140

 Score = 29.3 bits (64), Expect = 2.0,   Method: Compositional matrix adjust.
 Identities = 10/38 (26%), Positives = 21/38 (55%)

Query: 107 MTQSPATLSVTPGDRVSLSCRASQSISDYLHWYQQKSH 144
           + + P+ L V+ G+  +L+C+A    +  + WY+   H
Sbjct: 29  IVEDPSDLIVSKGEPATLNCKAEGRPAPTVQWYKDGEH 66


&gt;ref|NP_571727.1| novel immune-type receptor 3a precursor [Danio rerio]
          Length = 337

 Score = 29.3 bits (64), Expect = 2.2,   Method: Compositional matrix adjust.
 Identities = 22/90 (24%), Positives = 39/90 (43%), Gaps = 5/90 (5%)

Query: 107 MTQSPATLSVTPGDRVSLSCRASQSI--SDY-LHWYQQKS-HESPRLLIKYASQXXX-XX 161
           + Q P +  + PGD V+L C  S  I    Y ++W++  S +  P ++  + ++      
Sbjct: 143 VIQQPVSDRLHPGDSVTLQCSVSSHICAGHYRVYWFKHSSGYSQPGIIYTHDNRSDQCLK 202

Query: 162 XXXXXXXXXXXDFTLSINSVEPEDVGVYYC 191
                       ++LS   +   D GVYYC
Sbjct: 203 SSEKSSFVQSCVYSLSQTELTTSDAGVYYC 232


&gt;ref|NP_001007203.1| novel immune-type receptor precursor [Danio rerio]
          Length = 333

 Score = 29.3 bits (64), Expect = 2.4,   Method: Compositional matrix adjust.
 Identities = 23/91 (25%), Positives = 34/91 (37%), Gaps = 10/91 (10%)

Query: 109 QSPATLSVTPGDRVSLSCRASQSISDYLHWYQQKSHESPRLLIKYAS--------QXXXX 160
           Q P  L+   G  V+L C  S      + WY+  + + P LLI Y+         +    
Sbjct: 26  QQPLVLA-ELGSSVTLPCFHSDDFVTTISWYKHSAGKKP-LLIAYSDFNSGRVTYRNAFN 83

Query: 161 XXXXXXXXXXXXDFTLSINSVEPEDVGVYYC 191
                        + L+I  +E ED   YYC
Sbjct: 84  NTNRFFITVASGSYNLTIIHLEKEDFATYYC 114



 Score = 28.9 bits (63), Expect = 3.2,   Method: Compositional matrix adjust.
 Identities = 22/90 (24%), Positives = 39/90 (43%), Gaps = 5/90 (5%)

Query: 107 MTQSPATLSVTPGDRVSLSCRASQSI--SDY-LHWYQQKS-HESPRLLIKYASQXXX-XX 161
           + Q P +  + PGD V+L C  S  I    Y ++W++  S +  P ++  + ++      
Sbjct: 143 VIQQPVSDRLHPGDSVTLQCSVSSHICAGHYRVYWFKHSSGYSQPGIIYTHDNRSDQCLE 202

Query: 162 XXXXXXXXXXXDFTLSINSVEPEDVGVYYC 191
                       ++LS   +   D GVYYC
Sbjct: 203 SSEKSSFVQSCVYSLSQTELTTSDAGVYYC 232


&gt;ref|XP_001921605.2| PREDICTED: hypothetical protein LOC559375 [Danio rerio]
          Length = 814

 Score = 28.9 bits (63), Expect = 2.7,   Method: Compositional matrix adjust.
 Identities = 19/88 (21%), Positives = 33/88 (37%), Gaps = 20/88 (22%)

Query: 44  LGRKRSGENRHRYRDVCISFIRRSFSALCHNPQKLR-----ENLKMVSTSQLLGLLLFWT 98
           LG   S E        C S+++    A+C +  KL+     EN+   +            
Sbjct: 150 LGGTESPEKAANQTQNCASYVKDEMEAVCGDHDKLKCQDGEENIDFEAV----------- 198

Query: 99  SASRCDIVMTQSPATLSVTPGDRVSLSC 126
               CD+ +T +P +L +  G  +   C
Sbjct: 199 ----CDVNLTNTPPSLEILKGKEILKPC 222


&gt;ref|XP_002663948.1| PREDICTED: hypothetical protein LOC100329481 [Danio rerio]
          Length = 315

 Score = 28.9 bits (63), Expect = 2.8,   Method: Compositional matrix adjust.
 Identities = 31/129 (24%), Positives = 50/129 (38%), Gaps = 18/129 (13%)

Query: 90  LLG---LLLFWTSASRCDIVMTQSPATLSVTPGDRVSLSCRASQSISD--YLHWYQQKSH 144
           LLG   + L + SA    I++ Q P  + V  G  V + C  S        + W++ K+ 
Sbjct: 112 LLGFSVMALIYLSAEG-SIIVFQKPRFIGVKTGRTVIIYCVPSNPTRGPARVQWFKNKN- 169

Query: 145 ESPRLLIKYASQXXXXXXXXXXXXXXXXDFTLSINSVEPEDVGVYYCQNGHSFPPTMMQT 204
                   Y S                 + ++++  VE ED G Y C+   +F P    T
Sbjct: 170 --------YQSSLTNDQRITINDRNNDKNASITLKKVETEDSGTYVCKLNDTFGP---GT 218

Query: 205 IAKTSRKAE 213
             + SR +E
Sbjct: 219 ALQVSRYSE 227


&gt;ref|NP_001153138.1| immunoglobulin superfamily member 3 precursor [Danio rerio]
          Length = 1042

 Score = 28.9 bits (63), Expect = 2.9,   Method: Compositional matrix adjust.
 Identities = 22/105 (20%), Positives = 42/105 (40%), Gaps = 17/105 (16%)

Query: 105 IVMTQSPATLSVTPGDRVSLSCRASQSISDYLH----WYQQKSHE---SPRLLIKYASQX 157
           + +T +  TL+   GD + LSC  ++  S + H    W+ + + +    PR L+  +   
Sbjct: 146 LAVTAAAQTLAKVEGDTLQLSCEVTRQTSQHTHVSVGWFLRPTDDPSSGPRDLVTLSRDF 205

Query: 158 XXXXXXXXXXXXXXXD----------FTLSINSVEPEDVGVYYCQ 192
                          D          + L+I  ++P D G +YC+
Sbjct: 206 VLRPGGQYRQRLSSGDLRLDKTSATSYRLTIYKLQPSDQGEFYCE 250


&gt;ref|XP_003197821.1| PREDICTED: polymeric immunoglobulin receptor-like [Danio rerio]
          Length = 423

 Score = 28.9 bits (63), Expect = 2.9,   Method: Compositional matrix adjust.
 Identities = 17/79 (21%), Positives = 39/79 (49%), Gaps = 2/79 (2%)

Query: 114 LSVTPGDRVSLSCRASQSISDYLHWYQQKSHESP-RLLIKYASQXXXXXXXXXXXXXXXX 172
           +S  PG  ++++C  ++ + + + +  + S  S  + +IK +S+                
Sbjct: 131 VSGEPGKHLNITCSYTRDLKELIRFLCKGSDPSDCKKMIKVSSETNTTGRFSLTDDSER- 189

Query: 173 DFTLSINSVEPEDVGVYYC 191
           +FT++I  +  ED G+Y+C
Sbjct: 190 NFTVNIRDLTEEDSGIYWC 208


&gt;ref|NP_694508.1| mucosa associated lymphoid tissue lymphoma translocation gene 1
           [Danio rerio]
          Length = 806

 Score = 28.9 bits (63), Expect = 3.0,   Method: Compositional matrix adjust.
 Identities = 22/82 (26%), Positives = 35/82 (42%), Gaps = 7/82 (8%)

Query: 78  LRENLKMVSTSQLLGLLLFWTSASRCDIVMTQSPATLSVTPGDRVSLSCRASQSISDYLH 137
           L + LK +   + +G L       + DI+  Q P +  +  G  V LSC+A   +     
Sbjct: 82  LLQCLKKIEHREAVGFLTM-NVMEQIDII--QQPQSQRIAEGSPVVLSCKAVGPVELQYQ 138

Query: 138 WYQQKSH----ESPRLLIKYAS 155
           W++ K       SP L+   AS
Sbjct: 139 WFKGKDEMPEGNSPDLVFNRAS 160


&gt;ref|NP_001245161.1| diverse immunoglobulin domain-containing protein 1.3-4 precursor
           [Danio rerio]
          Length = 286

 Score = 28.9 bits (63), Expect = 3.2,   Method: Compositional matrix adjust.
 Identities = 32/114 (28%), Positives = 49/114 (42%), Gaps = 12/114 (10%)

Query: 84  MVSTSQLLGLLLFWTSASRC----DI-VMTQSPATLSVTPGDRVSLSCRASQSISDYLHW 138
           M   S L  LL+FWT ++      DI V ++S  T+ + P +     CR S++   Y  +
Sbjct: 1   MADRSPLFLLLIFWTLSTGVFGVEDIDVFSRSGETVHL-PCNNTVQHCRPSETTWTYGRY 59

Query: 139 YQQKSHESPRLLIKYASQXXXXXXXXXXXXXXXXDFTLSINSVEPEDVGVYYCQ 192
            Q  + E    LI +  +                D +L I+ V  ED G+Y CQ
Sbjct: 60  RQTTADE----LICFGRKKAKPEKHERLNLGS--DCSLIISRVTTEDAGLYICQ 107


&gt;ref|NP_001018172.1| novel immune-type receptor 2a precursor [Danio rerio]
          Length = 328

 Score = 28.9 bits (63), Expect = 3.2,   Method: Compositional matrix adjust.
 Identities = 18/82 (21%), Positives = 30/82 (36%), Gaps = 6/82 (7%)

Query: 116 VTPGDRVSLSCRASQSISDYLHWYQQKSHESPRLLI------KYASQXXXXXXXXXXXXX 169
           V  GD V+ +C     I + + W +Q   ++P  ++       +  +             
Sbjct: 28  VQAGDTVNFTCSLLNEIRNSVVWIKQSVGKNPLTVVSSFQNVDHKFENDFDKHNRFFVTK 87

Query: 170 XXXDFTLSINSVEPEDVGVYYC 191
               F LSI + E  D   YYC
Sbjct: 88  DDVSFNLSITNTEVSDSATYYC 109


&gt;ref|NP_001005578.1| novel immune-type receptor 7b precursor [Danio rerio]
          Length = 243

 Score = 28.5 bits (62), Expect = 3.3,   Method: Compositional matrix adjust.
 Identities = 26/96 (27%), Positives = 37/96 (38%), Gaps = 9/96 (9%)

Query: 105 IVMTQSPATL-SVTPGDRVSLSCRASQSISDYLHWYQQKSHESPRLLI-KYA------SQ 156
           I+  +  A + +V  G  VSL C      S YL W++Q     P  ++ +Y       S 
Sbjct: 16  IIFVKHAAFMETVYSGTNVSLLCSNILKESSYLAWFKQTDDSLPLCIVTQYVGLNPADST 75

Query: 157 XXXXXXXXXXXXXXXXDF-TLSINSVEPEDVGVYYC 191
                            F TL I SV+  D G +YC
Sbjct: 76  YLNGFQKTRIEMSVNATFSTLRIISVDVSDSGTFYC 111


&gt;ref|XP_001342685.4| PREDICTED: hypothetical protein LOC100003040 [Danio rerio]
          Length = 416

 Score = 28.5 bits (62), Expect = 3.5,   Method: Compositional matrix adjust.
 Identities = 18/48 (37%), Positives = 25/48 (52%), Gaps = 1/48 (2%)

Query: 93  LLLFWTSASRCDIVMTQSPATLSVTPGDRVSLSCRASQSIS-DYLHWY 139
           +LLF  S +    V T+    LSVT GD ++L    SQ  S D + W+
Sbjct: 7   ILLFVLSINAVIRVETEDVECLSVTVGDSLTLQTNVSQIESNDAIEWW 54


&gt;ref|XP_003201336.1| PREDICTED: hypothetical protein LOC100537913 [Danio rerio]
          Length = 720

 Score = 28.5 bits (62), Expect = 3.7,   Method: Composition-based stats.
 Identities = 11/20 (55%), Positives = 14/20 (70%)

Query: 173 DFTLSINSVEPEDVGVYYCQ 192
           +F+L + SV  ED GVY CQ
Sbjct: 217 NFSLRLKSVRTEDKGVYMCQ 236


&gt;ref|NP_001007220.1| novel immune-type receptor 1a precursor [Danio rerio]
          Length = 329

 Score = 28.5 bits (62), Expect = 4.0,   Method: Compositional matrix adjust.
 Identities = 19/80 (23%), Positives = 34/80 (42%), Gaps = 4/80 (5%)

Query: 115 SVTPGDRVSLSCR--ASQSISDY-LHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXXXX 171
           +V PGD V+L C         D+ ++W++Q S   P  ++    +               
Sbjct: 147 TVDPGDAVNLQCSIFTESCAGDHSVYWFKQSSGH-PEGVLYTKGERNGRCKKTTESQTQS 205

Query: 172 XDFTLSINSVEPEDVGVYYC 191
             ++L  N++   D G+YYC
Sbjct: 206 CVYSLHKNNISRSDSGIYYC 225


&gt;ref|XP_003200226.1| PREDICTED: hypothetical protein LOC100006926 [Danio rerio]
          Length = 578

 Score = 28.5 bits (62), Expect = 4.0,   Method: Compositional matrix adjust.
 Identities = 22/84 (26%), Positives = 30/84 (35%), Gaps = 6/84 (7%)

Query: 119 GDRVSLSCRASQSISDYLHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXXXXXDFTLSI 178
           G+ V L C  + S      W   +  E+  L+   A                  D +L+I
Sbjct: 320 GENVHLPCNNAHSGCTSTTWNYNRHSETVELV---AGGSLKTNIKRRERLSLGSDCSLNI 376

Query: 179 NSVEPEDVGVYYCQ---NGHSFPP 199
             V  ED G Y CQ   NG  + P
Sbjct: 377 KKVTKEDYGSYICQQYVNGEQYGP 400


&gt;ref|NP_001005576.1| novel immune-type receptor 9 precursor [Danio rerio]
          Length = 285

 Score = 28.1 bits (61), Expect = 4.3,   Method: Compositional matrix adjust.
 Identities = 14/61 (22%), Positives = 30/61 (49%), Gaps = 3/61 (4%)

Query: 93  LLLFWTSASRCDIVMTQSPAT---LSVTPGDRVSLSCRASQSISDYLHWYQQKSHESPRL 149
           ++ FW     C I ++ +      + V  G+ V++SC     ++ +  WY+ K  ++P L
Sbjct: 1   MINFWIFGLFCLIGISSTHPNAPPVFVKLGESVNMSCIYQSQMARHFSWYKHKPGQNPML 60

Query: 150 L 150
           +
Sbjct: 61  I 61


&gt;ref|XP_003197826.1| PREDICTED: CMRF35-like molecule 9-like [Danio rerio]
          Length = 423

 Score = 28.1 bits (61), Expect = 4.4,   Method: Compositional matrix adjust.
 Identities = 18/79 (22%), Positives = 38/79 (48%), Gaps = 2/79 (2%)

Query: 114 LSVTPGDRVSLSCRASQSISDYLHWYQQKSHESP-RLLIKYASQXXXXXXXXXXXXXXXX 172
           +S  PG  +S++C  ++ + + + +  + S  S  + +IK +S+                
Sbjct: 131 VSGEPGKHLSITCSYTRDLKELIRFLCKGSDPSDCKNIIKVSSKTKTNGRFSLTDDSETV 190

Query: 173 DFTLSINSVEPEDVGVYYC 191
            FT++I  +  ED G+Y+C
Sbjct: 191 -FTVNIRDLTEEDSGIYWC 208


&gt;ref|XP_001923800.1| PREDICTED: titin [Danio rerio]
          Length = 32757

 Score = 28.1 bits (61), Expect = 4.4,   Method: Composition-based stats.
 Identities = 11/32 (34%), Positives = 18/32 (56%)

Query: 109  QSPATLSVTPGDRVSLSCRASQSISDYLHWYQ 140
            + P  +SVT GD  +L C+ S S    + W++
Sbjct: 5181 EKPPCISVTAGDSATLECKVSGSPDLKVKWFK 5212


&gt;ref|NP_001073429.2| matrix-remodelling associated 8b precursor [Danio rerio]
          Length = 434

 Score = 28.1 bits (61), Expect = 5.2,   Method: Compositional matrix adjust.
 Identities = 9/25 (36%), Positives = 18/25 (72%)

Query: 173 DFTLSINSVEPEDVGVYYCQNGHSF 197
           DF+LS+++++P D G++ C   H +
Sbjct: 246 DFSLSLHNIQPADKGLFSCHLHHHY 270



 Score = 28.1 bits (61), Expect = 5.3,   Method: Compositional matrix adjust.
 Identities = 15/46 (32%), Positives = 23/46 (50%), Gaps = 5/46 (10%)

Query: 173 DFTLSINSVEPEDVGVYYCQNGHSFPPT-----MMQTIAKTSRKAE 213
           +F+L IN+V+  D G+Y C   H +        M   I K+ RK +
Sbjct: 110 NFSLVINNVDMNDKGIYTCNLHHHYCKVHQSVQMQLNITKSPRKEK 155


&gt;ref|XP_003200434.1| PREDICTED: hypothetical protein LOC100534684 [Danio rerio]
          Length = 302

 Score = 28.1 bits (61), Expect = 5.4,   Method: Compositional matrix adjust.
 Identities = 16/61 (26%), Positives = 30/61 (49%), Gaps = 5/61 (8%)

Query: 136 LHWYQQKSHESPRLLI--KYASQXXXXXXXXXXXXXXXXDFT---LSINSVEPEDVGVYY 190
           ++WY+Q+  E+ RL++     S+                D T   L++ ++EPED  +Y+
Sbjct: 1   MYWYRQRPGETMRLIVFTMENSKPDFGDFDEKKYDASKEDPTSGSLTVKNLEPEDSAIYF 60

Query: 191 C 191
           C
Sbjct: 61  C 61


&gt;ref|NP_956958.1| cell adhesion molecule 2a precursor [Danio rerio]
          Length = 405

 Score = 28.1 bits (61), Expect = 5.5,   Method: Compositional matrix adjust.
 Identities = 18/78 (23%), Positives = 33/78 (42%), Gaps = 5/78 (6%)

Query: 114 LSVTPGDRVSLSCRASQSISDYLHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXXXXXD 173
           ++VT G   +L+CR   + +  L W    S+ + + L  +  +                +
Sbjct: 47  MTVTEGGTANLTCRVEHNDNTSLQW----SNPAQQTLF-FGDKKALRDNRIELVRATSQE 101

Query: 174 FTLSINSVEPEDVGVYYC 191
            T+SI+ V   D G+Y C
Sbjct: 102 LTISISEVSLSDEGLYTC 119


&gt;ref|XP_003197943.1| PREDICTED: immunoglobulin omega chain-like [Danio rerio]
          Length = 125

 Score = 27.7 bits (60), Expect = 6.3,   Method: Compositional matrix adjust.
 Identities = 22/79 (27%), Positives = 31/79 (39%), Gaps = 7/79 (8%)

Query: 119 GDRVSLSCRASQSISDYLHWYQQKSHESPR-LLIKYASQXXXXXX-----XXXXXXXXXX 172
            + VSLSC+     ++ L WY+Q    +P  LL+ Y S                      
Sbjct: 34  SETVSLSCKYDTQANN-LQWYRQYPGSTPEYLLLVYESNTVFHAEPPFPRLDASVNKVEK 92

Query: 173 DFTLSINSVEPEDVGVYYC 191
              L+I+S    D  VYYC
Sbjct: 93  QVNLTISSAAVTDSAVYYC 111


&gt;ref|XP_002662355.1| PREDICTED: myosin light chain kinase, smooth muscle [Danio rerio]
          Length = 1002

 Score = 27.7 bits (60), Expect = 6.5,   Method: Composition-based stats.
 Identities = 20/92 (21%), Positives = 35/92 (38%), Gaps = 16/92 (17%)

Query: 106 VMTQSPATLSVTPGDRVSLSCRASQSISDYLHWYQQKSHESPRLLIKYASQXXXXXXXXX 165
           V  Q  + ++V  G+R+ L C+ +  +   + W        P   I  + +         
Sbjct: 167 VFKQPLSDITVVDGERLRLECQVTSDLQAAITWILDGKTVKPSKFIVLSHEGEKC----- 221

Query: 166 XXXXXXXDFTLSINSVEPEDVGVYYC--QNGH 195
                    +L+I+   PED G Y C  +N H
Sbjct: 222 ---------SLTIDKALPEDEGRYTCRAENAH 244


&gt;ref|NP_001018542.1| uncharacterized protein LOC553735 [Danio rerio]
          Length = 325

 Score = 27.7 bits (60), Expect = 6.9,   Method: Compositional matrix adjust.
 Identities = 11/20 (55%), Positives = 14/20 (70%)

Query: 173 DFTLSINSVEPEDVGVYYCQ 192
           +F+L + SV  ED GVY CQ
Sbjct: 226 NFSLKLKSVRTEDKGVYMCQ 245


&gt;ref|NP_571730.1| novel immune-type receptor 4a isoform 2 precursor [Danio rerio]
          Length = 337

 Score = 27.7 bits (60), Expect = 7.0,   Method: Compositional matrix adjust.
 Identities = 18/89 (20%), Positives = 37/89 (41%), Gaps = 6/89 (6%)

Query: 107 MTQSPATLSVTPGDRVSLSCRASQSISDY----LHWYQQKSHESPRLLIKYASQXXXXXX 162
           + Q P    + PG  V+L C   +   D     ++W++Q S+ + + ++    +      
Sbjct: 142 VIQLPVHAVINPGSNVTLRCSVERKTCDKDVQSVYWFRQSSNTTHQGIVYTHGKSKNECA 201

Query: 163 XXXXXXXXXXDFTLSINSVEPEDVGVYYC 191
                     +  L+  SV+  + G+YYC
Sbjct: 202 DSSDTQSCLQN--LAKMSVDVSEAGLYYC 228


&gt;ref|NP_001007293.1| novel immune-type receptor 1f precursor [Danio rerio]
          Length = 327

 Score = 27.7 bits (60), Expect = 7.1,   Method: Compositional matrix adjust.
 Identities = 19/80 (23%), Positives = 33/80 (41%), Gaps = 4/80 (5%)

Query: 115 SVTPGDRVSLSCR--ASQSISDY-LHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXXXX 171
           +V PGD V+L C         D+ ++W++Q   +S  +L     +               
Sbjct: 146 TVDPGDSVNLQCSIFTESCAGDHSVYWFKQSLGDSEGVLYTKGERNGRCKNSTESQTQSC 205

Query: 172 XDFTLSINSVEPEDVGVYYC 191
             ++L  N+    D G+YYC
Sbjct: 206 V-YSLHKNNFSGSDSGIYYC 224


&gt;ref|XP_684938.3| PREDICTED: butyrophilin subfamily 2 member A2-like [Danio rerio]
          Length = 316

 Score = 27.7 bits (60), Expect = 7.2,   Method: Compositional matrix adjust.
 Identities = 11/20 (55%), Positives = 14/20 (70%)

Query: 173 DFTLSINSVEPEDVGVYYCQ 192
           +F+L + SV  ED GVY CQ
Sbjct: 226 NFSLRLKSVRTEDKGVYMCQ 245


&gt;ref|NP_001120726.1| novel immune-type receptor 13 isoform 2 precursor [Danio rerio]
          Length = 136

 Score = 27.3 bits (59), Expect = 7.3,   Method: Compositional matrix adjust.
 Identities = 14/44 (31%), Positives = 23/44 (52%), Gaps = 1/44 (2%)

Query: 104 DIVMTQSPATLSVTPGDRVSLSCRASQSISDYLHWYQQKSHESP 147
           +IV    P  ++ + G+ V+L C   Q  SD + WY+Q +   P
Sbjct: 20  EIVQQPDPVMMA-SVGENVTLRCIILQEHSDPITWYKQTAGHQP 62


&gt;ref|XP_001341831.3| PREDICTED: hypothetical protein LOC100001930 [Danio rerio]
          Length = 417

 Score = 27.3 bits (59), Expect = 7.5,   Method: Compositional matrix adjust.
 Identities = 11/21 (52%), Positives = 14/21 (66%)

Query: 173 DFTLSINSVEPEDVGVYYCQN 193
           +F+L + SV  ED GVY CQ 
Sbjct: 199 NFSLRLKSVRTEDKGVYMCQG 219


&gt;ref|XP_003199104.1| PREDICTED: leucine-rich repeat neuronal protein 1 [Danio rerio]
          Length = 744

 Score = 27.3 bits (59), Expect = 7.7,   Method: Compositional matrix adjust.
 Identities = 31/125 (24%), Positives = 50/125 (40%), Gaps = 14/125 (11%)

Query: 71  LCHNPQKL-RENLKMVSTSQLLGLLLFWTSASR-CDIVMTQS--PATLSVTPGDRVSLSC 126
           LC +PQ L  + L+ V ++        W  AS  C  +++Q   P  L+VT G  ++L C
Sbjct: 397 LCASPQHLVGQALQDVVSAS-------WNGASNTCLPLISQHAFPPQLNVTLGQLLTLDC 449

Query: 127 RASQSISDYLHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXXXXXDFTLSINSVEPEDV 186
            A    +   +W    +    ++  +  S                    L I  +EPED 
Sbjct: 450 WAVADPAPQFYWV---TPTGDKVTSEAVSPSSNEGGGTPKKHRMQDQGALEIPHIEPEDT 506

Query: 187 GVYYC 191
           G+Y C
Sbjct: 507 GLYTC 511


&gt;ref|XP_001922166.2| PREDICTED: e3 ubiquitin-protein ligase TRIM21 [Danio rerio]
          Length = 372

 Score = 27.3 bits (59), Expect = 7.8,   Method: Compositional matrix adjust.
 Identities = 18/90 (20%), Positives = 42/90 (46%), Gaps = 6/90 (6%)

Query: 112 ATLSVTPGDRVSLSCRASQSISDYLHWYQQKSHESPRLLIK-YASQXXXXXXXXXXXXXX 170
           A L    G  V+++C   +   + ++W  QK+ + P ++++ +++               
Sbjct: 19  ADLLTALGSNVTINCDLDE---NDVYWILQKTPDPPTVILRSFSTPASSLYLNKSFRIKY 75

Query: 171 XXDFT--LSINSVEPEDVGVYYCQNGHSFP 198
              F   L IN++  +++G+YYC +  + P
Sbjct: 76  SVQFKHRLVINNLTADELGLYYCMSTRTPP 105


&gt;ref|XP_003197961.1| PREDICTED: CMRF35-like molecule 5-like [Danio rerio]
          Length = 246

 Score = 27.3 bits (59), Expect = 7.8,   Method: Compositional matrix adjust.
 Identities = 21/87 (24%), Positives = 35/87 (40%), Gaps = 11/87 (12%)

Query: 122 VSLSCRASQSISDY--LHWYQQKSHESPRLLIKYASQXXXXXXXXXXXXXXXXDFT---- 175
           ++L C    S + Y  L WYQQ ++ SP+ ++   S                 + +    
Sbjct: 39  ITLECTYKASSTQYPNLLWYQQMTNGSPKYMLMRLSGSSDTNEEFKQRFNAELNTSALSI 98

Query: 176 -LSINSVEPEDVGVYYCQNGHSFPPTM 201
            L+I  +   D  VYYC    +  PT+
Sbjct: 99  PLTIKDLRASDSAVYYC----ALKPTI 121


&gt;ref|XP_002667765.2| PREDICTED: butyrophilin-like protein 2-like [Danio rerio]
          Length = 508

 Score = 27.3 bits (59), Expect = 8.0,   Method: Compositional matrix adjust.
 Identities = 11/20 (55%), Positives = 14/20 (70%)

Query: 173 DFTLSINSVEPEDVGVYYCQ 192
           +F+L + SV  ED GVY CQ
Sbjct: 467 NFSLRLKSVRTEDKGVYMCQ 486


&gt;ref|XP_003199219.1| PREDICTED: myosin light chain kinase, smooth muscle-like [Danio
           rerio]
          Length = 722

 Score = 27.3 bits (59), Expect = 8.6,   Method: Compositional matrix adjust.
 Identities = 20/94 (21%), Positives = 37/94 (39%), Gaps = 14/94 (14%)

Query: 99  SASRCDIVMTQSPATLSVTPGDRVSLSCRASQSISDYLHWYQQKSHESPRLLIKYASQXX 158
           +A + + V  +  + ++V  G+R+ L C+ S      + W        P   I   ++  
Sbjct: 593 AAGKKEPVFVEKLSDVTVLDGERLHLQCQLSSESPAKVTWMLDGKVIRPSKFIIITNEGD 652

Query: 159 XXXXXXXXXXXXXXDFTLSINSVEPEDVGVYYCQ 192
                          ++LSI+   PED G Y C+
Sbjct: 653 R--------------YSLSIDKALPEDEGQYVCR 672


&gt;ref|NP_001018583.1| leucine rich repeat containing 4C precursor [Danio rerio]
          Length = 647

 Score = 27.3 bits (59), Expect = 8.7,   Method: Compositional matrix adjust.
 Identities = 24/95 (25%), Positives = 38/95 (40%), Gaps = 23/95 (24%)

Query: 106 VMTQSPATLSVTPGDRVSLSCRASQSISDYLHWYQQK----SHESPRLLIKYASQXXXXX 161
           V+ + PA L+VT G    L CRA+   S  + W        +H + ++ I   +      
Sbjct: 356 VIVEPPADLNVTEGMAAELKCRANSLTS--VSWITPNGSIMTHGAYKIRISVLN------ 407

Query: 162 XXXXXXXXXXXDFTLSINSVEPEDVGVYYCQNGHS 196
                      D TL+  +V  +D G Y C   +S
Sbjct: 408 -----------DGTLNFTNVTMQDTGTYTCMVSNS 431


&gt;ref|XP_003197822.1| PREDICTED: polymeric immunoglobulin receptor-like [Danio rerio]
          Length = 391

 Score = 27.3 bits (59), Expect = 9.0,   Method: Compositional matrix adjust.
 Identities = 17/79 (21%), Positives = 38/79 (48%), Gaps = 2/79 (2%)

Query: 114 LSVTPGDRVSLSCRASQSISDYLHWYQQKSHESP-RLLIKYASQXXXXXXXXXXXXXXXX 172
           +S  PG  +S++C  ++ + +++ +  + S  S  + +IK +S+                
Sbjct: 137 VSGEPGKHLSITCSYTRDLKEHIRFLCKGSDPSDCKNMIKVSSETKTNGRFSLTDDSERV 196

Query: 173 DFTLSINSVEPEDVGVYYC 191
            FT++I  +   D G+Y+C
Sbjct: 197 -FTVNIRDLAEGDSGIYWC 214


&gt;ref|XP_003201334.1| PREDICTED: butyrophilin subfamily 2 member A2-like [Danio rerio]
          Length = 261

 Score = 27.3 bits (59), Expect = 9.1,   Method: Compositional matrix adjust.
 Identities = 24/100 (24%), Positives = 42/100 (42%), Gaps = 12/100 (12%)

Query: 105 IVMTQSPATLSVTPGDRVSLSCRASQSIS-DYLH---WYQ-QKSHESPRLLIKYASQXXX 159
           ++++ S  ++S   G+ V+L+C     I+ +Y+    W +  K  + P LL +       
Sbjct: 141 LLVSGSDESISAYAGEDVTLNCSVDSHITPEYIEEVSWKKTDKDGDFPVLLYQNNETVSE 200

Query: 160 XX-------XXXXXXXXXXXDFTLSINSVEPEDVGVYYCQ 192
                               +F+L + SV  ED GVY CQ
Sbjct: 201 ATHDKFRGRVEFFTAEIPKGNFSLRLKSVRTEDKGVYMCQ 240


&gt;ref|XP_003198941.1| PREDICTED: b-cell receptor CD22-like [Danio rerio]
          Length = 319

 Score = 27.3 bits (59), Expect = 9.4,   Method: Compositional matrix adjust.
 Identities = 13/29 (44%), Positives = 17/29 (58%)

Query: 115 SVTPGDRVSLSCRASQSISDYLHWYQQKS 143
           SV  GD V+LSC A +S    + WY+  S
Sbjct: 13  SVFTGDTVTLSCDAGRSTVRRIIWYKDSS 41


&gt;ref|XP_684768.2| PREDICTED: butyrophilin-like protein 8-like [Danio rerio]
          Length = 272

 Score = 26.9 bits (58), Expect = 9.5,   Method: Compositional matrix adjust.
 Identities = 11/20 (55%), Positives = 14/20 (70%)

Query: 173 DFTLSINSVEPEDVGVYYCQ 192
           +F+L + SV  ED GVY CQ
Sbjct: 216 NFSLRLKSVRTEDKGVYMCQ 235


&gt;ref|XP_001340824.2| PREDICTED: butyrophilin-like protein 8-like [Danio rerio]
          Length = 246

 Score = 26.9 bits (58), Expect = 9.7,   Method: Compositional matrix adjust.
 Identities = 11/20 (55%), Positives = 14/20 (70%)

Query: 173 DFTLSINSVEPEDVGVYYCQ 192
           +F+L + SV  ED GVY CQ
Sbjct: 209 NFSLRLKSVRTEDKGVYMCQ 228


BLASTP 2.2.24 [Aug-08-2010]


Reference: Altschul, Stephen F., Thomas L. Madden, Alejandro A. Schaffer, 
Jinghui Zhang, Zheng Zhang, Webb Miller, and David J. Lipman (1997), 
"Gapped BLAST and PSI-BLAST: a new generation of protein database search
programs",  Nucleic Acids Res. 25:3389-3402.

Reference for compositional score matrix adjustment: Altschul, Stephen F., 
John C. Wootton, E. Michael Gertz, Richa Agarwala, Aleksandr Morgulis,
Alejandro A. Schaffer, and Yi-Kuo Yu (2005) "Protein database searches
using compositionally adjusted substitution matrices", FEBS J. 272:5101-5109.

Query= gi|70778726|ref|NP_067474.2| WD repeat domain 8 [Mus musculus]
         (462 letters)

Database: zebrafish.protein.faa 
           27,291 sequences; 14,382,362 total letters

Searching..................................................done



                                                                 Score    E
Sequences producing significant alignments:                      (bits) Value

ref|NP_956187.1| WD repeat-containing protein 8 [Danio rerio]         668   0.0  
ref|XP_003200833.1| PREDICTED: WD repeat-containing protein 20-l...    48   1e-05
ref|NP_001032508.1| transcription initiation factor TFIID subuni...    46   7e-05
ref|XP_001919110.3| PREDICTED: echinoderm microtubule associated...    44   2e-04
ref|XP_003200333.1| PREDICTED: WD repeat-containing protein 20 [...    42   8e-04
ref|NP_998228.1| WD repeat-containing protein 24 [Danio rerio]         41   0.002
ref|XP_003200416.1| PREDICTED: wu:fb77g05 isoform 2 [Danio rerio]      40   0.003
ref|XP_698531.3| PREDICTED: wu:fb77g05 isoform 3 [Danio rerio]         40   0.003
ref|XP_003200415.1| PREDICTED: wu:fb77g05 isoform 1 [Danio rerio]      40   0.003
ref|NP_001003758.1| THO complex subunit 3 [Danio rerio]                40   0.003
ref|NP_571855.1| protein groucho-1 [Danio rerio]                       39   0.010
ref|NP_571087.1| protein groucho-2 [Danio rerio]                       39   0.010
ref|NP_957147.1| WD repeat-containing protein 61 [Danio rerio]         38   0.015
ref|NP_001188350.1| WD repeat-containing protein 47-like [Danio ...    38   0.017
ref|NP_958875.1| pre-mRNA-processing factor 19 [Danio rerio]           38   0.020
ref|NP_001159740.1| WD repeat-containing protein 82 [Danio rerio]      37   0.023
ref|NP_001018418.1| notchless protein homolog 1 [Danio rerio]          37   0.031
ref|NP_998245.1| cell division cycle protein 20 homolog [Danio r...    37   0.043
ref|XP_001922619.3| PREDICTED: WD repeat-containing protein 47 [...    36   0.067
ref|NP_956547.1| fizzy-related protein homolog [Danio rerio]           36   0.069
ref|NP_998214.1| POC1 centriolar protein homolog A [Danio rerio]       36   0.075
ref|NP_998183.1| katanin p80 WD40-containing subunit B1 [Danio r...    35   0.099
ref|XP_001921656.3| PREDICTED: WD repeat-containing protein 26-l...    35   0.12 
ref|NP_956441.2| probable cytosolic iron-sulfur protein assembly...    35   0.13 
ref|NP_571683.1| apoptotic protease-activating factor 1 isoform ...    35   0.17 
ref|NP_001124076.1| transducin-like enhancer of split 2 [Danio r...    34   0.19 
ref|XP_002662201.1| PREDICTED: u4/U6 small nuclear ribonucleopro...    34   0.21 
ref|NP_956049.1| U4/U6 small nuclear ribonucleoprotein Prp4 [Dan...    34   0.21 
ref|NP_998212.1| periodic tryptophan protein 2 homolog [Danio re...    34   0.23 
ref|NP_001017766.1| polycomb protein eed [Danio rerio]                 34   0.24 
ref|NP_998264.1| WD repeat-containing protein 5 [Danio rerio]          33   0.32 
ref|XP_700740.5| PREDICTED: WD repeat-containing protein 20-like...    33   0.33 
ref|XP_692471.3| PREDICTED: transducin-like enhancer protein 1 [...    33   0.34 
ref|NP_001189371.1| WD repeat-containing protein 26 [Danio rerio]      33   0.40 
ref|NP_001008625.1| intraflagellar transport protein 80 homolog ...    33   0.45 
ref|NP_001181857.1| WD repeat-containing protein 33 isoform 1 [D...    33   0.49 
ref|NP_998177.1| chromatin assembly factor 1 subunit B [Danio re...    32   0.73 
ref|NP_991272.1| mitotic checkpoint protein BUB3 [Danio rerio]         32   1.1  
ref|NP_998321.1| actin-related protein 2/3 complex subunit 1B [D...    32   1.1  
ref|XP_696012.3| PREDICTED: sperm-associated antigen 5 [Danio re...    32   1.2  
ref|NP_001076463.1| F-box-like/WD repeat-containing protein TBL1...    31   1.5  
ref|XP_001345204.3| PREDICTED: WD repeat-containing protein 65-l...    31   1.9  
ref|NP_956412.1| POC1 centriolar protein homolog B [Danio rerio]       31   2.3  
ref|NP_001071237.1| WD repeat, SAM and U-box domain-containing p...    31   2.3  
ref|XP_001342078.1| PREDICTED: probable G-protein coupled recept...    31   2.5  
ref|XP_002663392.1| PREDICTED: uncharacterized protein C3orf38 [...    30   2.6  
ref|NP_001007403.1| transducin beta-like protein 3 [Danio rerio]       30   3.8  
ref|NP_958500.1| actin-related protein 2/3 complex subunit 1A [D...    30   5.6  
ref|NP_001002100.1| actin-related protein 2/3 complex subunit 1A...    29   6.2  
ref|XP_682841.5| PREDICTED: coronin-2A-like [Danio rerio]              29   7.7  
ref|NP_001018176.1| WD repeat-containing protein 43 [Danio rerio]      29   8.1  
ref|NP_001002312.1| intraflagellar transport protein 172 homolog...    29   9.2  

&gt;ref|NP_956187.1| WD repeat-containing protein 8 [Danio rerio]
          Length = 446

 Score =  668 bits (1723), Expect = 0.0,   Method: Compositional matrix adjust.
 Identities = 309/452 (68%), Positives = 372/452 (82%), Gaps = 11/452 (2%)

Query: 1   MNFSESFKLSGLLCRFSPDGKYLASCVQYRLVIRDVTTLQILQLYTCLDQIQHIEWSADS 60
           MNFSE FK S  LC+ SPDGKYLA+CVQYRLV+RD+ TLQI+ LYTCLDQ+ H+EWS+DS
Sbjct: 1   MNFSEVFKQSNQLCKVSPDGKYLATCVQYRLVVRDIGTLQIVHLYTCLDQVMHMEWSSDS 60

Query: 61  LFILCAMYRRGLVQVWSLEQPEWHCKIDEGSAGLVASCWSPDGRHILNTTEFHLRITVWS 120
           LFILCA+Y+RGLVQVWSLEQP+WHCKIDEGS GLV+S WSPDGRHILNTTEF+LRITVWS
Sbjct: 61  LFILCAVYKRGLVQVWSLEQPDWHCKIDEGSIGLVSSRWSPDGRHILNTTEFNLRITVWS 120

Query: 121 LCTKSVSYIKYPKACQQGLTFTRDGRYLALAERRDCRDYVSIFVCSDWQLLRHFDTDTQD 180
           LCTKSVSYIKYPKACQ+G+ FT DGRY+ALAERRDC+DY+S+FVC DW LLRHF+++TQD
Sbjct: 121 LCTKSVSYIKYPKACQKGMDFTADGRYMALAERRDCKDYISVFVCDDWHLLRHFESETQD 180

Query: 181 LTGIEWAPNGCVLAAWDTCLEYKVLLYSLDGRLLSAYCAYEWSLGIKSVAWSPSSQFLAI 240
           L G+EW+PNGCVLA WD CLEYK+LLYSLDGRLLS Y AYEWSLGIKSVAWSPSSQFLAI
Sbjct: 181 LAGLEWSPNGCVLAVWDNCLEYKILLYSLDGRLLSFYSAYEWSLGIKSVAWSPSSQFLAI 240

Query: 241 GSYDGKVRLLNHVTWKMITEFGHPATINNPKTVVYKEAEKSPL----LGLGHLSFPPPRA 296
           GSYD KVR+LNH+TWK ITEF HPATI N K VV+KE EK P+    L +  L+      
Sbjct: 241 GSYDEKVRILNHITWKKITEFEHPATITNSKAVVFKEVEKRPVVSEDLSIRQLT------ 294

Query: 297 MAGALSTSESKYEIASGPVSLQTLKPVADRANPRMGVGMLAFSSDSYFLASRNDNVPNAV 356
           +  AL +++SKYEI   PV +  +KP  +RANP++G+  +AFS+D+ +LA++NDN+P ++
Sbjct: 295 VDNALFSTQSKYEITQLPVQVPVVKPDPERANPKIGISAVAFSADNRYLATKNDNMPQSL 354

Query: 357 WIWDIQKLKLFVVLEHMSPVRSFQWDPQQPRLAICTGGSKVYLWSPAGCVSVQVPGEGDF 416
           W+WD+QK  L  VLE  +PVR F WDP  PRLA+CTG +K+Y+WSPAGC+SV VP EG F
Sbjct: 355 WVWDMQKFSLLAVLEQTAPVRCFVWDPHLPRLALCTGNTKLYMWSPAGCISVTVPVEG-F 413

Query: 417 PVLGLCWHLSGDSLALLSKDHFCLCFLETKER 448
            V  L WH +G SL LLSKD  CLC+ + +E+
Sbjct: 414 QVQSLFWHCTGSSLVLLSKDQLCLCYTDAEEQ 445


&gt;ref|XP_003200833.1| PREDICTED: WD repeat-containing protein 20-like [Danio rerio]
          Length = 572

 Score = 48.1 bits (113), Expect = 1e-05,   Method: Compositional matrix adjust.
 Identities = 26/76 (34%), Positives = 39/76 (51%), Gaps = 2/76 (2%)

Query: 51  IQHIEWSADSLFILCAMYRRGLVQVWSLEQPEWHCKIDEGSAGLVASCWSPDGRHILNTT 110
           +    +S D  F+ CA  + G ++V+  +  E H  +     GL+  CWSPDGR+I+   
Sbjct: 223 VNEFAFSPDGKFLACAG-QDGFLRVFCFDSAELHGTMKSYFGGLLCVCWSPDGRYIVTGG 281

Query: 111 EFHLRITVWSLCTKSV 126
           E  L +TVWS     V
Sbjct: 282 EDDL-VTVWSFSDSRV 296


&gt;ref|NP_001032508.1| transcription initiation factor TFIID subunit 5 [Danio rerio]
          Length = 743

 Score = 45.8 bits (107), Expect = 7e-05,   Method: Compositional matrix adjust.
 Identities = 61/197 (30%), Positives = 92/197 (46%), Gaps = 20/197 (10%)

Query: 79  EQPEWHCKIDEGSAGLV-ASCWSPDGRHILNTTEFHLRITVWSLCTKS--VSYI--KYPK 133
           E+     KI  G +G V    +SPD  ++L+++E    I +WSL T +  V Y    YP 
Sbjct: 473 EKTSSESKILHGHSGPVYGVSFSPDRNYLLSSSE-DGTIRLWSLQTFTCLVGYKGHNYPV 531

Query: 134 ACQQGLTFTRDGRYLALAERRDCRDYVSIFVCSD-WQLLRHFDTDTQDLTGIEWAPNGCV 192
              Q   F+  G Y          D V+    +D +Q LR F     D+T   + PN   
Sbjct: 532 WDTQ---FSPFGYYFVSG----GHDRVARLWATDHYQPLRIFAGHLADVTCTRFHPNSNY 584

Query: 193 LAAWDTCLEYKVLLYS-LDGRLLSAYCAYEWSLGIKSVAWSPSSQFLAIGSYDGKVRLLN 251
           +A   +  +  V L+  L+G  +  +  ++    I S+A+SP+ +FLA GS DG+V L +
Sbjct: 585 VATGSS--DRTVRLWDVLNGNCVRIFTGHKGP--IHSLAFSPNGKFLASGSTDGRVLLWD 640

Query: 252 HVTWKMITEF-GHPATI 267
                MI E  GH  TI
Sbjct: 641 IGHGLMIAELKGHTGTI 657


&gt;ref|XP_001919110.3| PREDICTED: echinoderm microtubule associated protein like 6 [Danio
           rerio]
          Length = 1734

 Score = 44.3 bits (103), Expect = 2e-04,   Method: Compositional matrix adjust.
 Identities = 40/155 (25%), Positives = 57/155 (36%), Gaps = 11/155 (7%)

Query: 273 VVYKEAEKSPLLGLGH------LSFPPPRAMAGALSTSESKYEIASGPVSLQTLKPVADR 326
           VVY   E +    LGH      L+  P +         +  Y       +L T+  + D 
Sbjct: 45  VVYNTREHTQKFYLGHNDDIISLALHPDKIQVATGQVGKDPYICVWDSYALTTVSILRDV 104

Query: 327 ANPRMGVGMLAFSSDSYFLASRNDNVPNAVWIWDIQKLKLFVVLE-HMSPVRSFQWDPQQ 385
                GV  LAF +D   LAS   +  N V +WD +K ++      H   +    WDP  
Sbjct: 105 HT--HGVACLAFDADGQRLASVGLDAKNTVCVWDWKKGRVLATATGHSDRIFDISWDPFL 162

Query: 386 PRLAICTGGSKVYLWSPAGCVSVQVPGEGDFPVLG 420
           P   +  G   +  W  A C +   P  G F   G
Sbjct: 163 PHRLVSCGVKHIKFW--ALCGNALTPKRGIFGKTG 195


&gt;ref|XP_003200333.1| PREDICTED: WD repeat-containing protein 20 [Danio rerio]
          Length = 596

 Score = 42.4 bits (98), Expect = 8e-04,   Method: Compositional matrix adjust.
 Identities = 23/71 (32%), Positives = 38/71 (53%), Gaps = 2/71 (2%)

Query: 51  IQHIEWSADSLFILCAMYRRGLVQVWSLEQPEWHCKIDEGSAGLVASCWSPDGRHILNTT 110
           +    +S D  F L  + + G ++V++ +  E H  +     GL+  CWSPDG++I+   
Sbjct: 226 LNEFAFSPDGKF-LAVVSQDGFLRVFNFDAVELHGTMKSYFGGLLCVCWSPDGKYIVAGG 284

Query: 111 EFHLRITVWSL 121
           E  L +TVWS 
Sbjct: 285 EDDL-VTVWSF 294



 Score = 29.3 bits (64), Expect = 6.2,   Method: Compositional matrix adjust.
 Identities = 13/34 (38%), Positives = 22/34 (64%), Gaps = 3/34 (8%)

Query: 221 EWSLG---IKSVAWSPSSQFLAIGSYDGKVRLLN 251
           +W++G   +   A+SP  +FLA+ S DG +R+ N
Sbjct: 218 KWTVGDGALNEFAFSPDGKFLAVVSQDGFLRVFN 251


&gt;ref|NP_998228.1| WD repeat-containing protein 24 [Danio rerio]
          Length = 779

 Score = 40.8 bits (94), Expect = 0.002,   Method: Compositional matrix adjust.
 Identities = 29/107 (27%), Positives = 46/107 (42%), Gaps = 7/107 (6%)

Query: 49  DQIQHIEWSADSLFILCAMYRRGLVQVWSLEQPEWHCKIDEGSAGLVASC-WSPDGRHIL 107
           + ++ +++S    F   A +  G VQ+W + +P+ + ++     G V  C W P+ R  L
Sbjct: 158 ESVRDVQFSMKDYFTFAASFENGNVQLWDIRRPDRYERMFTAHTGPVFCCDWHPEDRGWL 217

Query: 108 NTTEFHLRITVWSLCTKSVSYIKYPKACQQGLTFTRDGRYLALAERR 154
            T      + VW + T  V  I     C Q  TF    R     ERR
Sbjct: 218 ATGGRDKMVKVWDMSTNRVKEI----YCVQ--TFASVARVKWRPERR 258


&gt;ref|XP_003200416.1| PREDICTED: wu:fb77g05 isoform 2 [Danio rerio]
          Length = 1959

 Score = 40.4 bits (93), Expect = 0.003,   Method: Compositional matrix adjust.
 Identities = 40/156 (25%), Positives = 58/156 (37%), Gaps = 13/156 (8%)

Query: 273 VVYKEAEKSPLLGLGH------LSFPPPRAMAGALSTSESKYEIASGPVSLQTLKPVADR 326
           VVY   E      LGH      L+  P R +       +  Y       ++QT+  + D 
Sbjct: 45  VVYNTREHKQKFYLGHNDDIISLALHPERVLVATGQVGKEPYICVWDSYTVQTVSILRDM 104

Query: 327 ANPRMGVGMLAFSSDSYFLASRNDNVPNAVWIWDIQKLKLFVVLE-HMSPVRSFQWDP-Q 384
                G+  LAF  D   L S   +  N + +WD ++ K+      H   +    WD  Q
Sbjct: 105 HT--HGIACLAFDLDGQCLVSVGLDSKNTICVWDWRQGKVLAAAPGHTDRIFDISWDLYQ 162

Query: 385 QPRLAICTGGSKVYLWSPAGCVSVQVPGEGDFPVLG 420
           Q +L  C G   +  WS   C +   P  G F   G
Sbjct: 163 QSKLVSC-GVKHIKFWSL--CGNALTPKRGVFGKTG 195


&gt;ref|XP_698531.3| PREDICTED: wu:fb77g05 isoform 3 [Danio rerio]
          Length = 1974

 Score = 40.4 bits (93), Expect = 0.003,   Method: Compositional matrix adjust.
 Identities = 40/156 (25%), Positives = 58/156 (37%), Gaps = 13/156 (8%)

Query: 273 VVYKEAEKSPLLGLGH------LSFPPPRAMAGALSTSESKYEIASGPVSLQTLKPVADR 326
           VVY   E      LGH      L+  P R +       +  Y       ++QT+  + D 
Sbjct: 45  VVYNTREHKQKFYLGHNDDIISLALHPERVLVATGQVGKEPYICVWDSYTVQTVSILRDM 104

Query: 327 ANPRMGVGMLAFSSDSYFLASRNDNVPNAVWIWDIQKLKLFVVLE-HMSPVRSFQWDP-Q 384
                G+  LAF  D   L S   +  N + +WD ++ K+      H   +    WD  Q
Sbjct: 105 HT--HGIACLAFDLDGQCLVSVGLDSKNTICVWDWRQGKVLAAAPGHTDRIFDISWDLYQ 162

Query: 385 QPRLAICTGGSKVYLWSPAGCVSVQVPGEGDFPVLG 420
           Q +L  C G   +  WS   C +   P  G F   G
Sbjct: 163 QSKLVSC-GVKHIKFWSL--CGNALTPKRGVFGKTG 195


&gt;ref|XP_003200415.1| PREDICTED: wu:fb77g05 isoform 1 [Danio rerio]
          Length = 1966

 Score = 40.4 bits (93), Expect = 0.003,   Method: Compositional matrix adjust.
 Identities = 40/156 (25%), Positives = 58/156 (37%), Gaps = 13/156 (8%)

Query: 273 VVYKEAEKSPLLGLGH------LSFPPPRAMAGALSTSESKYEIASGPVSLQTLKPVADR 326
           VVY   E      LGH      L+  P R +       +  Y       ++QT+  + D 
Sbjct: 45  VVYNTREHKQKFYLGHNDDIISLALHPERVLVATGQVGKEPYICVWDSYTVQTVSILRDM 104

Query: 327 ANPRMGVGMLAFSSDSYFLASRNDNVPNAVWIWDIQKLKLFVVLE-HMSPVRSFQWDP-Q 384
                G+  LAF  D   L S   +  N + +WD ++ K+      H   +    WD  Q
Sbjct: 105 HT--HGIACLAFDLDGQCLVSVGLDSKNTICVWDWRQGKVLAAAPGHTDRIFDISWDLYQ 162

Query: 385 QPRLAICTGGSKVYLWSPAGCVSVQVPGEGDFPVLG 420
           Q +L  C G   +  WS   C +   P  G F   G
Sbjct: 163 QSKLVSC-GVKHIKFWSL--CGNALTPKRGVFGKTG 195


&gt;ref|NP_001003758.1| THO complex subunit 3 [Danio rerio]
          Length = 322

 Score = 40.4 bits (93), Expect = 0.003,   Method: Compositional matrix adjust.
 Identities = 60/257 (23%), Positives = 104/257 (40%), Gaps = 40/257 (15%)

Query: 223 SLGIKSVAWSPSSQFLAIGSYDGKVRLLNHVTWKMITEF---GHPATIN-------NPKT 272
           S  + SVAWS   + LA GS+D    +      +++ E    GH  +++       NP  
Sbjct: 26  SAKVHSVAWSCDGKRLASGSFDKTASVFVLEKDRLVKENNYRGHGDSVDQLCWHPTNPDL 85

Query: 273 VVYKEAEKSPLLGLGHLSFPPPRAMAGALSTSESKYEIASGP----------------VS 316
            V    +K+  +          + MA  +ST      I   P                + 
Sbjct: 86  FVTASGDKTIRI----WDVRTTKCMA-TVSTKGENINICWSPDGQTIAVGNKDDVVTFID 140

Query: 317 LQTLKPVADRANPRMGVGMLAFSSDS--YFLASRNDNVPNAVWIWDIQKLKLFVVLE-HM 373
            +T +P A+    +  V  +++++D+  +FL + N      + I    +LKL   +  H 
Sbjct: 141 AKTHRPRAEE-QFKFEVNEISWNNDNDMFFLTNGN----GCINILSYPELKLIQSINAHP 195

Query: 374 SPVRSFQWDPQQPRLAICTGGSKVYLWSPAGCVSVQVPGEGDFPVLGLCWHLSGDSLALL 433
           S     ++DP     A  +  + V LW+    V V+     D+PV  L +   G  LA  
Sbjct: 196 SNCICIKFDPTGKYFATGSADALVSLWNVEELVCVRCFSRLDWPVRTLSFSHDGKMLASA 255

Query: 434 SKDHFC-LCFLETKERV 449
           S+DHF  +  +ET E++
Sbjct: 256 SEDHFIDIAEVETGEKL 272



 Score = 35.8 bits (81), Expect = 0.077,   Method: Compositional matrix adjust.
 Identities = 34/128 (26%), Positives = 57/128 (44%), Gaps = 10/128 (7%)

Query: 333 VGMLAFSSDSYFLASRNDNVPNAVWIWDIQKL-KLFVVLEHMSPVRSFQWDPQQPRLAIC 391
           V  +A+S D   LAS + +   +V++ +  +L K      H   V    W P  P L + 
Sbjct: 29  VHSVAWSCDGKRLASGSFDKTASVFVLEKDRLVKENNYRGHGDSVDQLCWHPTNPDLFVT 88

Query: 392 TGGSK-VYLWS--PAGCV-SVQVPGEGDFPVLGLCWHLSGDSLALLSKDHFCLCFLETKE 447
             G K + +W      C+ +V   GE     + +CW   G ++A+ +KD   + F++ K 
Sbjct: 89  ASGDKTIRIWDVRTTKCMATVSTKGEN----INICWSPDGQTIAVGNKDDV-VTFIDAKT 143

Query: 448 RVGTAYEQ 455
               A EQ
Sbjct: 144 HRPRAEEQ 151


&gt;ref|NP_571855.1| protein groucho-1 [Danio rerio]
          Length = 771

 Score = 38.5 bits (88), Expect = 0.010,   Method: Compositional matrix adjust.
 Identities = 49/200 (24%), Positives = 86/200 (43%), Gaps = 32/200 (16%)

Query: 70  RGLVQVWSLEQPEWHCKIDE----GSAGLVASC-WSPDGRHILNTTEFHLRITVWSLCTK 124
           +G V++W + QP     + +         + SC   PDGR ++   E    +T+W L ++
Sbjct: 505 KGCVKIWDISQPGSKSPVSQLDCLNRDNYIRSCKLLPDGRTLIVGGEAST-LTIWDLASQ 563

Query: 125 SVSYIKYPKACQQGLTFTRDGRYLALAERRDCRDYVSIFVCSDWQ----------LLRHF 174
           +          +  LT +    Y ALA   D +  V    CSD            L+R F
Sbjct: 564 T-------PRIKAELTSSAPACY-ALAISPDAK--VCFSCCSDGNIAVWDLHNQTLVRQF 613

Query: 175 DTDTQDLTGIEWAPNGCVLAAWDTCLEYKVLLYSL-DGRLLSAYCAYEWSLGIKSVAWSP 233
              T   + I+ + +G  L  W   L+  V  + L +GR L  +   +++  I S+ + P
Sbjct: 614 QGHTDGASCIDISHDGTKL--WTGGLDNTVRSWDLREGRQLQQH---DFTSQIFSLGYCP 668

Query: 234 SSQFLAIGSYDGKVRLLNHV 253
           + ++LA+G     V +L+H 
Sbjct: 669 TGEWLAVGMESSNVEVLHHT 688


&gt;ref|NP_571087.1| protein groucho-2 [Danio rerio]
          Length = 761

 Score = 38.5 bits (88), Expect = 0.010,   Method: Compositional matrix adjust.
 Identities = 50/200 (25%), Positives = 88/200 (44%), Gaps = 32/200 (16%)

Query: 70  RGLVQVWSLEQPEWHCKIDE----GSAGLVASC-WSPDGRHILNTTEFHLRITVWSLCTK 124
           +G V++W + QP     + +         + SC   PDGR ++   E    +T+W L ++
Sbjct: 495 KGCVKIWDISQPGSKSPVSQLDCLNRDNYIRSCKLLPDGRTLIVGGEAST-LTIWDLASQ 553

Query: 125 SVSYIKYPKACQQGLTFTRDGRYLALAERRDCRDYVSIFVCSDWQ----------LLRHF 174
           +      P+  +  LT +    Y ALA   D +  V    CSD            L+R F
Sbjct: 554 T------PR-IKAELTSSAPACY-ALAISPDAK--VCFSCCSDGNIAVWDLHNQTLVRQF 603

Query: 175 DTDTQDLTGIEWAPNGCVLAAWDTCLEYKVLLYSL-DGRLLSAYCAYEWSLGIKSVAWSP 233
              T   + I+ + +G  L  W   L+  V  + L +GR L  +   +++  I S+ + P
Sbjct: 604 QGHTDGASCIDISHDGTKL--WTGGLDNTVRSWDLREGRQLQQH---DFASQIFSLGYCP 658

Query: 234 SSQFLAIGSYDGKVRLLNHV 253
           + ++LA+G     V +L+H 
Sbjct: 659 TGEWLAVGMESSNVEVLHHT 678


&gt;ref|NP_957147.1| WD repeat-containing protein 61 [Danio rerio]
          Length = 305

 Score = 38.1 bits (87), Expect = 0.015,   Method: Compositional matrix adjust.
 Identities = 56/233 (24%), Positives = 104/233 (44%), Gaps = 35/233 (15%)

Query: 181 LTGIEWAPNGCVLAAWDTCLEYKVLLYSLD-GRLLSAYCAYEWSLGIKSVAWSPSSQFLA 239
           +  +  + NG + A+  + L+  + L+ L+ G+ + +  A    +   +VA+SP S+++A
Sbjct: 67  VVSVNISQNGAIAAS--SSLDAHIRLWDLETGKQIKSMDAGP--VDAWTVAFSPDSKYIA 122

Query: 240 IGSYDGKVRLLNHVTWKMITEFGHPATINNPKTVVYKEAEKSPLLGLGHLSFPPPRAMA- 298
            GS+ GKV +           FG    + + K     +     +L + +   P  + +A 
Sbjct: 123 TGSHLGKVNI-----------FG----VESGKKEHSLDTRGKFILSIAY--SPDGKYLAS 165

Query: 299 GALSTSESKYEIASGPVSLQTLKPVADRANPRMGVGMLAFSSDSYFLASRNDNVPNAVWI 358
           GA+    + ++IA+G + L TL+  A      M +  L FS DS  L + +D+    + I
Sbjct: 166 GAIDGIINIFDIATGKL-LHTLEGHA------MPIRSLTFSPDSQLLVTASDD--GYIKI 216

Query: 359 WDIQKLKLFVVLE-HMSPVRSFQWDPQQPRLAICTGGSKVYLW--SPAGCVSV 408
           +D+Q   L   L  H S V S  + P        +    + +W  S   CV+ 
Sbjct: 217 YDVQHANLAGTLSGHGSWVLSVAFSPDDTHFVSSSSDKSIKVWDTSSRSCVNT 269



 Score = 37.0 bits (84), Expect = 0.028,   Method: Compositional matrix adjust.
 Identities = 29/116 (25%), Positives = 58/116 (50%), Gaps = 4/116 (3%)

Query: 12  LLCRFSPDGKYLAS-CVQYRLVIRDVTTLQILQ-LYTCLDQIQHIEWSADSLFILCAMYR 69
           L   +SPDGKYLAS  +   + I D+ T ++L  L      I+ + +S DS  ++ A   
Sbjct: 152 LSIAYSPDGKYLASGAIDGIINIFDIATGKLLHTLEGHAMPIRSLTFSPDSQLLVTAS-D 210

Query: 70  RGLVQVWSLEQPEWHCKIDEGSAGLVASCWSPDGRHILNTTEFHLRITVWSLCTKS 125
            G ++++ ++       +    + +++  +SPD  H ++++     I VW   ++S
Sbjct: 211 DGYIKIYDVQHANLAGTLSGHGSWVLSVAFSPDDTHFVSSSS-DKSIKVWDTSSRS 265



 Score = 31.2 bits (69), Expect = 1.9,   Method: Compositional matrix adjust.
 Identities = 38/182 (20%), Positives = 79/182 (43%), Gaps = 38/182 (20%)

Query: 226 IKSVAWSPSSQFLAIGSYDGKVRLLNHVTWKMI-TEFGHPATINNPKTVVYKEAEKSPLL 284
           I S+A+SP  ++LA G+ DG + + +  T K++ T  GH   I +               
Sbjct: 151 ILSIAYSPDGKYLASGAIDGIINIFDIATGKLLHTLEGHAMPIRS--------------- 195

Query: 285 GLGHLSFPPPRAMAGALSTSESKYEIASGPVSLQTLKPVADRANPRMGVG----MLAFS- 339
               L+F P   +   ++ S+  Y      + +  ++  A+ A    G G     +AFS 
Sbjct: 196 ----LTFSPDSQL--LVTASDDGY------IKIYDVQH-ANLAGTLSGHGSWVLSVAFSP 242

Query: 340 SDSYFLASRNDNVPNAVWIWDIQKLK-LFVVLEHMSPVRSFQWDPQQPRLAICTGGSKVY 398
            D++F++S +D    ++ +WD      +    +H   V S +++P   ++        ++
Sbjct: 243 DDTHFVSSSSD---KSIKVWDTSSRSCVNTFFDHQDQVWSVKYNPTGSKIVSAGDDRAIH 299

Query: 399 LW 400
           ++
Sbjct: 300 IY 301


&gt;ref|NP_001188350.1| WD repeat-containing protein 47-like [Danio rerio]
          Length = 871

 Score = 37.7 bits (86), Expect = 0.017,   Method: Compositional matrix adjust.
 Identities = 32/95 (33%), Positives = 50/95 (52%), Gaps = 16/95 (16%)

Query: 188 PNGCVLAAWDTCLEY-KVLLYSL-DGRLLSAYCAYEWSLGIKSVAWSPSSQFLAIGSYDG 245
           P+G ++A   T LE  + +LY +  GR +  Y  +  S  ++SV +SP + +L  GSYD 
Sbjct: 762 PSGRLMA---TGLEDCRCMLYDIRGGRSVQVYRPH--SSDVRSVRFSPGAHYLLTGSYDS 816

Query: 246 KVRLLNHVTWKMITEFGHPATINNPKTVVYKEAEK 280
           +V         MI++     T N P TVV + A+K
Sbjct: 817 RV---------MISDLQGDLTKNLPVTVVAEHADK 842


&gt;ref|NP_958875.1| pre-mRNA-processing factor 19 [Danio rerio]
          Length = 505

 Score = 37.7 bits (86), Expect = 0.020,   Method: Compositional matrix adjust.
 Identities = 24/84 (28%), Positives = 40/84 (47%), Gaps = 1/84 (1%)

Query: 356 VWIWDIQKLKLFVVLE-HMSPVRSFQWDPQQPRLAICTGGSKVYLWSPAGCVSVQVPGEG 414
           V ++D ++ ++   L+ H   V S  + P Q  +   +  S + +WS  G   VQV    
Sbjct: 247 VVVFDRREEQIVATLKGHTKKVSSVIYHPAQAVVFSASSDSTIRVWSVTGGNCVQVVRAH 306

Query: 415 DFPVLGLCWHLSGDSLALLSKDHF 438
           +  V GL  H +GD L   S+D +
Sbjct: 307 ESAVTGLSLHATGDYLLSSSEDQY 330



 Score = 29.6 bits (65), Expect = 5.3,   Method: Compositional matrix adjust.
 Identities = 31/139 (22%), Positives = 61/139 (43%), Gaps = 10/139 (7%)

Query: 73  VQVWSLEQPEWHCKIDEGSAGLVASCWSPDGRHILNTTEFHLRITVWSLCT-KSVSYIKY 131
           +++W L++          S  + A  +S +G + L T      + +W L   K+   I  
Sbjct: 375 IKIWDLKERTNVANFPGHSGPVTAIAFSENG-YYLATGAQDSSLKLWDLRKLKNFKTITL 433

Query: 132 PKACQ-QGLTFTRDGRYLALAERRDCRDYVSIFVCSDWQLLRHFDTDTQDLTGIEWAPNG 190
               + + L F + G YLA+    D R    +++C  W  + +F   +  +TG+ +  + 
Sbjct: 434 DNNYEVKSLVFDQSGTYLAVG-GSDIR----VYICKQWSEVLNFSDHSGLVTGVAFGEHA 488

Query: 191 CVLAAWDTCLEYKVLLYSL 209
             LA+  T ++  +  YSL
Sbjct: 489 QFLAS--TGMDRSLKFYSL 505


&gt;ref|NP_001159740.1| WD repeat-containing protein 82 [Danio rerio]
          Length = 313

 Score = 37.4 bits (85), Expect = 0.023,   Method: Compositional matrix adjust.
 Identities = 22/71 (30%), Positives = 40/71 (56%), Gaps = 2/71 (2%)

Query: 182 TGIEWAPNGCVLAAWDTCLEYKVLLYSLDGRLLSAYCAYEWSLG-IKSVAWSPSSQFLAI 240
           TG++++ +G ++         +VL  +  G +L ++  Y  S G I   +++P SQF+ I
Sbjct: 198 TGLKFSNDGKLILVSTNGGTLRVL-DAFKGAVLHSFGGYNNSKGVILEASFTPDSQFIMI 256

Query: 241 GSYDGKVRLLN 251
           GS DGK+ + N
Sbjct: 257 GSEDGKIHVWN 267


&gt;ref|NP_001018418.1| notchless protein homolog 1 [Danio rerio]
          Length = 476

 Score = 37.0 bits (84), Expect = 0.031,   Method: Compositional matrix adjust.
 Identities = 34/122 (27%), Positives = 51/122 (41%), Gaps = 6/122 (4%)

Query: 321 KPVADRANPRMGVGMLAFSSDSYFLASRNDNVPNAVWIWDIQKLKLFVVLE-HMSPVRSF 379
           KPVA     +  V  + FS D+  +AS   +   ++ IWD +  K    L  H+ PV   
Sbjct: 354 KPVARLTGHQALVNEVLFSPDTRLIASA--SFDKSIKIWDGKTGKYLNSLRGHVGPVYQV 411

Query: 380 QWDPQQPRLAICTGGSKVYLWS-PAGCVSVQVPGEGDFPVLGLCWHLSGDSLALLSKDHF 438
            W      L   +  S + +W    G ++  +PG  D  V  + W   G  +A   KD  
Sbjct: 412 AWSADSRLLVSGSSDSTLKVWDIKTGKLNADLPGHAD-EVFAVDWSPDGQRVASGGKDK- 469

Query: 439 CL 440
           CL
Sbjct: 470 CL 471



 Score = 33.1 bits (74), Expect = 0.47,   Method: Compositional matrix adjust.
 Identities = 25/93 (26%), Positives = 44/93 (47%), Gaps = 3/93 (3%)

Query: 16  FSPDGKYLASC-VQYRLVIRDVTTLQILQ-LYTCLDQIQHIEWSADSLFILCAMYRRGLV 73
           FSPD + +AS      + I D  T + L  L   +  +  + WSADS  ++       L 
Sbjct: 371 FSPDTRLIASASFDKSIKIWDGKTGKYLNSLRGHVGPVYQVAWSADSRLLVSGSSDSTL- 429

Query: 74  QVWSLEQPEWHCKIDEGSAGLVASCWSPDGRHI 106
           +VW ++  + +  +   +  + A  WSPDG+ +
Sbjct: 430 KVWDIKTGKLNADLPGHADEVFAVDWSPDGQRV 462



 Score = 29.3 bits (64), Expect = 5.9,   Method: Compositional matrix adjust.
 Identities = 29/109 (26%), Positives = 43/109 (39%), Gaps = 7/109 (6%)

Query: 336 LAFSSDSYFLASRNDNVPNAVWIWDIQ-KLKLFVVLEHMSPVRSFQWDPQQPRLAICTGG 394
           +AFS    +LAS + +    V  WD+  +        H   V S  W P   +LA     
Sbjct: 111 VAFSPTGKYLASGSGDT--TVRFWDLSTETPHHTSRGHTHWVLSIAWSPDGKKLASGCKN 168

Query: 395 SKVYLWSPAGCVSVQVPGEGDFP-VLGLCW---HLSGDSLALLSKDHFC 439
           S+++LW P     +     G    +  LCW   HL+ +   L S    C
Sbjct: 169 SQIFLWDPVTGKQIGKTLTGHTKWITWLCWEPLHLNPECRYLASTSKDC 217


&gt;ref|NP_998245.1| cell division cycle protein 20 homolog [Danio rerio]
          Length = 496

 Score = 36.6 bits (83), Expect = 0.043,   Method: Compositional matrix adjust.
 Identities = 47/192 (24%), Positives = 77/192 (40%), Gaps = 37/192 (19%)

Query: 226 IKSVAWSPSSQFLAIGSYDGKVRLLNHVTWKMITEF-GHPATINNPKTVVYKEAEKSPLL 284
           I SV+WS    FLAIG+ D KV L +    K +    GH A +                 
Sbjct: 230 ICSVSWSKDGNFLAIGTSDCKVELWDVQYQKRLRSMDGHSARV----------------- 272

Query: 285 GLGHLSFPPPRAMAGALSTSESKYEIASGPVSLQTLKPVADRANPRMGVGMLAFSSDSYF 344
             G LS+      +G+ S    ++++      + T             V  L +S D  +
Sbjct: 273 --GCLSWNDHILSSGSRSGLIHQHDVRVADHHIFTF------GGHTQEVCGLTWSPDGRY 324

Query: 345 LASRNDNVPNAVWIWDI----QKLKLFVVLEHMSPVRSFQWDPQQPRLAICTGGSK---V 397
           LAS  ++  N ++IW +    +   +  + EH   V++  W P QP +    GG+    +
Sbjct: 325 LASGGND--NMMYIWPMTTGSENQAIHALSEHQGAVKALAWCPWQPNIPASGGGTSDRHI 382

Query: 398 YLW--SPAGCVS 407
            +W  S   C+S
Sbjct: 383 RIWNASSGSCIS 394


&gt;ref|XP_001922619.3| PREDICTED: WD repeat-containing protein 47 [Danio rerio]
          Length = 902

 Score = 35.8 bits (81), Expect = 0.067,   Method: Compositional matrix adjust.
 Identities = 28/94 (29%), Positives = 46/94 (48%), Gaps = 14/94 (14%)

Query: 188 PNGCVLAAWDTCLEYKVLLYSL-DGRLLSAYCAYEWSLGIKSVAWSPSSQFLAIGSYDGK 246
           P+G +LA      +   +LY +  GR++  Y  +  S  ++SV +SP + +L  GSYD K
Sbjct: 793 PSGRLLATGQE--DSSCMLYDIRGGRMVQTYKPH--SSDVRSVRFSPGAHYLLTGSYDNK 848

Query: 247 VRLLNHVTWKMITEFGHPATINNPKTVVYKEAEK 280
           V         ++T+     T   P TVV +  +K
Sbjct: 849 V---------IVTDLQGDLTKQLPLTVVGEHGDK 873


&gt;ref|NP_956547.1| fizzy-related protein homolog [Danio rerio]
          Length = 495

 Score = 35.8 bits (81), Expect = 0.069,   Method: Compositional matrix adjust.
 Identities = 39/172 (22%), Positives = 73/172 (42%), Gaps = 28/172 (16%)

Query: 226 IKSVAWSPSSQFLAIGSYDGKVRLLNHVTWKMITEF-GHPATINNPKTVVYKEAEKSPLL 284
           + SV WS     +A+G++ G V++ +    K +    GH A +                 
Sbjct: 234 VTSVGWSERGNLVAVGTHKGFVQIWDATAGKKLFALEGHTARV----------------- 276

Query: 285 GLGHLSFPPPRAMAGALSTSESKYEIASGPVSLQTLKPVADRANPRMGVGMLAFSSDSYF 344
             G L++   +  +G+      + +I + P  LQ+ + +      R  V  L +S+D   
Sbjct: 277 --GALAWNADQLSSGSRDRMILQRDIRTPP--LQSERRLQGH---RQEVCGLKWSTDHQL 329

Query: 345 LASRNDNVPNAVWIWDIQK-LKLFVVLEHMSPVRSFQWDPQQPRLAICTGGS 395
           LAS  ++  N + +W+    L +    EH++ V++  W P Q  L    GG+
Sbjct: 330 LASGGND--NKLLVWNHSSVLPMQQYTEHLAAVKAIAWSPHQHGLLASGGGT 379


&gt;ref|NP_998214.1| POC1 centriolar protein homolog A [Danio rerio]
          Length = 416

 Score = 35.8 bits (81), Expect = 0.075,   Method: Compositional matrix adjust.
 Identities = 62/291 (21%), Positives = 124/291 (42%), Gaps = 30/291 (10%)

Query: 15  RFSPDGKYLASCVQ---YRLVIRDVTTLQILQLYTCLDQIQHIEWSADSLFILCAMYRRG 71
           +FSP    LAS  +    RL +  V    +L        ++ + +SAD   +L A   + 
Sbjct: 67  QFSPSAHLLASSSRDKTVRLWVPSVKGESVL-FRAHTGSVRSVCFSADGQSLLTASDDQS 125

Query: 72  LVQVWSLEQPEWHCKIDEGSAGLVASCWSPDGRHILNTTEFHLRITVWSLCTKSVSYIK- 130
            +++WS+ + +  C + E +  +  + +SPDG+ +++ ++    + +W   ++ + +   
Sbjct: 126 -IKLWSVHRQKIICTLREHNNWVRCARFSPDGQLMVSVSDDRT-VKLWDASSRQLIHTFC 183

Query: 131 YPKACQQGLTFTRDGRYLALAERRDCRDYVSIFVCSDWQLLRHFDTDTQDLTGIEWAPNG 190
            P      + F      +A A      + V ++      LL+H+   +  +  + + P+G
Sbjct: 184 EPGGYSSYVDFHPSSTCIATAS---SDNTVRVWDIRTHTLLQHYQVHSAAVNALSFHPSG 240

Query: 191 CVL--AAWDTCLEYKVLLYSLDGRLLSAYCAYEWSLGIKSVAWSPSSQFLAIGSYDGKV- 247
             L  A+ D+ L+   +L  L+GRLL     ++ S    S + S   QF + GS D +V 
Sbjct: 241 NHLLTASSDSTLK---ILDLLEGRLLYTLHGHQGSASCVSFSRS-GDQFASAGS-DQQVM 295

Query: 248 ------------RLLNHVTWKMITEFGHPATINNPKTVVYKEAEKSPLLGL 286
                       R+L                  +P++   ++ E SPLLG+
Sbjct: 296 VWRTNFDSVDYSRVLQQKRDHRTPSAQASGAAGDPESRSGQKTEVSPLLGV 346


&gt;ref|NP_998183.1| katanin p80 WD40-containing subunit B1 [Danio rerio]
          Length = 694

 Score = 35.4 bits (80), Expect = 0.099,   Method: Compositional matrix adjust.
 Identities = 29/106 (27%), Positives = 52/106 (49%), Gaps = 3/106 (2%)

Query: 16  FSPDGKYLASCVQYRLV-IRDVTTLQILQLYTCLDQIQHIEWSADSLFILCAMYRRGLVQ 74
           FSPDGK+LAS      V + D+   +++  +T      ++     + ++L +      V+
Sbjct: 155 FSPDGKWLASASDDSTVKLWDLIAGKMITEFTSHTSAVNVVQFHPNEYLLASGSADRTVK 214

Query: 75  VWSLEQPEWHCKIDEGSAGLVASC-WSPDGRHILNTTEFHLRITVW 119
           +W LE+        EG  G+V S  ++PDG  + + +E  LR+  W
Sbjct: 215 LWDLEKFNM-IGSSEGETGVVRSVLFNPDGSCLYSGSENTLRVYGW 259



 Score = 34.3 bits (77), Expect = 0.20,   Method: Compositional matrix adjust.
 Identities = 26/104 (25%), Positives = 49/104 (47%), Gaps = 10/104 (9%)

Query: 169 QLLRHFDTDTQDLTGIEWAPNGCVLAAWDTCLEYKVLLYSLDGRLLSAYCAYEW---SLG 225
           ++LR        ++ +++ P G  LA+       K+      G      C + +   +  
Sbjct: 96  KILRTLMGHKASISSLDFHPMGEYLASGSVDSNIKLWDVRRKG------CVFRYKGHTQA 149

Query: 226 IKSVAWSPSSQFLAIGSYDGKVRLLNHVTWKMITEF-GHPATIN 268
           ++ +A+SP  ++LA  S D  V+L + +  KMITEF  H + +N
Sbjct: 150 VRCLAFSPDGKWLASASDDSTVKLWDLIAGKMITEFTSHTSAVN 193



 Score = 33.5 bits (75), Expect = 0.31,   Method: Compositional matrix adjust.
 Identities = 23/66 (34%), Positives = 33/66 (50%), Gaps = 3/66 (4%)

Query: 336 LAFSSDSYFLASRNDNVPNAVWIWDIQKLKLFVVL-EHMSPVRSFQWDPQQPRLAICTGG 394
           LAFS D  +LAS +D+  + V +WD+   K+      H S V   Q+ P +  LA  +  
Sbjct: 153 LAFSPDGKWLASASDD--STVKLWDLIAGKMITEFTSHTSAVNVVQFHPNEYLLASGSAD 210

Query: 395 SKVYLW 400
             V LW
Sbjct: 211 RTVKLW 216


&gt;ref|XP_001921656.3| PREDICTED: WD repeat-containing protein 26-like [Danio rerio]
          Length = 474

 Score = 35.0 bits (79), Expect = 0.12,   Method: Compositional matrix adjust.
 Identities = 25/104 (24%), Positives = 49/104 (47%), Gaps = 7/104 (6%)

Query: 295 RAMAGALSTSESKYEIASGPVSLQTLKPVADRANPRMGVGMLAFSSDSYFLASRNDNVPN 354
           +  +G+  T+   +++      L+ LK +   A    GV  LA+S D  +L +   +  +
Sbjct: 179 KLASGSKDTTVIIWQVDPDSQQLKQLKTLEGHA---YGVSYLAWSPDDTYLIACGPDDCS 235

Query: 355 AVWIWDIQ--KLKLFVVLEHMSPVRSFQWDPQQPRLAICTGGSK 396
            +W+W++Q  +L+  +   H   + S  W+P   R    TGG +
Sbjct: 236 ELWLWNVQTGELRTKMSQSHEDSLTSVAWNPDGKRFV--TGGQR 277


&gt;ref|NP_956441.2| probable cytosolic iron-sulfur protein assembly protein ciao1
           [Danio rerio]
          Length = 330

 Score = 34.7 bits (78), Expect = 0.13,   Method: Compositional matrix adjust.
 Identities = 31/121 (25%), Positives = 51/121 (42%), Gaps = 8/121 (6%)

Query: 333 VGMLAFSSDSYFLASRNDNVPNAVWIWDIQKLKLFVVLE-HMSPVRSFQWDPQQPRLAIC 391
           V  +A+S    +LAS + +    +W    +  +   VLE H + V+   W P    LA C
Sbjct: 64  VRKVAWSPCGKYLASASFDATTCIWKKTDEDFECLTVLEGHENEVKCVAWAPSGSLLATC 123

Query: 392 TGGSKVYLW-----SPAGCVSVQVPGEGDFPVLGLCWHLSGDSLALLSKDHFCLCFLETK 446
           +    V++W         C+SV      D  V  + WH + + LA  S D+    + E  
Sbjct: 124 SRDKSVWIWEVDEEDEYECLSVVNSHTQD--VKHVVWHPTQELLASASYDNKICIYKEED 181

Query: 447 E 447
           +
Sbjct: 182 D 182



 Score = 34.7 bits (78), Expect = 0.15,   Method: Compositional matrix adjust.
 Identities = 31/108 (28%), Positives = 51/108 (47%), Gaps = 11/108 (10%)

Query: 145 GRYLALAERRDCRDYVSIFVCSDWQLLRHFDTDTQDLTGIEWAPNGCVLAAWDTCLEYK- 203
           G+YLA A   D    +      D++ L   +    ++  + WAP+G +LA   TC   K 
Sbjct: 73  GKYLASASF-DATTCIWKKTDEDFECLTVLEGHENEVKCVAWAPSGSLLA---TCSRDKS 128

Query: 204 VLLYSLDGR----LLSAYCAYEWSLGIKSVAWSPSSQFLAIGSYDGKV 247
           V ++ +D       LS   ++  +  +K V W P+ + LA  SYD K+
Sbjct: 129 VWIWEVDEEDEYECLSVVNSH--TQDVKHVVWHPTQELLASASYDNKI 174


&gt;ref|NP_571683.1| apoptotic protease-activating factor 1 isoform 2 [Danio rerio]
          Length = 1261

 Score = 34.7 bits (78), Expect = 0.17,   Method: Compositional matrix adjust.
 Identities = 89/387 (22%), Positives = 149/387 (38%), Gaps = 72/387 (18%)

Query: 56   WSADSLFILCAMYRRGLVQVWSLEQPEWHCKIDEGSAGLVASCWSPDGRHILNTTEFHLR 115
            WSAD   I+CA   R  V V+ +E  +   K+       +  C +     +L     H  
Sbjct: 808  WSADGSQIICAA--RNTVFVFDVETSDLLLKLKTSRLSTIQFCHACPNSSLLAVALSHYT 865

Query: 116  ITVWSL--------CTKSVSYIKYPKACQQGLTFTRDGRYLALAERRDCRDYVSIFVCSD 167
            + +W+         C+  +S++     C Q   F+ DG  L                 SD
Sbjct: 866  VELWNFESSKKKAECSGHLSWVH----CVQ---FSPDGSLL--------------LSSSD 904

Query: 168  WQLLRHFDTDT---------QDLTGIEWAPNGCVLAAWDTCLEYKVLLYSLDGRLLSAYC 218
             Q +R ++TD          +  T +  + +   + A D+    +VL  S    +L +  
Sbjct: 905  DQTIRLWETDRVHTSSAVALKRDTDVLSSHSDATIIAPDSSNRLQVLSGSTGAVVLESE- 963

Query: 219  AYEWSLGIKSVAWSPSSQFLAIGSYDGKVRLLNHVTWKMITEF-GHPATINNPK-----T 272
              E S  I+    S ++ F+A+GS DG V+++   + K   +  GH  T+++ +      
Sbjct: 964  --ELSSRIRCSCISRNAAFVALGSEDGTVQVIEVPSSKASVKLSGHTKTVHHCQFTDDCE 1021

Query: 273  VVYKEAEKSP-----------LLGLGHLSFPPPRAMAGALSTSESKYEIAS--GPVSLQT 319
            ++   +E S            ++  GH+   P R      S+S       S  G V +  
Sbjct: 1022 ILITSSEDSTIRVWKWRTGECMVLQGHME--PVRKFHLLSSSSSPHLFSWSFDGTVKVWD 1079

Query: 320  L---KPVADRANPRMGVGMLAFSSDSYFLASRNDNVPNAVWIWDIQKLKLFVVLEHMSPV 376
            L   + + D       V     SSD    A+ + N    VW     K+ LF++  H   V
Sbjct: 1080 LTRGQMLQDLVCHEGAVLSCDVSSDGRLFATTSANRTAKVWSSASWKM-LFLLEGHKDCV 1138

Query: 377  RS--FQWDPQQPRLAICTGGSKVYLWS 401
            RS  F WD +  RLA      ++ LWS
Sbjct: 1139 RSCRFSWDNK--RLATGDDNGEIRLWS 1163



 Score = 31.6 bits (70), Expect = 1.2,   Method: Compositional matrix adjust.
 Identities = 87/408 (21%), Positives = 161/408 (39%), Gaps = 59/408 (14%)

Query: 12   LLCRFSPDGKYLASCVQYRLV-IRDVTTLQILQLYTCL--DQIQHIEWSADSLFIL---C 65
            L C FSPD +++A+C   R V + +V    +++ +     +QI H +++     +L   C
Sbjct: 663  LCCAFSPDDRHIATCASDRKVKLWNVERGVLIREFEVEHEEQINHCQFTNTGRRVLLATC 722

Query: 66   AMYRRGLVQVWSLEQPEWHCKIDEGSAGLVASCWSPDGRHILNTTE------FHLR---- 115
            +  +    ++W+  +      +      +   C+SP+  ++  ++       F +     
Sbjct: 723  SNDKFTNTRLWNPNKKTSQNTMFGHMEPVNHCCFSPNDLYLATSSSDGSLKLFEVSSANE 782

Query: 116  ---ITVWSLCTKSVSYIKYPKACQQGLTFTRDGRYLALAERRDCRDYVSIFVCSDWQLLR 172
               I V S   +S   IK    C    T++ DG  +  A     R+ V +F      LL 
Sbjct: 783  WKSIDVDSFFPESDEEIKAMVKCS---TWSADGSQIICA----ARNTVFVFDVETSDLL- 834

Query: 173  HFDTDTQDLTGIEWA---PNGCVLAAWDTCLEYKVLLYSLDGRLLSAYCAYEWSLGIKSV 229
                 T  L+ I++    PN  +LA       Y V L++ +     A C+   S  +  V
Sbjct: 835  -LKLKTSRLSTIQFCHACPNSSLLAV--ALSHYTVELWNFESSKKKAECSGHLSW-VHCV 890

Query: 230  AWSPSSQFLAIGSYDGKVRL--------LNHVTWKMITEF---GHPATINNPKTV----V 274
             +SP    L   S D  +RL         + V  K  T+       ATI  P +     V
Sbjct: 891  QFSPDGSLLLSSSDDQTIRLWETDRVHTSSAVALKRDTDVLSSHSDATIIAPDSSNRLQV 950

Query: 275  YKEAEKSPLLGLGHLSFPPPRAMAGALSTSESKYEIASGPVSLQTLKPVADRANPRMG-- 332
               +  + +L    LS    R     +S + +   + S   ++Q ++  + +A+ ++   
Sbjct: 951  LSGSTGAVVLESEELS---SRIRCSCISRNAAFVALGSEDGTVQVIEVPSSKASVKLSGH 1007

Query: 333  ---VGMLAFSSDSYFLASRNDNVPNAVWIWDIQKLKLFVVLEHMSPVR 377
               V    F+ D   L + +++    VW W  +  +  V+  HM PVR
Sbjct: 1008 TKTVHHCQFTDDCEILITSSEDSTIRVWKW--RTGECMVLQGHMEPVR 1053


&gt;ref|NP_001124076.1| transducin-like enhancer of split 2 [Danio rerio]
          Length = 811

 Score = 34.3 bits (77), Expect = 0.19,   Method: Compositional matrix adjust.
 Identities = 51/201 (25%), Positives = 86/201 (42%), Gaps = 33/201 (16%)

Query: 70  RGLVQVWSLEQPEWHCKIDE----GSAGLVASC-WSPDGRHILNTTEFHLRITVWSLCTK 124
           +G V+VW + QP     + +         + SC   PDGR ++   E    +++W L T 
Sbjct: 545 KGCVKVWDISQPGSKSPMAQLDCLNRDNYIRSCKLLPDGRTLIVGGEAST-LSIWDLATP 603

Query: 125 SVSYIKYPKACQQGLTFTRDGRYLALAERRDCRDYVSIFVCSDWQ----------LLRHF 174
           +          +  LT +    Y ALA   D +  V    CSD            L+R F
Sbjct: 604 T-------PRIKAELTSSAPACY-ALAISPDNK--VCFSCCSDGNIVVWDLHNQTLVRQF 653

Query: 175 DTDTQDLTGIEWAPNGCVLAAWDTCLEYKVLLYSL-DGRLLSAYCAYEWSLGIKSVAWSP 233
              T   + I+ + +G  L  W   L+  V  + L +GR L     ++++  I S+ + P
Sbjct: 654 QGHTDGASCIDISNDGTKL--WTGGLDNTVRCWDLREGRQLQ---QHDFTSQIFSLGYCP 708

Query: 234 SSQFLAIGSYDGKVRLLNHVT 254
           + ++LA+G     V +L HV+
Sbjct: 709 TGEWLAVGMESSNVEVL-HVS 728


&gt;ref|XP_002662201.1| PREDICTED: u4/U6 small nuclear ribonucleoprotein Prp4-like [Danio
           rerio]
          Length = 507

 Score = 34.3 bits (77), Expect = 0.21,   Method: Compositional matrix adjust.
 Identities = 61/272 (22%), Positives = 107/272 (39%), Gaps = 31/272 (11%)

Query: 14  CRFSPDGKYLASCVQYRLV----IRDVTTLQILQLYTCLDQIQHIEWSADSLFIL----- 64
           C+FSP+ K L +     L     + D T ++ L+ +     +  I +   +   L     
Sbjct: 222 CQFSPNSKLLVTASWSGLCKLWKVPDCTLIRTLRGHNT--NVGAISFHPQATLTLEESDV 279

Query: 65  ----CAMYRRGLVQVWSLEQPEWHCKIDEGSAGLVASCWSPDGRHILNTTEFHLRITVWS 120
               CA    G V++WSL+  E    I+  S  +    W P GR  L TT +     +W 
Sbjct: 280 NMASCAA--DGSVKLWSLDSDEPIADIEGHSMRVARVTWHPSGR-FLGTTCYDHSWRLWD 336

Query: 121 LCTKSVSYIKYPKACQQG---LTFTRDGRYLALAERRDCRDYVSIFVCSDWQLLRHFDTD 177
           L  +    I + +   +G   L F  DG   +LA       +  ++     + +   +  
Sbjct: 337 LEVQ--EEILHQEGHSKGVHDLHFHPDG---SLAGTGGLDSFARVWDLRTGRCVMFLEGH 391

Query: 178 TQDLTGIEWAPNGCVLAAWDTCLEYKVLLYSLDGRLLSAYCAYEWSLGIKSVAWSPSS-Q 236
            +++  I ++PNG  +A        KV  + L  R          +L + SV + P+   
Sbjct: 392 IKEIYSINFSPNGFHVATGSGDNACKV--WDLRQRRCIYTIPAHQNL-VSSVKFQPNDGH 448

Query: 237 FLAIGSYDGKVRLLNHVTWK-MITEFGHPATI 267
           +L  G+YD   ++  H  W  + T  GH   +
Sbjct: 449 YLLTGAYDNTAKIWTHPGWSPLKTLAGHEGKV 480


&gt;ref|NP_956049.1| U4/U6 small nuclear ribonucleoprotein Prp4 [Danio rerio]
          Length = 507

 Score = 34.3 bits (77), Expect = 0.21,   Method: Compositional matrix adjust.
 Identities = 61/272 (22%), Positives = 107/272 (39%), Gaps = 31/272 (11%)

Query: 14  CRFSPDGKYLASCVQYRLV----IRDVTTLQILQLYTCLDQIQHIEWSADSLFIL----- 64
           C+FSP+ K L +     L     + D T ++ L+ +     +  I +   +   L     
Sbjct: 222 CQFSPNSKLLVTASWSGLCKLWKVPDCTLIRTLRGHNT--NVGAISFHPQATLTLEESDV 279

Query: 65  ----CAMYRRGLVQVWSLEQPEWHCKIDEGSAGLVASCWSPDGRHILNTTEFHLRITVWS 120
               CA    G V++WSL+  E    I+  S  +    W P GR  L TT +     +W 
Sbjct: 280 NMASCAA--DGSVKLWSLDSDEPIADIEGHSMRVARVTWHPSGR-FLGTTCYDHSWRLWD 336

Query: 121 LCTKSVSYIKYPKACQQG---LTFTRDGRYLALAERRDCRDYVSIFVCSDWQLLRHFDTD 177
           L  +    I + +   +G   L F  DG   +LA       +  ++     + +   +  
Sbjct: 337 LEVQ--EEILHQEGHSKGVHDLHFHPDG---SLAGTGGLDSFARVWDLRTGRCVMFLEGH 391

Query: 178 TQDLTGIEWAPNGCVLAAWDTCLEYKVLLYSLDGRLLSAYCAYEWSLGIKSVAWSPSS-Q 236
            +++  I ++PNG  +A        KV  + L  R          +L + SV + P+   
Sbjct: 392 IKEIYSINFSPNGFHVATGSGDNACKV--WDLRQRRCIYTIPAHQNL-VSSVKFQPNDGH 448

Query: 237 FLAIGSYDGKVRLLNHVTWK-MITEFGHPATI 267
           +L  G+YD   ++  H  W  + T  GH   +
Sbjct: 449 YLLTGAYDNTAKIWTHPGWSPLKTLAGHEGKV 480


&gt;ref|NP_998212.1| periodic tryptophan protein 2 homolog [Danio rerio]
          Length = 937

 Score = 33.9 bits (76), Expect = 0.23,   Method: Compositional matrix adjust.
 Identities = 49/222 (22%), Positives = 90/222 (40%), Gaps = 44/222 (19%)

Query: 226 IKSVAWSPSSQFLAIGSYDGKVRLLNHVTWKMITEFGHPATINNPKTVVYKEAEKSPLLG 285
           + S+A+SP  Q+LA G  DGKV++ N  +      F   ++                  G
Sbjct: 388 MNSLAYSPDGQYLATGGDDGKVKVWNTNSGLCFVTFTEHSS------------------G 429

Query: 286 LGHLSFPPPR--AMAGALSTSESKYEIA--SGPVSLQTLKPVADRANPRMGVGML--AFS 339
           +  ++F       ++ +L  +   +++       ++ + +P    +    G G L  A S
Sbjct: 430 VTEVAFTSSGFVVVSASLDGTVRAFDLHRYRNFRTMTSPRPAQFSSLAVDGSGELVCAGS 489

Query: 340 SDSYFLASRNDNVPNAVWIWDIQKLKLFVVLE-HMSPVRSFQWDPQQPRLAICTGGSKVY 398
            DS+            +++W +Q  +L  VL  H  PV +  + P Q  LA  +    V 
Sbjct: 490 QDSF-----------EIFLWSMQTGRLLEVLSGHEGPVSNLCFSPVQSVLASVSWDKTVR 538

Query: 399 LWSPAGC----VSVQVPGEGDFPVLGLCWHLSGDSLALLSKD 436
           LW          ++Q+  +G    L + +H +G  LA+ S D
Sbjct: 539 LWDMLDSWQTKETLQLTSDG----LAVTYHPNGMELAVASLD 576


&gt;ref|NP_001017766.1| polycomb protein eed [Danio rerio]
          Length = 426

 Score = 33.9 bits (76), Expect = 0.24,   Method: Compositional matrix adjust.
 Identities = 20/65 (30%), Positives = 31/65 (47%), Gaps = 5/65 (7%)

Query: 212 RLLSAYCAYEWSLGIKSVAW----SPSSQFLAIGSYDGKVRLLNHVTWKMITEF-GHPAT 266
           RLL +Y   +      + AW    S S   LA+    G +R++NH+T + +  + GH   
Sbjct: 118 RLLQSYVDADADENFYTCAWTFDCSSSHPLLAVAGSRGIIRIINHITMQCVKHYVGHGNA 177

Query: 267 INNPK 271
           IN  K
Sbjct: 178 INELK 182


&gt;ref|NP_998264.1| WD repeat-containing protein 5 [Danio rerio]
          Length = 334

 Score = 33.5 bits (75), Expect = 0.32,   Method: Compositional matrix adjust.
 Identities = 54/237 (22%), Positives = 103/237 (43%), Gaps = 31/237 (13%)

Query: 2   NFSESFKLSGLL-----CRFSPDGKYLASCVQYRLV----IRDVTTLQILQLYTCLDQIQ 52
           N++  F L+G        +FSP G++LAS    +L+      D    + +  +     I 
Sbjct: 34  NYTLKFTLAGHTKAVSSVKFSPSGEWLASSSADKLIKIWGAYDGKFEKTISGHKL--GIS 91

Query: 53  HIEWSADSLFILCAMYRRGLVQVWSLEQPEWHC-KIDEGSAGLVASCWSPDGRHILNTTE 111
            + WS+DS  ++ A   + L ++W +   +  C K  +G +  V  C      +++ +  
Sbjct: 92  DVAWSSDSNLLVSASDDKTL-KIWDVSSGK--CLKTLKGHSNYVFCCNFNPQSNLIVSGS 148

Query: 112 FHLRITVWSL----CTKSVSYIKYPKACQQGLTFTRDGRYLALAERRD--CRDYVSIFVC 165
           F   + +W +    C K++     P +    + F RDG  L ++   D  CR    I+  
Sbjct: 149 FDESVRIWDVKTGKCLKTLPAHSDPVS---AVHFNRDGS-LIVSSSYDGLCR----IWDT 200

Query: 166 SDWQLLRHF-DTDTQDLTGIEWAPNGCVLAAWDTCLEYKVLLYSLDGRLLSAYCAYE 221
           +  Q L+   D D   ++ ++++PNG  + A       K+  YS  G+ L  Y  ++
Sbjct: 201 ASGQCLKTLIDDDNPPVSFVKFSPNGKYILAATLDNTLKLWDYS-KGKCLKTYTGHK 256



 Score = 33.1 bits (74), Expect = 0.48,   Method: Compositional matrix adjust.
 Identities = 29/112 (25%), Positives = 50/112 (44%), Gaps = 5/112 (4%)

Query: 330 RMGVGMLAFSSDSYFLASRNDNVPNAVWIWDIQKLKLFVVLE-HMSPVRSFQWDPQQPRL 388
           ++G+  +A+SSDS  L S +D+    + IWD+   K    L+ H + V    ++PQ   +
Sbjct: 87  KLGISDVAWSSDSNLLVSASDD--KTLKIWDVSSGKCLKTLKGHSNYVFCCNFNPQSNLI 144

Query: 389 AICTGGSKVYLWS-PAGCVSVQVPGEGDFPVLGLCWHLSGDSLALLSKDHFC 439
              +    V +W    G     +P   D PV  + ++  G  +   S D  C
Sbjct: 145 VSGSFDESVRIWDVKTGKCLKTLPAHSD-PVSAVHFNRDGSLIVSSSYDGLC 195


&gt;ref|XP_700740.5| PREDICTED: WD repeat-containing protein 20-like [Danio rerio]
          Length = 546

 Score = 33.5 bits (75), Expect = 0.33,   Method: Compositional matrix adjust.
 Identities = 20/76 (26%), Positives = 36/76 (47%), Gaps = 2/76 (2%)

Query: 51  IQHIEWSADSLFILCAMYRRGLVQVWSLEQPEWHCKIDEGSAGLVASCWSPDGRHILNTT 110
           +    +S D + + C + + G ++V+  +  E    +     GL+   WSPDG+++    
Sbjct: 165 LNEFAFSPDGVHVAC-VGQDGCLRVFHFDSMELQGVMKSYFGGLLCVSWSPDGKYLATGG 223

Query: 111 EFHLRITVWSLCTKSV 126
           E  L +TVWS     V
Sbjct: 224 EDDL-VTVWSFSESRV 238


&gt;ref|XP_692471.3| PREDICTED: transducin-like enhancer protein 1 [Danio rerio]
          Length = 721

 Score = 33.5 bits (75), Expect = 0.34,   Method: Compositional matrix adjust.
 Identities = 52/201 (25%), Positives = 88/201 (43%), Gaps = 33/201 (16%)

Query: 70  RGLVQVWSLEQPEWHCKIDE----GSAGLVASCWS-PDGRHILNTTEFHLRITVWSLCTK 124
           +G V+VW + QP     + +         + SC   PDGR ++   E    +++W L T 
Sbjct: 506 KGCVKVWDISQPGSKSPMAQLDCLNRDNYIRSCKILPDGRTLIVGGEAST-LSIWDLATP 564

Query: 125 SVSYIKYPKACQQGLTFTRDGRYLALAERRDCRDYVSIFVCSDWQ----------LLRHF 174
           +      P+  +  LT +    Y ALA   D  + V    CSD            L+R F
Sbjct: 565 T------PR-IKAELTSSAPACY-ALAISPD--NKVCFSCCSDGNIVVWDLHNQTLVRQF 614

Query: 175 DTDTQDLTGIEWAPNGCVLAAWDTCLEYKVLLYSL-DGRLLSAYCAYEWSLGIKSVAWSP 233
              T   + I+ + +G  L  W   L+  V  + L +GR L     ++++  I S+ + P
Sbjct: 615 QGHTDGASCIDISNDGTKL--WTGGLDNTVRCWDLREGRQLQ---QHDFTSQIFSLGYCP 669

Query: 234 SSQFLAIGSYDGKVRLLNHVT 254
           + ++LA+G     V +L HV+
Sbjct: 670 TGEWLAVGMESSNVEVL-HVS 689


&gt;ref|NP_001189371.1| WD repeat-containing protein 26 [Danio rerio]
          Length = 576

 Score = 33.1 bits (74), Expect = 0.40,   Method: Compositional matrix adjust.
 Identities = 19/67 (28%), Positives = 34/67 (50%), Gaps = 4/67 (5%)

Query: 332 GVGMLAFSSDSYFLASRNDNVPNAVWIWDIQ--KLKLFVVLEHMSPVRSFQWDPQQPRLA 389
           GV  LA+S D  +L +   +  + +W+W++Q  +L+  +   H   + S  W+P   R  
Sbjct: 315 GVSYLAWSPDDVYLIACGPDDCSELWLWNVQTGELRTKMSQSHEDSLTSVAWNPDGKRFV 374

Query: 390 ICTGGSK 396
             TGG +
Sbjct: 375 --TGGQR 379


&gt;ref|NP_001008625.1| intraflagellar transport protein 80 homolog [Danio rerio]
          Length = 777

 Score = 33.1 bits (74), Expect = 0.45,   Method: Compositional matrix adjust.
 Identities = 24/73 (32%), Positives = 41/73 (56%), Gaps = 7/73 (9%)

Query: 184 IEW-APNGCVLAAWDTCLEYKVLLYSLDGRLLSAYCAYEWSLGIKSVAWSPSSQFLAIGS 242
           ++W A N  +L+  + C +YKV  +   GRLL +   +++   I SVAW+P  +  A+GS
Sbjct: 194 VDWSAVNDLILSGGEDC-KYKV--WDSYGRLLFSSAPHDYP--ITSVAWAPDGELFAMGS 248

Query: 243 YDGKVRLLNHVTW 255
           +   +RL +   W
Sbjct: 249 FH-TLRLCDKTGW 260


&gt;ref|NP_001181857.1| WD repeat-containing protein 33 isoform 1 [Danio rerio]
          Length = 1102

 Score = 33.1 bits (74), Expect = 0.49,   Method: Compositional matrix adjust.
 Identities = 28/109 (25%), Positives = 45/109 (41%), Gaps = 12/109 (11%)

Query: 340 SDSYFLASRNDNVPNAVWIWDIQKLKLFVVLE-HMSPVRSFQWDPQQPRLAICTGGSK-- 396
           +D+ F    +D     V IWD  +     +L  H + V+   W P +    +   GSK  
Sbjct: 213 TDNKFATCSDDGT---VRIWDFLRCHEERILRGHGADVKCVDWHPTK---GLVVSGSKDS 266

Query: 397 ---VYLWSPAGCVSVQVPGEGDFPVLGLCWHLSGDSLALLSKDHFCLCF 442
              +  W P    S+         V+ + W+L+G+ L   S+DH C  F
Sbjct: 267 QQPIKFWDPKTGQSLATLHAHKNTVMEVKWNLNGNWLLTASRDHLCKLF 315


&gt;ref|NP_998177.1| chromatin assembly factor 1 subunit B [Danio rerio]
          Length = 236

 Score = 32.3 bits (72), Expect = 0.73,   Method: Compositional matrix adjust.
 Identities = 17/49 (34%), Positives = 29/49 (59%), Gaps = 3/49 (6%)

Query: 336 LAFSSDSYFLASRNDNVPNAVWIWDIQK-LKLFVVLEHMSPVRSFQWDP 383
           ++++SD  F+AS   +V N   +WD+ K  K+ +  +H S V+   WDP
Sbjct: 136 ISWTSDGNFMAS--GSVDNTAIMWDVNKGQKMCIFNDHKSYVQGVAWDP 182


&gt;ref|NP_991272.1| mitotic checkpoint protein BUB3 [Danio rerio]
          Length = 326

 Score = 32.0 bits (71), Expect = 1.1,   Method: Compositional matrix adjust.
 Identities = 17/43 (39%), Positives = 26/43 (60%), Gaps = 1/43 (2%)

Query: 226 IKSVAWSPSS-QFLAIGSYDGKVRLLNHVTWKMITEFGHPATI 267
           + +V +SPSS QFL + S+DG VRL +     M  ++ H A +
Sbjct: 17  VSAVKFSPSSSQFLLVSSWDGSVRLYDASANSMRMKYQHLAPV 59


&gt;ref|NP_998321.1| actin-related protein 2/3 complex subunit 1B [Danio rerio]
          Length = 369

 Score = 32.0 bits (71), Expect = 1.1,   Method: Compositional matrix adjust.
 Identities = 12/40 (30%), Positives = 18/40 (45%)

Query: 365 KLFVVLEHMSPVRSFQWDPQQPRLAICTGGSKVYLWSPAG 404
           K+ V+ EH   V    W P+  R+  C      Y+W+  G
Sbjct: 44  KIHVLKEHNGQVTGIDWAPESNRIVTCGTDRNAYVWTLKG 83


&gt;ref|XP_696012.3| PREDICTED: sperm-associated antigen 5 [Danio rerio]
          Length = 1532

 Score = 31.6 bits (70), Expect = 1.2,   Method: Composition-based stats.
 Identities = 18/66 (27%), Positives = 30/66 (45%), Gaps = 8/66 (12%)

Query: 263 HPATINNPKTVVYKEAEKSPLLGLGHLSFPPPRAMAGALSTSESKYEIASGPVSLQTLKP 322
           H  +  NP + ++ E  +SP+        PPP   + +L  + S   + S P   + +KP
Sbjct: 557 HDPSAENPMSHLWPELPESPM--------PPPLLNSTSLVNAFSYTPVPSDPPKKKDVKP 608

Query: 323 VADRAN 328
            AD  N
Sbjct: 609 SADHQN 614


&gt;ref|NP_001076463.1| F-box-like/WD repeat-containing protein TBL1X [Danio rerio]
          Length = 510

 Score = 31.2 bits (69), Expect = 1.5,   Method: Compositional matrix adjust.
 Identities = 43/213 (20%), Positives = 82/213 (38%), Gaps = 28/213 (13%)

Query: 51  IQHIEWSADSLFILCAMYRRGLVQVWSLEQPEWHCKIDEGSAGLVASCWSPDGRHILNTT 110
           +  ++W+++   +    Y  G  ++W+ +       + +    + A  W+  G  IL+  
Sbjct: 224 VTSLDWNSEGTLLATGSYD-GFARIWT-KDGNLSSTLGQHKGPIFALKWNKKGNSILSAG 281

Query: 111 EFHLRITVWSLCTKSVSYIKYPKACQQGLTFTRDGRYLALAERRDCRDYVSIFVC--SDW 168
                I +W   T      ++P      L    D  +   +    C   + I VC     
Sbjct: 282 VDKTTI-IWDAHTGEAKQ-QFPFHSAPAL----DVDWQNNSTFASCSTDMCIHVCRLGSE 335

Query: 169 QLLRHFDTDTQDLTGIEWAPNGCVLAAWDTCLEYKVLLYSLDGRLLSAYCAYEW---SLG 225
           + L+ F   T ++  I+W P+G +LA+    +  K+     D       C ++    S  
Sbjct: 336 RPLKTFQGHTNEVNAIKWDPSGMLLASCSDDMTLKIWSMKQDS------CVHDLQAHSKE 389

Query: 226 IKSVAWSPS---------SQFLAIGSYDGKVRL 249
           I ++ WSP+         +  LA  S+D  VRL
Sbjct: 390 IYTIKWSPTGIGSNNPNANILLASASFDSTVRL 422



 Score = 30.4 bits (67), Expect = 2.8,   Method: Compositional matrix adjust.
 Identities = 20/74 (27%), Positives = 37/74 (50%), Gaps = 2/74 (2%)

Query: 188 PNGCVLAAWDTCLEYKVLLYSLDGRLLSAYCAYEWSLGIKSVAWSPSSQFLAIGSYDGKV 247
           PN  +L A     +  V L+ +D R +  +        + SVA+SP  +FLA GS+D  V
Sbjct: 405 PNANILLA-SASFDSTVRLWDVD-RGVCTHTLTRHQEPVYSVAFSPDGKFLASGSFDKCV 462

Query: 248 RLLNHVTWKMITEF 261
            + + ++  ++  +
Sbjct: 463 HIWDTMSGSLVNSY 476


&gt;ref|XP_001345204.3| PREDICTED: WD repeat-containing protein 65-like [Danio rerio]
          Length = 686

 Score = 31.2 bits (69), Expect = 1.9,   Method: Compositional matrix adjust.
 Identities = 18/66 (27%), Positives = 33/66 (50%), Gaps = 5/66 (7%)

Query: 336 LAFSSDSYFLASRNDNVPN---AVWIWDIQKLKLFVVLEHMSPVRSFQWDPQQPRLAICT 392
           +AFS DS +L  +    P+    +W+W+ +K+   V + +  P+    ++PQ     IC 
Sbjct: 111 MAFSPDSKYLIGQAGG-PDWTLFLWMWEKKKVMATVNISNNGPINQVSFNPQDNS-KICV 168

Query: 393 GGSKVY 398
            G  V+
Sbjct: 169 SGKGVF 174


&gt;ref|NP_956412.1| POC1 centriolar protein homolog B [Danio rerio]
          Length = 490

 Score = 30.8 bits (68), Expect = 2.3,   Method: Compositional matrix adjust.
 Identities = 32/148 (21%), Positives = 64/148 (43%), Gaps = 23/148 (15%)

Query: 14  CRFSPDGKYLASCVQYRLV-IRDVTTLQILQLYTCLDQIQHIEWSADSLFI--------L 64
            RFSPDG+ +ASC   R V + D ++ Q   ++T        ++   + F+        +
Sbjct: 150 ARFSPDGRLIASCGDDRTVRLWDTSSHQCTNIFT--------DYGGSATFVDFNSSGTCI 201

Query: 65  CAMYRRGLVQVWSLEQPE--WHCKIDEGSAGLVASCWSPDGRHILNTTEFHLRITVWSLC 122
            +      +++W +   +   H K+   +AG+    + P G ++++ +     I +  L 
Sbjct: 202 ASSGADNTIKIWDIRTNKLIQHYKVH--NAGVNCFSFHPSGNYLISGSS-DSTIKILDLL 258

Query: 123 TKSVSYIKY-PKACQQGLTFTRDGRYLA 149
              + Y  +  K     +TF+RDG   A
Sbjct: 259 EGRLIYTLHGHKGPVLTVTFSRDGDLFA 286



 Score = 29.6 bits (65), Expect = 4.5,   Method: Compositional matrix adjust.
 Identities = 28/103 (27%), Positives = 45/103 (43%), Gaps = 10/103 (9%)

Query: 333 VGMLAFSSDSYFLASRNDNVPNAVWIWDIQKLK-LFVVLEHMSPVRSFQWDPQQPRLAIC 391
           V  + FS D   L + +D+   +V +W +++ K L+ +  H + VR  ++ P    +A C
Sbjct: 105 VRSVNFSRDGQRLVTASDD--KSVKVWGVERKKFLYSLNRHTNWVRCARFSPDGRLIASC 162

Query: 392 TGGSKVYLW--SPAGCVSVQVPGEG-----DFPVLGLCWHLSG 427
                V LW  S   C ++     G     DF   G C   SG
Sbjct: 163 GDDRTVRLWDTSSHQCTNIFTDYGGSATFVDFNSSGTCIASSG 205


&gt;ref|NP_001071237.1| WD repeat, SAM and U-box domain-containing protein 1 [Danio rerio]
          Length = 487

 Score = 30.8 bits (68), Expect = 2.3,   Method: Compositional matrix adjust.
 Identities = 22/71 (30%), Positives = 31/71 (43%), Gaps = 4/71 (5%)

Query: 332 GVGMLAFSSDSYFLASRNDNVPNAVWIWDIQKLKLFVVLEH--MSPVRSFQWDPQQPRLA 389
           GV    FS+   +LAS + +    VW  D  +++   VLEH   SPVR   + P    L 
Sbjct: 56  GVHCCCFSACGQYLASCSTDATTMVWSMDTGEIE--AVLEHPGRSPVRVCAFSPDSSHLV 113

Query: 390 ICTGGSKVYLW 400
                  + LW
Sbjct: 114 SGGSDGSIALW 124


&gt;ref|XP_001342078.1| PREDICTED: probable G-protein coupled receptor-like [Danio rerio]
          Length = 437

 Score = 30.8 bits (68), Expect = 2.5,   Method: Compositional matrix adjust.
 Identities = 19/59 (32%), Positives = 26/59 (44%)

Query: 270 PKTVVYKEAEKSPLLGLGHLSFPPPRAMAGALSTSESKYEIASGPVSLQTLKPVADRAN 328
           P      E E +  L  GH    PP    G++S  +S+    SG V + TL  +A  AN
Sbjct: 8   PAAAELNETELNETLTHGHPIHLPPSMQLGSMSNPQSRLRDMSGLVVMVTLNAIALIAN 66


&gt;ref|XP_002663392.1| PREDICTED: uncharacterized protein C3orf38 [Danio rerio]
          Length = 285

 Score = 30.4 bits (67), Expect = 2.6,   Method: Compositional matrix adjust.
 Identities = 18/57 (31%), Positives = 27/57 (47%)

Query: 381 WDPQQPRLAICTGGSKVYLWSPAGCVSVQVPGEGDFPVLGLCWHLSGDSLALLSKDH 437
           W+    RL +CTG  +V  +S A  VS ++    +   L  C +L G  L  +S  H
Sbjct: 157 WENVSLRLLLCTGEQRVDEFSGAELVSKRLQALAEEERLLFCPNLEGHGLKCMSSPH 213


&gt;ref|NP_001007403.1| transducin beta-like protein 3 [Danio rerio]
          Length = 897

 Score = 30.0 bits (66), Expect = 3.8,   Method: Compositional matrix adjust.
 Identities = 27/90 (30%), Positives = 43/90 (47%), Gaps = 4/90 (4%)

Query: 179 QDLTGIEWAPNGCVLAAWDTCLEYKVLLYSLDGRLLSAYCAYEWSLGIKSVAWSPSSQFL 238
           +D+  +  +PN  +LA+       K+   S D RLL     +    G+ SV +SP  Q L
Sbjct: 485 KDVNSVCVSPNDKLLASGSQDRTAKLWALS-DLRLLGVCRGHR--RGVWSVQFSPVDQIL 541

Query: 239 AIGSYDGKVRLLNHVTWKMITEF-GHPATI 267
              S DG VR+ +   +  +  F GH A++
Sbjct: 542 GSASADGSVRIWSIQDFSCLKTFEGHDASV 571


&gt;ref|NP_958500.1| actin-related protein 2/3 complex subunit 1A [Danio rerio]
          Length = 370

 Score = 29.6 bits (65), Expect = 5.6,   Method: Compositional matrix adjust.
 Identities = 11/40 (27%), Positives = 18/40 (45%)

Query: 362 QKLKLFVVLEHMSPVRSFQWDPQQPRLAICTGGSKVYLWS 401
           Q +K+  + EH   +    W P+  R+  C      Y+WS
Sbjct: 41  QWVKVHELKEHNGHITGIDWAPKSDRIVTCGADRNAYVWS 80


&gt;ref|NP_001002100.1| actin-related protein 2/3 complex subunit 1A [Danio rerio]
          Length = 357

 Score = 29.3 bits (64), Expect = 6.2,   Method: Compositional matrix adjust.
 Identities = 9/38 (23%), Positives = 17/38 (44%)

Query: 364 LKLFVVLEHMSPVRSFQWDPQQPRLAICTGGSKVYLWS 401
           +K+  + EH   +    W P+  R+  C      Y+W+
Sbjct: 43  IKIHELTEHSGRITGIDWAPESNRIVTCASDRNAYVWT 80


&gt;ref|XP_682841.5| PREDICTED: coronin-2A-like [Danio rerio]
          Length = 514

 Score = 28.9 bits (63), Expect = 7.7,   Method: Compositional matrix adjust.
 Identities = 18/62 (29%), Positives = 29/62 (46%), Gaps = 6/62 (9%)

Query: 99  WSPDGRHILNTTEFHLRITVWSLCTKSVSYIKYPKACQQG--LTFTRDGRYLALAERRDC 156
           W P  R +L ++ +  ++ +W L +  V  I     C+    L+F  DG  LA A    C
Sbjct: 139 WHPTARDLLLSSAYDCKVFMWRLDSSDVPVIVINTHCELVLCLSFNADGSLLATA----C 194

Query: 157 RD 158
           +D
Sbjct: 195 KD 196


&gt;ref|NP_001018176.1| WD repeat-containing protein 43 [Danio rerio]
          Length = 650

 Score = 28.9 bits (63), Expect = 8.1,   Method: Compositional matrix adjust.
 Identities = 42/168 (25%), Positives = 67/168 (39%), Gaps = 30/168 (17%)

Query: 10  SGLLCRFSPDG-KYLASCVQY-RLVIRDVTTLQILQLY----------TCLD-----QIQ 52
           + L C FSP   ++LA C Q  RL I +  +  + Q Y          TC+       +Q
Sbjct: 8   AALPCAFSPKSHQFLALCAQDGRLRIWNTESKTLQQEYVPSAHLSAACTCVTWGPCRAVQ 67

Query: 53  HIEW----------SADSLFILCAMYRRGLVQVWSLEQPEWHCKIDEGSAGLVASC-WSP 101
            +            SA+   +L      G + ++S  + + HC +D G +G V S  W P
Sbjct: 68  EVPQRKRRKCDAASSAEQSDLLALGTAAGTILIYSTLKGDLHCTLDGGHSGPVNSVQWHP 127

Query: 102 DGRHILNTTEFHLRITVWSLCTKSVSYI-KYPKACQQGLTFTRDGRYL 148
           +   +L +      I  W L T  V    K  ++    L  + DG+ L
Sbjct: 128 EDS-VLYSGSDDTHIAEWDLKTGKVCCKWKADRSAISSLCISPDGKML 174


&gt;ref|NP_001002312.1| intraflagellar transport protein 172 homolog [Danio rerio]
          Length = 1745

 Score = 28.9 bits (63), Expect = 9.2,   Method: Compositional matrix adjust.
 Identities = 23/94 (24%), Positives = 39/94 (41%), Gaps = 7/94 (7%)

Query: 183 GIEWAPNGCVLAAWDTCLEYKVLLYSLDGRLLSA--YCAYEWSLGIKSVAWSPSSQFLAI 240
            + W  N  ++A  D     K++ Y  +G ++    Y            A SPS Q + +
Sbjct: 202 ALAWGTNSIIVAGCDK----KIVAYGKEGNVIQTFDYSRDRAEKEFTVAASSPSGQSIVV 257

Query: 241 GSYDGKVRLLNHVTWKMITEFGHPATINNPKTVV 274
           GS+D ++R+ N    K   +   P  I N  T+ 
Sbjct: 258 GSFD-RLRVFNWSPRKGTWDESSPKEIPNLYTIT 290


  Database: zebrafish.protein.faa
    Posted date:  Jun 5, 2012  5:30 PM
  Number of letters in database: 14,382,362
  Number of sequences in database:  27,291
  
Lambda     K      H
   0.319    0.132    0.404 

Gapped
Lambda     K      H
   0.267   0.0410    0.140 


Matrix: BLOSUM62
Gap Penalties: Existence: 11, Extension: 1
Number of Sequences: 27291
Number of Hits to DB: 21,135,535
Number of extensions: 930643
Number of successful extensions: 2899
Number of sequences better than 10.0: 191
Number of HSP's gapped: 2825
Number of HSP's successfully gapped: 217
Length of database: 14,382,362
Neighboring words threshold: 11
Window for multiple hits: 40
X1: 16 ( 7.4 bits)
X2: 38 (14.6 bits)
X3: 64 (24.7 bits)
S1: 41 (21.8 bits)
S2: 59 (27.3 bits)
</pre></div></div></div></div><div class="cell text_cell border-box-sizing" tabindex="2"><div class="text_cell_input border-box-sizing" style="display: none; "><div class="CodeMirror CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px;"><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-default" tabindex="-1"><div style="position: relative"><div style="position: relative"><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"><pre><span>x</span></pre></div><pre class="CodeMirror-cursor">&nbsp;</pre><div style="position: relative; z-index: -1"></div><div></div></div></div></div></div></div></div></div><div class="text_cell_render border-box-sizing rendered_html" tabindex="-1" style=""><p>Yeah, lots of output, huh?  You can control that with the '-b' and '-v' flags, which set how many matches and alignments are reported.</p></div></div><div class="cell border-box-sizing code_cell vbox" tabindex="2"><div class="input hbox"><div class="prompt input_prompt">In&nbsp;[31]:</div><div class="input_area box-flex1"><div class="CodeMirror"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 5px; left: 5px; "><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-ipython" tabindex="-1"><div style="position: relative; height: 27px; "><div style="position: relative; top: 0px; "><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"></div><pre class="CodeMirror-cursor" style="top: 0px; left: 0px; ">&nbsp;</pre><div style="position: relative; z-index: -1; display: none; "></div><div style=""><pre><span class="cm-operator">!</span><span class="cm-variable">blastall</span> <span class="cm-operator">-</span><span class="cm-variable">i</span> <span class="cm-variable">mm</span><span class="cm-operator">-</span><span class="cm-variable">first.fa</span> <span class="cm-operator">-</span><span class="cm-variable">d</span> <span class="cm-variable">zebrafish.protein.faa</span> <span class="cm-operator">-</span><span class="cm-variable">p</span> <span class="cm-variable">blastp</span> <span class="cm-operator">-</span><span class="cm-variable">b</span> <span class="cm-number">2</span> <span class="cm-operator">-</span><span class="cm-variable">v</span> <span class="cm-number">2</span> <span class="cm-operator">-</span><span class="cm-variable">e</span> <span class="cm-number">1e-6</span></pre></div></div></div></div></div></div></div></div></div><div class="output vbox" style=""><div class="hbox output_area"><div class="prompt"></div><div class="box-flex1 output_subarea output_text output_stream output_stdout"><pre>BLASTP 2.2.24 [Aug-08-2010]


Reference: Altschul, Stephen F., Thomas L. Madden, Alejandro A. Schaffer, 
Jinghui Zhang, Zheng Zhang, Webb Miller, and David J. Lipman (1997), 
"Gapped BLAST and PSI-BLAST: a new generation of protein database search
programs",  Nucleic Acids Res. 25:3389-3402.

Reference for compositional score matrix adjustment: Altschul, Stephen F., 
John C. Wootton, E. Michael Gertz, Richa Agarwala, Aleksandr Morgulis,
Alejandro A. Schaffer, and Yi-Kuo Yu (2005) "Protein database searches
using compositionally adjusted substitution matrices", FEBS J. 272:5101-5109.

Query= gi|83029092|ref|XP_357633.3| PREDICTED: similar to Ig kappa
chain V-V region L7 precursor [Mus musculus]
         (216 letters)

Database: zebrafish.protein.faa 
           27,291 sequences; 14,382,362 total letters

Searching..................................................done



                                                                 Score    E
Sequences producing significant alignments:                      (bits) Value

ref|NP_001038873.1| uncharacterized protein LOC751695 precursor ...    74   6e-14
ref|XP_002667009.2| PREDICTED: hypothetical protein LOC793066 [D...    74   1e-13

&gt;ref|NP_001038873.1| uncharacterized protein LOC751695 precursor [Danio rerio]
          Length = 238

 Score = 74.3 bits (181), Expect = 6e-14,   Method: Compositional matrix adjust.
 Identities = 41/110 (37%), Positives = 58/110 (52%), Gaps = 7/110 (6%)

Query: 94  LLFWTSASRCD-----IVMTQSPATLSVTPGDRVSLSCRASQS--ISDYLHWYQQKSHES 146
           +L WT A+ C      + +TQ+P+ L   PG  +++ CR S+      Y+ WY QK  E+
Sbjct: 6   ILIWTLAALCGECIGQVTVTQTPSVLLSEPGQSLTVHCRLSRDPVCCYYVSWYLQKPGEA 65

Query: 147 PRLLIKYASQXXXXXXXXXXXXXXXXDFTLSINSVEPEDVGVYYCQNGHS 196
           P+LLI Y +                 DFTLSI+ V+ ED G YYCQ+ HS
Sbjct: 66  PKLLIYYTTVLQSGTASRFSGSGSNSDFTLSISGVQTEDAGHYYCQSYHS 115


&gt;ref|XP_002667009.2| PREDICTED: hypothetical protein LOC793066 [Danio rerio]
          Length = 270

 Score = 73.6 bits (179), Expect = 1e-13,   Method: Compositional matrix adjust.
 Identities = 41/96 (42%), Positives = 55/96 (57%), Gaps = 5/96 (5%)

Query: 105 IVMTQSPATLSVTPGDRVSLSCRASQSIS---DYLHWYQQKSHESPRLLIKYASQ--XXX 159
           + +TQSPA  S+T G+ +++SC+ SQ IS   D + WYQQK  E+P+LLI Y S      
Sbjct: 55  VTVTQSPAVRSITQGETLTISCKTSQPISDCNDCVSWYQQKPGETPKLLIYYISSPYSNT 114

Query: 160 XXXXXXXXXXXXXDFTLSINSVEPEDVGVYYCQNGH 195
                        DFTLSI+ V+ ED G YYC + H
Sbjct: 115 PSRFSGGGSDYGSDFTLSISGVQTEDAGDYYCLSYH 150


BLASTP 2.2.24 [Aug-08-2010]


Reference: Altschul, Stephen F., Thomas L. Madden, Alejandro A. Schaffer, 
Jinghui Zhang, Zheng Zhang, Webb Miller, and David J. Lipman (1997), 
"Gapped BLAST and PSI-BLAST: a new generation of protein database search
programs",  Nucleic Acids Res. 25:3389-3402.

Reference for compositional score matrix adjustment: Altschul, Stephen F., 
John C. Wootton, E. Michael Gertz, Richa Agarwala, Aleksandr Morgulis,
Alejandro A. Schaffer, and Yi-Kuo Yu (2005) "Protein database searches
using compositionally adjusted substitution matrices", FEBS J. 272:5101-5109.

Query= gi|70778726|ref|NP_067474.2| WD repeat domain 8 [Mus musculus]
         (462 letters)

Database: zebrafish.protein.faa 
           27,291 sequences; 14,382,362 total letters

Searching..................................................done



                                                                 Score    E
Sequences producing significant alignments:                      (bits) Value

ref|NP_956187.1| WD repeat-containing protein 8 [Danio rerio]         668   0.0  

&gt;ref|NP_956187.1| WD repeat-containing protein 8 [Danio rerio]
          Length = 446

 Score =  668 bits (1723), Expect = 0.0,   Method: Compositional matrix adjust.
 Identities = 309/452 (68%), Positives = 372/452 (82%), Gaps = 11/452 (2%)

Query: 1   MNFSESFKLSGLLCRFSPDGKYLASCVQYRLVIRDVTTLQILQLYTCLDQIQHIEWSADS 60
           MNFSE FK S  LC+ SPDGKYLA+CVQYRLV+RD+ TLQI+ LYTCLDQ+ H+EWS+DS
Sbjct: 1   MNFSEVFKQSNQLCKVSPDGKYLATCVQYRLVVRDIGTLQIVHLYTCLDQVMHMEWSSDS 60

Query: 61  LFILCAMYRRGLVQVWSLEQPEWHCKIDEGSAGLVASCWSPDGRHILNTTEFHLRITVWS 120
           LFILCA+Y+RGLVQVWSLEQP+WHCKIDEGS GLV+S WSPDGRHILNTTEF+LRITVWS
Sbjct: 61  LFILCAVYKRGLVQVWSLEQPDWHCKIDEGSIGLVSSRWSPDGRHILNTTEFNLRITVWS 120

Query: 121 LCTKSVSYIKYPKACQQGLTFTRDGRYLALAERRDCRDYVSIFVCSDWQLLRHFDTDTQD 180
           LCTKSVSYIKYPKACQ+G+ FT DGRY+ALAERRDC+DY+S+FVC DW LLRHF+++TQD
Sbjct: 121 LCTKSVSYIKYPKACQKGMDFTADGRYMALAERRDCKDYISVFVCDDWHLLRHFESETQD 180

Query: 181 LTGIEWAPNGCVLAAWDTCLEYKVLLYSLDGRLLSAYCAYEWSLGIKSVAWSPSSQFLAI 240
           L G+EW+PNGCVLA WD CLEYK+LLYSLDGRLLS Y AYEWSLGIKSVAWSPSSQFLAI
Sbjct: 181 LAGLEWSPNGCVLAVWDNCLEYKILLYSLDGRLLSFYSAYEWSLGIKSVAWSPSSQFLAI 240

Query: 241 GSYDGKVRLLNHVTWKMITEFGHPATINNPKTVVYKEAEKSPL----LGLGHLSFPPPRA 296
           GSYD KVR+LNH+TWK ITEF HPATI N K VV+KE EK P+    L +  L+      
Sbjct: 241 GSYDEKVRILNHITWKKITEFEHPATITNSKAVVFKEVEKRPVVSEDLSIRQLT------ 294

Query: 297 MAGALSTSESKYEIASGPVSLQTLKPVADRANPRMGVGMLAFSSDSYFLASRNDNVPNAV 356
           +  AL +++SKYEI   PV +  +KP  +RANP++G+  +AFS+D+ +LA++NDN+P ++
Sbjct: 295 VDNALFSTQSKYEITQLPVQVPVVKPDPERANPKIGISAVAFSADNRYLATKNDNMPQSL 354

Query: 357 WIWDIQKLKLFVVLEHMSPVRSFQWDPQQPRLAICTGGSKVYLWSPAGCVSVQVPGEGDF 416
           W+WD+QK  L  VLE  +PVR F WDP  PRLA+CTG +K+Y+WSPAGC+SV VP EG F
Sbjct: 355 WVWDMQKFSLLAVLEQTAPVRCFVWDPHLPRLALCTGNTKLYMWSPAGCISVTVPVEG-F 413

Query: 417 PVLGLCWHLSGDSLALLSKDHFCLCFLETKER 448
            V  L WH +G SL LLSKD  CLC+ + +E+
Sbjct: 414 QVQSLFWHCTGSSLVLLSKDQLCLCYTDAEEQ 445


  Database: zebrafish.protein.faa
    Posted date:  Jun 5, 2012  5:30 PM
  Number of letters in database: 14,382,362
  Number of sequences in database:  27,291
  
Lambda     K      H
   0.319    0.132    0.404 

Gapped
Lambda     K      H
   0.267   0.0410    0.140 


Matrix: BLOSUM62
Gap Penalties: Existence: 11, Extension: 1
Number of Sequences: 27291
Number of Hits to DB: 21,135,535
Number of extensions: 930643
Number of successful extensions: 2899
Number of sequences better than 1.0e-06: 9
Number of HSP's gapped: 2886
Number of HSP's successfully gapped: 9
Length of database: 14,382,362
Neighboring words threshold: 11
Window for multiple hits: 40
X1: 16 ( 7.4 bits)
X2: 38 (14.6 bits)
X3: 64 (24.7 bits)
S1: 41 (21.8 bits)
S2: 119 (50.4 bits)
</pre></div></div></div></div><div class="cell text_cell border-box-sizing" tabindex="2"><div class="text_cell_input border-box-sizing" style="display: none; "><div class="CodeMirror CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px;"><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-default" tabindex="-1"><div style="position: relative"><div style="position: relative"><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"><pre><span>x</span></pre></div><pre class="CodeMirror-cursor">&nbsp;</pre><div style="position: relative; z-index: -1"></div><div></div></div></div></div></div></div></div></div><div class="text_cell_render border-box-sizing rendered_html" tabindex="-1" style=""><p>Much better, right?</p>

<p>This is generally how you control UNIX programs -- you specify parameters with '-X' and then arguments to those parameters right after.</p>

<p>Now, let's dump all that output into a text file using '-o':</p></div></div><div class="cell border-box-sizing code_cell vbox" tabindex="2"><div class="input hbox"><div class="prompt input_prompt">In&nbsp;[32]:</div><div class="input_area box-flex1"><div class="CodeMirror"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 5px; left: 5px; "><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-ipython" tabindex="-1"><div style="position: relative; height: 27px; "><div style="position: relative; top: 0px; "><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"></div><pre class="CodeMirror-cursor" style="top: 0px; left: 0px; ">&nbsp;</pre><div style="position: relative; z-index: -1; display: none; "></div><div style=""><pre><span class="cm-operator">!</span><span class="cm-variable">blastall</span> <span class="cm-operator">-</span><span class="cm-variable">i</span> <span class="cm-variable">mm</span><span class="cm-operator">-</span><span class="cm-variable">first.fa</span> <span class="cm-operator">-</span><span class="cm-variable">d</span> <span class="cm-variable">zebrafish.protein.faa</span> <span class="cm-operator">-</span><span class="cm-variable">p</span> <span class="cm-variable">blastp</span> <span class="cm-operator">-</span><span class="cm-variable">b</span> <span class="cm-number">2</span> <span class="cm-operator">-</span><span class="cm-variable">v</span> <span class="cm-number">2</span> <span class="cm-operator">-</span><span class="cm-variable">o</span> <span class="cm-variable">out.txt</span></pre></div></div></div></div></div></div></div></div></div><div class="output vbox" style="display: none; "></div></div><div class="cell border-box-sizing code_cell vbox" tabindex="2"><div class="input hbox"><div class="prompt input_prompt">In&nbsp;[33]:</div><div class="input_area box-flex1"><div class="CodeMirror"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 5px; left: 5px; "><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-ipython" tabindex="-1"><div style="position: relative; height: 27px; "><div style="position: relative; top: 0px; "><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"></div><pre class="CodeMirror-cursor" style="top: 0px; left: 0px; ">&nbsp;</pre><div style="position: relative; z-index: -1; display: none; "></div><div style=""><pre><span class="cm-operator">!</span><span class="cm-variable">cat</span> <span class="cm-variable">out.txt</span></pre></div></div></div></div></div></div></div></div></div><div class="output vbox" style=""><div class="hbox output_area"><div class="prompt"></div><div class="box-flex1 output_subarea output_text output_stream output_stdout"><pre>BLASTP 2.2.24 [Aug-08-2010]


Reference: Altschul, Stephen F., Thomas L. Madden, Alejandro A. Schaffer, 
Jinghui Zhang, Zheng Zhang, Webb Miller, and David J. Lipman (1997), 
"Gapped BLAST and PSI-BLAST: a new generation of protein database search
programs",  Nucleic Acids Res. 25:3389-3402.

Reference for compositional score matrix adjustment: Altschul, Stephen F., 
John C. Wootton, E. Michael Gertz, Richa Agarwala, Aleksandr Morgulis,
Alejandro A. Schaffer, and Yi-Kuo Yu (2005) "Protein database searches
using compositionally adjusted substitution matrices", FEBS J. 272:5101-5109.

Query= gi|83029092|ref|XP_357633.3| PREDICTED: similar to Ig kappa
chain V-V region L7 precursor [Mus musculus]
         (216 letters)

Database: zebrafish.protein.faa 
           27,291 sequences; 14,382,362 total letters

Searching..................................................done



                                                                 Score    E
Sequences producing significant alignments:                      (bits) Value

ref|NP_001038873.1| uncharacterized protein LOC751695 precursor ...    74   6e-14
ref|XP_002667009.2| PREDICTED: hypothetical protein LOC793066 [D...    74   1e-13

&gt;ref|NP_001038873.1| uncharacterized protein LOC751695 precursor [Danio rerio]
          Length = 238

 Score = 74.3 bits (181), Expect = 6e-14,   Method: Compositional matrix adjust.
 Identities = 41/110 (37%), Positives = 58/110 (52%), Gaps = 7/110 (6%)

Query: 94  LLFWTSASRCD-----IVMTQSPATLSVTPGDRVSLSCRASQS--ISDYLHWYQQKSHES 146
           +L WT A+ C      + +TQ+P+ L   PG  +++ CR S+      Y+ WY QK  E+
Sbjct: 6   ILIWTLAALCGECIGQVTVTQTPSVLLSEPGQSLTVHCRLSRDPVCCYYVSWYLQKPGEA 65

Query: 147 PRLLIKYASQXXXXXXXXXXXXXXXXDFTLSINSVEPEDVGVYYCQNGHS 196
           P+LLI Y +                 DFTLSI+ V+ ED G YYCQ+ HS
Sbjct: 66  PKLLIYYTTVLQSGTASRFSGSGSNSDFTLSISGVQTEDAGHYYCQSYHS 115


&gt;ref|XP_002667009.2| PREDICTED: hypothetical protein LOC793066 [Danio rerio]
          Length = 270

 Score = 73.6 bits (179), Expect = 1e-13,   Method: Compositional matrix adjust.
 Identities = 41/96 (42%), Positives = 55/96 (57%), Gaps = 5/96 (5%)

Query: 105 IVMTQSPATLSVTPGDRVSLSCRASQSIS---DYLHWYQQKSHESPRLLIKYASQ--XXX 159
           + +TQSPA  S+T G+ +++SC+ SQ IS   D + WYQQK  E+P+LLI Y S      
Sbjct: 55  VTVTQSPAVRSITQGETLTISCKTSQPISDCNDCVSWYQQKPGETPKLLIYYISSPYSNT 114

Query: 160 XXXXXXXXXXXXXDFTLSINSVEPEDVGVYYCQNGH 195
                        DFTLSI+ V+ ED G YYC + H
Sbjct: 115 PSRFSGGGSDYGSDFTLSISGVQTEDAGDYYCLSYH 150


BLASTP 2.2.24 [Aug-08-2010]


Reference: Altschul, Stephen F., Thomas L. Madden, Alejandro A. Schaffer, 
Jinghui Zhang, Zheng Zhang, Webb Miller, and David J. Lipman (1997), 
"Gapped BLAST and PSI-BLAST: a new generation of protein database search
programs",  Nucleic Acids Res. 25:3389-3402.

Reference for compositional score matrix adjustment: Altschul, Stephen F., 
John C. Wootton, E. Michael Gertz, Richa Agarwala, Aleksandr Morgulis,
Alejandro A. Schaffer, and Yi-Kuo Yu (2005) "Protein database searches
using compositionally adjusted substitution matrices", FEBS J. 272:5101-5109.

Query= gi|70778726|ref|NP_067474.2| WD repeat domain 8 [Mus musculus]
         (462 letters)

Database: zebrafish.protein.faa 
           27,291 sequences; 14,382,362 total letters

Searching..................................................done



                                                                 Score    E
Sequences producing significant alignments:                      (bits) Value

ref|NP_001032508.1| transcription initiation factor TFIID subuni...    46   7e-05
ref|XP_001919110.3| PREDICTED: echinoderm microtubule associated...    44   2e-04

&gt;ref|NP_001032508.1| transcription initiation factor TFIID subunit 5 [Danio rerio]
          Length = 743

 Score = 45.8 bits (107), Expect = 7e-05,   Method: Compositional matrix adjust.
 Identities = 61/197 (30%), Positives = 92/197 (46%), Gaps = 20/197 (10%)

Query: 79  EQPEWHCKIDEGSAGLV-ASCWSPDGRHILNTTEFHLRITVWSLCTKS--VSYI--KYPK 133
           E+     KI  G +G V    +SPD  ++L+++E    I +WSL T +  V Y    YP 
Sbjct: 473 EKTSSESKILHGHSGPVYGVSFSPDRNYLLSSSE-DGTIRLWSLQTFTCLVGYKGHNYPV 531

Query: 134 ACQQGLTFTRDGRYLALAERRDCRDYVSIFVCSD-WQLLRHFDTDTQDLTGIEWAPNGCV 192
              Q   F+  G Y          D V+    +D +Q LR F     D+T   + PN   
Sbjct: 532 WDTQ---FSPFGYYFVSG----GHDRVARLWATDHYQPLRIFAGHLADVTCTRFHPNSNY 584

Query: 193 LAAWDTCLEYKVLLYS-LDGRLLSAYCAYEWSLGIKSVAWSPSSQFLAIGSYDGKVRLLN 251
           +A   +  +  V L+  L+G  +  +  ++    I S+A+SP+ +FLA GS DG+V L +
Sbjct: 585 VATGSS--DRTVRLWDVLNGNCVRIFTGHKGP--IHSLAFSPNGKFLASGSTDGRVLLWD 640

Query: 252 HVTWKMITEF-GHPATI 267
                MI E  GH  TI
Sbjct: 641 IGHGLMIAELKGHTGTI 657


&gt;ref|XP_001919110.3| PREDICTED: echinoderm microtubule associated protein like 6 [Danio
           rerio]
          Length = 1734

 Score = 44.3 bits (103), Expect = 2e-04,   Method: Compositional matrix adjust.
 Identities = 40/155 (25%), Positives = 57/155 (36%), Gaps = 11/155 (7%)

Query: 273 VVYKEAEKSPLLGLGH------LSFPPPRAMAGALSTSESKYEIASGPVSLQTLKPVADR 326
           VVY   E +    LGH      L+  P +         +  Y       +L T+  + D 
Sbjct: 45  VVYNTREHTQKFYLGHNDDIISLALHPDKIQVATGQVGKDPYICVWDSYALTTVSILRDV 104

Query: 327 ANPRMGVGMLAFSSDSYFLASRNDNVPNAVWIWDIQKLKLFVVLE-HMSPVRSFQWDPQQ 385
                GV  LAF +D   LAS   +  N V +WD +K ++      H   +    WDP  
Sbjct: 105 HT--HGVACLAFDADGQRLASVGLDAKNTVCVWDWKKGRVLATATGHSDRIFDISWDPFL 162

Query: 386 PRLAICTGGSKVYLWSPAGCVSVQVPGEGDFPVLG 420
           P   +  G   +  W  A C +   P  G F   G
Sbjct: 163 PHRLVSCGVKHIKFW--ALCGNALTPKRGIFGKTG 195


  Database: zebrafish.protein.faa
    Posted date:  Jun 5, 2012  5:30 PM
  Number of letters in database: 14,382,362
  Number of sequences in database:  27,291
  
Lambda     K      H
   0.319    0.132    0.404 

Gapped
Lambda     K      H
   0.267   0.0410    0.140 


Matrix: BLOSUM62
Gap Penalties: Existence: 11, Extension: 1
Number of Sequences: 27291
Number of Hits to DB: 21,135,535
Number of extensions: 930643
Number of successful extensions: 2899
Number of sequences better than 10.0: 191
Number of HSP's gapped: 2825
Number of HSP's successfully gapped: 217
Length of database: 14,382,362
Neighboring words threshold: 11
Window for multiple hits: 40
X1: 16 ( 7.4 bits)
X2: 38 (14.6 bits)
X3: 64 (24.7 bits)
S1: 41 (21.8 bits)
S2: 59 (27.3 bits)
</pre></div></div></div></div><div class="cell text_cell border-box-sizing" tabindex="2"><div class="text_cell_input border-box-sizing" style="display: none; "><div class="CodeMirror CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px;"><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-default" tabindex="-1"><div style="position: relative"><div style="position: relative"><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"><pre><span>x</span></pre></div><pre class="CodeMirror-cursor">&nbsp;</pre><div style="position: relative; z-index: -1"></div><div></div></div></div></div></div></div></div></div><div class="text_cell_render border-box-sizing rendered_html" tabindex="-1" style=""><p>OK, and now... we want to do something programmatic.  We're no longer content to look at the BLAST output and drool; we want to ... I dunno, turn it into a spreadsheet.</p>

<p>Here's some Python code to do exactly that:</p></div></div><div class="cell border-box-sizing code_cell vbox" tabindex="2"><div class="input hbox"><div class="prompt input_prompt">In&nbsp;[34]:</div><div class="input_area box-flex1"><div class="CodeMirror"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 5px; left: 5px; "><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-ipython" tabindex="-1"><div style="position: relative; height: 299px; "><div style="position: relative; top: 0px; "><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"></div><pre class="CodeMirror-cursor" style="top: 0px; left: 0px; ">&nbsp;</pre><div style="position: relative; z-index: -1; display: none; "></div><div style=""><pre><span class="cm-keyword">import</span> <span class="cm-variable">csv</span></pre><pre><span class="cm-keyword">import</span> <span class="cm-variable">blastparser</span></pre><pre> </pre><pre><span class="cm-comment"># open the output file for reading</span></pre><pre><span class="cm-variable">fp</span> = <span class="cm-builtin">open</span>(<span class="cm-string">'out.txt'</span>)</pre><pre> </pre><pre><span class="cm-comment"># send output as comma-separated values to stdout</span></pre><pre><span class="cm-variable">output</span> = <span class="cm-variable">csv.writer</span>(<span class="cm-variable">sys.stdout</span>)</pre><pre> </pre><pre><span class="cm-comment"># parse BLAST records</span></pre><pre><span class="cm-keyword">for</span> <span class="cm-variable">record</span> <span class="cm-operator">in</span> <span class="cm-variable">blastparser.parse_fp</span>(<span class="cm-variable">fp</span>):</pre><pre>    <span class="cm-keyword">for</span> <span class="cm-variable">hit</span> <span class="cm-operator">in</span> <span class="cm-variable">record</span>:</pre><pre>        <span class="cm-keyword">for</span> <span class="cm-variable">match</span> <span class="cm-operator">in</span> <span class="cm-variable">hit.matches</span>:</pre><pre>            <span class="cm-comment"># output each match as a separate row</span></pre><pre>            <span class="cm-variable">row</span> = [<span class="cm-variable">record.query_name</span>, <span class="cm-variable">hit.subject_name</span>, <span class="cm-variable">match.score</span>,</pre><pre>                   <span class="cm-variable">match.expect</span>]</pre><pre>            <span class="cm-variable">output.writerow</span>(<span class="cm-variable">row</span>)</pre></div></div></div></div></div></div></div></div></div><div class="output vbox" style=""><div class="hbox output_area"><div class="prompt"></div><div class="box-flex1 output_subarea output_text output_stream output_stdout"><pre>gi|83029092|ref|XP_357633.3|,ref|NP_001038873.1|,74.3,6e-14
gi|83029092|ref|XP_357633.3|,ref|XP_002667009.2|,73.6,1e-13
gi|70778726|ref|NP_067474.2|,ref|NP_001032508.1|,45.8,7e-05
gi|70778726|ref|NP_067474.2|,ref|XP_001919110.3|,44.3,0.0002
</pre></div></div></div></div><div class="cell border-box-sizing code_cell vbox" tabindex="2"><div class="input hbox"><div class="prompt input_prompt">In&nbsp;[35]:</div><div class="input_area box-flex1"><div class="CodeMirror"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 5px; left: 5px; "><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-ipython" tabindex="-1"><div style="position: relative; height: 350px; "><div style="position: relative; top: 0px; "><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"></div><pre class="CodeMirror-cursor" style="top: 0px; left: 0px; ">&nbsp;</pre><div style="position: relative; z-index: -1; display: none; "></div><div style=""><pre><span class="cm-keyword">import</span> <span class="cm-variable">csv</span></pre><pre><span class="cm-keyword">import</span> <span class="cm-variable">blastparser</span></pre><pre> </pre><pre><span class="cm-comment"># open the output file for reading</span></pre><pre><span class="cm-variable">fp</span> = <span class="cm-builtin">open</span>(<span class="cm-string">'out.txt'</span>)</pre><pre> </pre><pre><span class="cm-comment"># send output as comma-separated values to Dropbox/out.csv</span></pre><pre><span class="cm-variable">outfp</span> = <span class="cm-builtin">open</span>(<span class="cm-string">'/root/Dropbox/out.csv'</span>, <span class="cm-string">'wb'</span>)</pre><pre><span class="cm-variable">output</span> = <span class="cm-variable">csv.writer</span>(<span class="cm-variable">outfp</span>)</pre><pre> </pre><pre><span class="cm-comment"># parse BLAST records</span></pre><pre><span class="cm-keyword">for</span> <span class="cm-variable">record</span> <span class="cm-operator">in</span> <span class="cm-variable">blastparser.parse_fp</span>(<span class="cm-variable">fp</span>):</pre><pre>    <span class="cm-keyword">for</span> <span class="cm-variable">hit</span> <span class="cm-operator">in</span> <span class="cm-variable">record</span>:</pre><pre>        <span class="cm-keyword">for</span> <span class="cm-variable">match</span> <span class="cm-operator">in</span> <span class="cm-variable">hit.matches</span>:</pre><pre>            <span class="cm-comment"># output each match as a separate row</span></pre><pre>            <span class="cm-variable">row</span> = [<span class="cm-variable">record.query_name</span>, <span class="cm-variable">hit.subject_name</span>, <span class="cm-variable">match.score</span>,</pre><pre>                   <span class="cm-variable">match.expect</span>]</pre><pre>            <span class="cm-variable">output.writerow</span>(<span class="cm-variable">row</span>)</pre><pre>            </pre><pre><span class="cm-variable">outfp.close</span>()</pre></div></div></div></div></div></div></div></div></div><div class="output vbox" style="display: none; "></div></div><div class="cell border-box-sizing code_cell vbox" tabindex="2"><div class="input hbox"><div class="prompt input_prompt">In&nbsp;[36]:</div><div class="input_area box-flex1"><div class="CodeMirror"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 5px; left: 5px; "><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-ipython" tabindex="-1"><div style="position: relative; height: 27px; "><div style="position: relative; top: 0px; "><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"></div><pre class="CodeMirror-cursor" style="top: 0px; left: 0px; ">&nbsp;</pre><div style="position: relative; z-index: -1; display: none; "></div><div style=""><pre><span class="cm-operator">!</span><span class="cm-variable">cat</span> <span class="cm-operator">/</span><span class="cm-variable">root</span><span class="cm-operator">/</span><span class="cm-variable">Dropbox</span><span class="cm-operator">/</span><span class="cm-variable">out.csv</span></pre></div></div></div></div></div></div></div></div></div><div class="output vbox" style=""><div class="hbox output_area"><div class="prompt"></div><div class="box-flex1 output_subarea output_text output_stream output_stdout"><pre>gi|83029092|ref|XP_357633.3|,ref|NP_001038873.1|,74.3,6e-14

gi|83029092|ref|XP_357633.3|,ref|XP_002667009.2|,73.6,1e-13

gi|70778726|ref|NP_067474.2|,ref|NP_001032508.1|,45.8,7e-05

gi|70778726|ref|NP_067474.2|,ref|XP_001919110.3|,44.3,0.0002

</pre></div></div></div></div><div class="cell border-box-sizing code_cell vbox" tabindex="2"><div class="input hbox"><div class="prompt input_prompt">In&nbsp;[36]:</div><div class="input_area box-flex1"><div class="CodeMirror"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 5px; left: 5px; "><textarea style="position: absolute; padding: 0; width: 1px; height: 1em" wrap="off" autocorrect="off" autocapitalize="off"></textarea></div><div class="CodeMirror-scroll cm-s-ipython" tabindex="-1"><div style="position: relative; height: 27px; "><div style="position: relative; top: 0px; "><div class="CodeMirror-gutter" style="display: none; "><div class="CodeMirror-gutter-text"></div></div><div class="CodeMirror-lines"><div style="position: relative; z-index: 0; outline: none; "><div style="position: absolute; width: 100%; height: 0; overflow: hidden; visibility: hidden; outline: 5px auto none"></div><pre class="CodeMirror-cursor" style="top: 0px; left: 0px; ">&nbsp;</pre><div style="position: relative; z-index: -1; display: none; "></div><div style=""><pre> </pre></div></div></div></div></div></div></div></div></div><div class="output vbox" style="display: none; "></div></div><div class="end_space" style="height: 30%; "></div></div>
</div>
</div>

</div>
<script src="./static-ngs-10-blast_files/jquery-1.7.1.min.js" type="text/javascript" charset="utf-8"></script>
<script src="./static-ngs-10-blast_files/jquery-ui.min.js" type="text/javascript" charset="utf-8"></script>
<script src="./static-ngs-10-blast_files/namespace.js" type="text/javascript" charset="utf-8"></script>
<script src="./static-ngs-10-blast_files/page.js" type="text/javascript" charset="utf-8"></script>
<script src="./static-ngs-10-blast_files/loginwidget.js" type="text/javascript" charset="utf-8"></script>

<script src="./static-ngs-10-blast_files/codemirror.js" charset="utf-8"></script>
<script src="./static-ngs-10-blast_files/python.js" charset="utf-8"></script>
<script src="./static-ngs-10-blast_files/htmlmixed.js" charset="utf-8"></script>
<script src="./static-ngs-10-blast_files/xml.js" charset="utf-8"></script>
<script src="./static-ngs-10-blast_files/javascript.js" charset="utf-8"></script>
<script src="./static-ngs-10-blast_files/css.js" charset="utf-8"></script>
<script src="./static-ngs-10-blast_files/rst.js" charset="utf-8"></script>
<script src="./static-ngs-10-blast_files/markdown.js" charset="utf-8"></script>
<script src="./static-ngs-10-blast_files/Markdown.Converter.js" charset="utf-8"></script>
<script src="./static-ngs-10-blast_files/prettify.js" charset="utf-8"></script>
<script src="./static-ngs-10-blast_files/date.format.js" charset="utf-8"></script>
<script src="./static-ngs-10-blast_files/events.js" type="text/javascript" charset="utf-8"></script>
<script src="./static-ngs-10-blast_files/utils.js" type="text/javascript" charset="utf-8"></script>
<script src="./static-ngs-10-blast_files/initmathjax.js" type="text/javascript" charset="utf-8"></script>
<script src="./static-ngs-10-blast_files/outputarea.js" type="text/javascript" charset="utf-8"></script>
<script src="./static-ngs-10-blast_files/cell.js" type="text/javascript" charset="utf-8"></script>
<script src="./static-ngs-10-blast_files/codecell.js" type="text/javascript" charset="utf-8"></script>
<script src="./static-ngs-10-blast_files/textcell.js" type="text/javascript" charset="utf-8"></script>
<script src="./static-ngs-10-blast_files/kernel.js" type="text/javascript" charset="utf-8"></script>
<script src="./static-ngs-10-blast_files/notebook.js" type="text/javascript" charset="utf-8"></script>
<script src="./static-ngs-10-blast_files/printnotebookmain.js" type="text/javascript" charset="utf-8"></script>



<div style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-family: STIXSizeOneSym, sans-serif; "></div></body></html>